{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import argparse\n",
    "from Bio import AlignIO\n",
    "from Bio.Align import AlignInfo\n",
    "import numpy as np\n",
    "from Bio import SeqIO\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Conserved Sequence python function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get conserved sequences function:\n",
    "def get_conserved_sites(msa_file, threshold):\n",
    "    \n",
    "    # Read the MSA file\n",
    "    alignment = AlignIO.read(msa_file, \"fasta\")\n",
    "\n",
    "    # Create an AlignInfo summary object\n",
    "    summary = AlignInfo.SummaryInfo(alignment) # obtains results from the aligment object created before\n",
    "\n",
    "    # Calculate the conservation of each position in the aln\n",
    "    conservation = summary.pos_specific_score_matrix()\n",
    "\n",
    "    # Get the length of the Alignment(Positions length)\n",
    "    aln_len = alignment.get_alignment_length()\n",
    "\n",
    "    # Get number of sequences for threshold\n",
    "    sequence_number = len(alignment)\n",
    "\n",
    "    # Calculate conservation threshold\n",
    "    thres_val = threshold * sequence_number # number of ocurrences needed to reach the threshold ex: 0.7*100 = 70 ocurrences threshold\n",
    "\n",
    "    # Extract conserved sites\n",
    "    conserved_pos = [] #positions\n",
    "    conserved_keys = [] #AA letter\n",
    "    conserved_prob = [] # %of conservation Ex: 0.7 = 7/10\n",
    "    '''\n",
    "    for position in conservation:\n",
    "        if conservation[position] >= thres_val:\n",
    "            conserved_sites.append(position + 1) # Convert to 1-based indexing\n",
    "    \n",
    "    #return conserved_sites\n",
    "    ''' \n",
    "    for i in range(aln_len):\n",
    "        max_conservation = max(conservation[i].values()) # Most present amino acid in each position number \n",
    "        max_key = max(conservation[i], key = conservation[i].get) # Gets letter from the most present aminoacid in position\n",
    "        if max_conservation >= thres_val:\n",
    "            conserved_pos.append(i+1) # Convert to 1-based indexing\n",
    "            conserved_keys.append(max_key)\n",
    "            conserved_prob.append(max_conservation/sequence_number) # returning it as a %\n",
    "\n",
    "    results = np.array([conserved_keys, conserved_pos, conserved_prob]).T\n",
    "    return results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def map_conserved_sites(original_fasta_file, conserved_sites, msa_file):\n",
    "    # Read the original FASTA file\n",
    "    sequences = SeqIO.to_dict(SeqIO.parse(original_fasta_file, \"fasta\"))\n",
    "\n",
    "    # Find the first sequence dynamically\n",
    "    first_sequence_name = next(iter(sequences))\n",
    "\n",
    "    # Convert first sequence to string\n",
    "    seq1 = str(sequences[first_sequence_name].seq)\n",
    "\n",
    "    # Read the MSA file to count gaps in the first sequence\n",
    "    with open(msa_file, 'r') as f:\n",
    "        msa_record = next(SeqIO.parse(f, 'fasta'))\n",
    "        msa_seq1 = str(msa_record.seq)\n",
    "\n",
    "    # Initialize list to store mapped conserved sites\n",
    "    mapped_conserved_sites = []\n",
    "\n",
    "    # Map conserved sites to the original sequence\n",
    "    for conserved_site in conserved_sites:\n",
    "        conserved_aa, conserved_pos, conserved_prob = conserved_site\n",
    "        conserved_pos = int(conserved_pos)\n",
    "\n",
    "        # Initialize gap count\n",
    "        gap_count = 0\n",
    "\n",
    "        # Adjust the position based on the number of gaps\n",
    "        for i in range(conserved_pos):\n",
    "            if msa_seq1[i] == '-':\n",
    "                gap_count += 1\n",
    "        \n",
    "        # Calculate mapped position\n",
    "        mapped_pos = conserved_pos - gap_count\n",
    "\n",
    "        # Append mapped conserved site to the list\n",
    "        mapped_conserved_sites.append([conserved_aa, mapped_pos, conserved_prob])\n",
    "\n",
    "    return mapped_conserved_sites"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "def map_conserved_sites(original_fasta_file, conserved_sites, msa_file):\n",
    "    # Read the original FASTA file\n",
    "    sequences = SeqIO.to_dict(SeqIO.parse(original_fasta_file, \"fasta\"))\n",
    "\n",
    "    # Find the first sequence dynamically\n",
    "    first_sequence_name = next(iter(sequences))\n",
    "\n",
    "    # Convert first sequence to string\n",
    "    original_seq = str(sequences[first_sequence_name].seq)\n",
    "\n",
    "    # Read the MSA file\n",
    "    alignment = AlignIO.read(msa_file, \"fasta\")\n",
    "    msa_seq1 = str(alignment[0].seq)\n",
    "\n",
    "    # Initialize list to store mapped conserved sites\n",
    "    mapped_conserved_sites = []\n",
    "\n",
    "    # Map conserved sites to the original sequence\n",
    "    for conserved_site in conserved_sites:\n",
    "        conserved_aa, conserved_pos, conserved_prob = conserved_site\n",
    "        conserved_pos = int(conserved_pos)\n",
    "\n",
    "        # Initialize gap count\n",
    "        gap_count = msa_seq1[:conserved_pos].count('-')\n",
    "\n",
    "        # Calculate mapped position\n",
    "        \n",
    "        mapped_pos = conserved_pos - gap_count\n",
    "        print(\"Conserved Position:\", conserved_pos)\n",
    "        print(\"Gap Count:\", gap_count)\n",
    "        print(\"Mapped Position:\", mapped_pos)\n",
    "\n",
    "        # Append mapped conserved site to the list\n",
    "        mapped_conserved_sites.append([conserved_aa, mapped_pos, conserved_prob])\n",
    "\n",
    "    return mapped_conserved_sites\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Look for header \n",
    "if header present:\n",
    "    work with index \n",
    "if not add the sequence\n",
    "\n",
    "then run the MSA\n",
    "then COnseverd on Wild type\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "x = get_conserved_sites(\"data/msa_dummy.fasta\", 0.0) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['M' '1' '0.9']\n",
      " ['G' '2' '0.2']\n",
      " ['G' '3' '0.1']\n",
      " ['G' '4' '0.6']\n",
      " ['T' '5' '0.6']\n",
      " ['V' '6' '0.6']\n",
      " ['V' '7' '0.7']\n",
      " ['S' '8' '0.7']\n",
      " ['G' '9' '0.7']\n",
      " ['G' '10' '0.6']\n",
      " ['S' '11' '0.6']\n",
      " ['M' '12' '0.6']\n",
      " ['N' '13' '0.6']\n",
      " ['R' '14' '0.2']\n",
      " ['D' '15' '0.1']\n",
      " ['E' '16' '0.2']\n",
      " ['A' '17' '0.1']\n",
      " ['L' '18' '0.4']\n",
      " ['L' '19' '0.3']\n",
      " ['L' '20' '0.2']\n",
      " ['R' '21' '0.2']\n",
      " ['L' '22' '0.4']\n",
      " ['L' '23' '0.3']\n",
      " ['A' '24' '0.4']\n",
      " ['L' '25' '0.4']\n",
      " ['Y' '26' '0.6']\n",
      " ['T' '27' '0.5']\n",
      " ['I' '28' '0.5']\n",
      " ['T' '29' '0.4']\n",
      " ['Q' '30' '0.5']\n",
      " ['H' '31' '0.4']\n",
      " ['L' '32' '0.6']\n",
      " ['A' '33' '0.5']\n",
      " ['L' '34' '0.3']\n",
      " ['I' '35' '0.2']\n",
      " ['L' '36' '0.4']\n",
      " ['N' '37' '0.2']\n",
      " ['S' '38' '0.2']\n",
      " ['V' '39' '0.2']\n",
      " ['P' '40' '0.1']\n",
      " ['L' '41' '0.1']\n",
      " ['Q' '42' '0.1']\n",
      " ['L' '43' '0.1']\n",
      " ['L' '44' '0.1']\n",
      " ['L' '45' '0.2']\n",
      " ['H' '46' '0.1']\n",
      " ['A' '47' '0.2']\n",
      " ['G' '48' '0.3']\n",
      " ['I' '49' '0.2']\n",
      " ['A' '50' '0.2']\n",
      " ['P' '51' '0.1']\n",
      " ['I' '52' '0.2']\n",
      " ['G' '53' '0.3']\n",
      " ['H' '54' '0.2']\n",
      " ['K' '55' '0.1']\n",
      " ['K' '56' '0.1']\n",
      " ['A' '57' '0.2']\n",
      " ['A' '58' '0.1']\n",
      " ['K' '59' '0.2']\n",
      " ['G' '60' '0.2']\n",
      " ['G' '61' '0.3']\n",
      " ['A' '62' '0.2']\n",
      " ['D' '63' '0.4']\n",
      " ['Y' '64' '0.4']\n",
      " ['S' '65' '0.4']\n",
      " ['V' '66' '0.4']\n",
      " ['V' '67' '0.5']\n",
      " ['L' '68' '0.3']\n",
      " ['L' '69' '0.3']\n",
      " ['Q' '70' '0.2']\n",
      " ['L' '71' '0.3']\n",
      " ['P' '72' '0.3']\n",
      " ['A' '73' '0.2']\n",
      " ['V' '74' '0.3']\n",
      " ['L' '75' '0.2']\n",
      " ['P' '76' '0.1']\n",
      " ['A' '77' '0.2']\n",
      " ['E' '78' '0.1']\n",
      " ['P' '79' '0.1']\n",
      " ['Q' '80' '0.1']\n",
      " ['D' '81' '0.1']\n",
      " ['V' '82' '0.1']\n",
      " ['L' '83' '0.1']\n",
      " ['L' '84' '0.1']\n",
      " ['E' '85' '0.1']\n",
      " ['H' '86' '0.1']]\n"
     ]
    }
   ],
   "source": [
    "print(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Conserved Position: 215\n",
      "Gap Count: 212\n",
      "Mapped Position: 3\n",
      "Conserved Position: 216\n",
      "Gap Count: 212\n",
      "Mapped Position: 4\n",
      "Conserved Position: 217\n",
      "Gap Count: 212\n",
      "Mapped Position: 5\n",
      "Conserved Position: 219\n",
      "Gap Count: 212\n",
      "Mapped Position: 7\n",
      "Conserved Position: 220\n",
      "Gap Count: 212\n",
      "Mapped Position: 8\n",
      "Conserved Position: 221\n",
      "Gap Count: 212\n",
      "Mapped Position: 9\n",
      "Conserved Position: 222\n",
      "Gap Count: 212\n",
      "Mapped Position: 10\n",
      "Conserved Position: 228\n",
      "Gap Count: 217\n",
      "Mapped Position: 11\n",
      "Conserved Position: 286\n",
      "Gap Count: 267\n",
      "Mapped Position: 19\n",
      "Conserved Position: 288\n",
      "Gap Count: 267\n",
      "Mapped Position: 21\n",
      "Conserved Position: 289\n",
      "Gap Count: 267\n",
      "Mapped Position: 22\n",
      "Conserved Position: 290\n",
      "Gap Count: 267\n",
      "Mapped Position: 23\n",
      "Conserved Position: 309\n",
      "Gap Count: 284\n",
      "Mapped Position: 25\n",
      "Conserved Position: 310\n",
      "Gap Count: 284\n",
      "Mapped Position: 26\n",
      "Conserved Position: 312\n",
      "Gap Count: 284\n",
      "Mapped Position: 28\n",
      "Conserved Position: 313\n",
      "Gap Count: 284\n",
      "Mapped Position: 29\n",
      "Conserved Position: 314\n",
      "Gap Count: 284\n",
      "Mapped Position: 30\n",
      "Conserved Position: 315\n",
      "Gap Count: 284\n",
      "Mapped Position: 31\n",
      "Conserved Position: 316\n",
      "Gap Count: 284\n",
      "Mapped Position: 32\n",
      "Conserved Position: 318\n",
      "Gap Count: 284\n",
      "Mapped Position: 34\n",
      "Conserved Position: 319\n",
      "Gap Count: 284\n",
      "Mapped Position: 35\n",
      "Conserved Position: 320\n",
      "Gap Count: 284\n",
      "Mapped Position: 36\n",
      "Conserved Position: 323\n",
      "Gap Count: 284\n",
      "Mapped Position: 39\n",
      "Conserved Position: 328\n",
      "Gap Count: 288\n",
      "Mapped Position: 40\n",
      "Conserved Position: 330\n",
      "Gap Count: 288\n",
      "Mapped Position: 42\n",
      "Conserved Position: 331\n",
      "Gap Count: 288\n",
      "Mapped Position: 43\n",
      "Conserved Position: 332\n",
      "Gap Count: 288\n",
      "Mapped Position: 44\n",
      "Conserved Position: 354\n",
      "Gap Count: 309\n",
      "Mapped Position: 45\n",
      "Conserved Position: 356\n",
      "Gap Count: 310\n",
      "Mapped Position: 46\n",
      "Conserved Position: 357\n",
      "Gap Count: 310\n",
      "Mapped Position: 47\n",
      "Conserved Position: 358\n",
      "Gap Count: 310\n",
      "Mapped Position: 48\n",
      "Conserved Position: 359\n",
      "Gap Count: 310\n",
      "Mapped Position: 49\n",
      "Conserved Position: 360\n",
      "Gap Count: 310\n",
      "Mapped Position: 50\n",
      "Conserved Position: 393\n",
      "Gap Count: 340\n",
      "Mapped Position: 53\n",
      "Conserved Position: 395\n",
      "Gap Count: 340\n",
      "Mapped Position: 55\n",
      "Conserved Position: 397\n",
      "Gap Count: 340\n",
      "Mapped Position: 57\n",
      "Conserved Position: 398\n",
      "Gap Count: 340\n",
      "Mapped Position: 58\n",
      "Conserved Position: 399\n",
      "Gap Count: 340\n",
      "Mapped Position: 59\n",
      "Conserved Position: 401\n",
      "Gap Count: 340\n",
      "Mapped Position: 61\n",
      "Conserved Position: 402\n",
      "Gap Count: 340\n",
      "Mapped Position: 62\n",
      "Conserved Position: 403\n",
      "Gap Count: 340\n",
      "Mapped Position: 63\n",
      "Conserved Position: 404\n",
      "Gap Count: 340\n",
      "Mapped Position: 64\n",
      "Conserved Position: 460\n",
      "Gap Count: 394\n",
      "Mapped Position: 66\n",
      "Conserved Position: 461\n",
      "Gap Count: 394\n",
      "Mapped Position: 67\n",
      "Conserved Position: 463\n",
      "Gap Count: 394\n",
      "Mapped Position: 69\n",
      "Conserved Position: 494\n",
      "Gap Count: 424\n",
      "Mapped Position: 70\n",
      "Conserved Position: 495\n",
      "Gap Count: 424\n",
      "Mapped Position: 71\n",
      "Conserved Position: 497\n",
      "Gap Count: 425\n",
      "Mapped Position: 72\n",
      "Conserved Position: 498\n",
      "Gap Count: 425\n",
      "Mapped Position: 73\n",
      "Conserved Position: 499\n",
      "Gap Count: 425\n",
      "Mapped Position: 74\n",
      "Conserved Position: 500\n",
      "Gap Count: 425\n",
      "Mapped Position: 75\n",
      "Conserved Position: 501\n",
      "Gap Count: 425\n",
      "Mapped Position: 76\n",
      "Conserved Position: 502\n",
      "Gap Count: 425\n",
      "Mapped Position: 77\n",
      "Conserved Position: 503\n",
      "Gap Count: 425\n",
      "Mapped Position: 78\n",
      "Conserved Position: 504\n",
      "Gap Count: 425\n",
      "Mapped Position: 79\n",
      "Conserved Position: 505\n",
      "Gap Count: 425\n",
      "Mapped Position: 80\n",
      "Conserved Position: 506\n",
      "Gap Count: 425\n",
      "Mapped Position: 81\n",
      "Conserved Position: 507\n",
      "Gap Count: 425\n",
      "Mapped Position: 82\n",
      "Conserved Position: 514\n",
      "Gap Count: 431\n",
      "Mapped Position: 83\n",
      "Conserved Position: 515\n",
      "Gap Count: 431\n",
      "Mapped Position: 84\n",
      "Conserved Position: 516\n",
      "Gap Count: 431\n",
      "Mapped Position: 85\n",
      "Conserved Position: 517\n",
      "Gap Count: 431\n",
      "Mapped Position: 86\n",
      "Conserved Position: 518\n",
      "Gap Count: 431\n",
      "Mapped Position: 87\n",
      "Conserved Position: 519\n",
      "Gap Count: 431\n",
      "Mapped Position: 88\n",
      "Conserved Position: 520\n",
      "Gap Count: 431\n",
      "Mapped Position: 89\n",
      "Conserved Position: 522\n",
      "Gap Count: 431\n",
      "Mapped Position: 91\n",
      "Conserved Position: 561\n",
      "Gap Count: 469\n",
      "Mapped Position: 92\n",
      "Conserved Position: 562\n",
      "Gap Count: 469\n",
      "Mapped Position: 93\n",
      "Conserved Position: 563\n",
      "Gap Count: 469\n",
      "Mapped Position: 94\n",
      "Conserved Position: 564\n",
      "Gap Count: 469\n",
      "Mapped Position: 95\n",
      "Conserved Position: 565\n",
      "Gap Count: 469\n",
      "Mapped Position: 96\n",
      "Conserved Position: 566\n",
      "Gap Count: 469\n",
      "Mapped Position: 97\n",
      "Conserved Position: 567\n",
      "Gap Count: 469\n",
      "Mapped Position: 98\n",
      "Conserved Position: 568\n",
      "Gap Count: 469\n",
      "Mapped Position: 99\n",
      "Conserved Position: 569\n",
      "Gap Count: 469\n",
      "Mapped Position: 100\n",
      "Conserved Position: 570\n",
      "Gap Count: 469\n",
      "Mapped Position: 101\n",
      "Conserved Position: 572\n",
      "Gap Count: 469\n",
      "Mapped Position: 103\n",
      "Conserved Position: 573\n",
      "Gap Count: 469\n",
      "Mapped Position: 104\n",
      "Conserved Position: 579\n",
      "Gap Count: 471\n",
      "Mapped Position: 108\n",
      "Conserved Position: 587\n",
      "Gap Count: 476\n",
      "Mapped Position: 111\n",
      "Conserved Position: 588\n",
      "Gap Count: 476\n",
      "Mapped Position: 112\n",
      "Conserved Position: 589\n",
      "Gap Count: 476\n",
      "Mapped Position: 113\n",
      "Conserved Position: 590\n",
      "Gap Count: 476\n",
      "Mapped Position: 114\n",
      "Conserved Position: 591\n",
      "Gap Count: 476\n",
      "Mapped Position: 115\n",
      "Conserved Position: 592\n",
      "Gap Count: 476\n",
      "Mapped Position: 116\n",
      "Conserved Position: 593\n",
      "Gap Count: 476\n",
      "Mapped Position: 117\n",
      "Conserved Position: 594\n",
      "Gap Count: 476\n",
      "Mapped Position: 118\n",
      "Conserved Position: 597\n",
      "Gap Count: 478\n",
      "Mapped Position: 119\n",
      "Conserved Position: 598\n",
      "Gap Count: 478\n",
      "Mapped Position: 120\n",
      "Conserved Position: 599\n",
      "Gap Count: 478\n",
      "Mapped Position: 121\n",
      "Conserved Position: 600\n",
      "Gap Count: 478\n",
      "Mapped Position: 122\n",
      "Conserved Position: 601\n",
      "Gap Count: 478\n",
      "Mapped Position: 123\n",
      "Conserved Position: 602\n",
      "Gap Count: 478\n",
      "Mapped Position: 124\n",
      "Conserved Position: 603\n",
      "Gap Count: 478\n",
      "Mapped Position: 125\n",
      "Conserved Position: 604\n",
      "Gap Count: 478\n",
      "Mapped Position: 126\n",
      "Conserved Position: 605\n",
      "Gap Count: 478\n",
      "Mapped Position: 127\n",
      "Conserved Position: 606\n",
      "Gap Count: 478\n",
      "Mapped Position: 128\n",
      "Conserved Position: 608\n",
      "Gap Count: 478\n",
      "Mapped Position: 130\n",
      "Conserved Position: 609\n",
      "Gap Count: 478\n",
      "Mapped Position: 131\n",
      "Conserved Position: 611\n",
      "Gap Count: 478\n",
      "Mapped Position: 133\n",
      "Conserved Position: 612\n",
      "Gap Count: 478\n",
      "Mapped Position: 134\n",
      "Conserved Position: 613\n",
      "Gap Count: 478\n",
      "Mapped Position: 135\n",
      "Conserved Position: 614\n",
      "Gap Count: 478\n",
      "Mapped Position: 136\n",
      "Conserved Position: 615\n",
      "Gap Count: 478\n",
      "Mapped Position: 137\n",
      "Conserved Position: 617\n",
      "Gap Count: 478\n",
      "Mapped Position: 139\n",
      "Conserved Position: 619\n",
      "Gap Count: 478\n",
      "Mapped Position: 141\n",
      "Conserved Position: 621\n",
      "Gap Count: 478\n",
      "Mapped Position: 143\n",
      "Conserved Position: 628\n",
      "Gap Count: 481\n",
      "Mapped Position: 147\n",
      "Conserved Position: 630\n",
      "Gap Count: 481\n",
      "Mapped Position: 149\n",
      "Conserved Position: 631\n",
      "Gap Count: 481\n",
      "Mapped Position: 150\n",
      "Conserved Position: 632\n",
      "Gap Count: 481\n",
      "Mapped Position: 151\n",
      "Conserved Position: 633\n",
      "Gap Count: 481\n",
      "Mapped Position: 152\n",
      "Conserved Position: 636\n",
      "Gap Count: 481\n",
      "Mapped Position: 155\n",
      "Conserved Position: 645\n",
      "Gap Count: 488\n",
      "Mapped Position: 157\n",
      "Conserved Position: 647\n",
      "Gap Count: 488\n",
      "Mapped Position: 159\n",
      "Conserved Position: 648\n",
      "Gap Count: 488\n",
      "Mapped Position: 160\n",
      "Conserved Position: 649\n",
      "Gap Count: 488\n",
      "Mapped Position: 161\n",
      "Conserved Position: 650\n",
      "Gap Count: 488\n",
      "Mapped Position: 162\n",
      "Conserved Position: 651\n",
      "Gap Count: 488\n",
      "Mapped Position: 163\n",
      "Conserved Position: 652\n",
      "Gap Count: 488\n",
      "Mapped Position: 164\n",
      "Conserved Position: 654\n",
      "Gap Count: 488\n",
      "Mapped Position: 166\n",
      "Conserved Position: 655\n",
      "Gap Count: 488\n",
      "Mapped Position: 167\n",
      "Conserved Position: 656\n",
      "Gap Count: 488\n",
      "Mapped Position: 168\n",
      "Conserved Position: 657\n",
      "Gap Count: 488\n",
      "Mapped Position: 169\n",
      "Conserved Position: 658\n",
      "Gap Count: 488\n",
      "Mapped Position: 170\n",
      "Conserved Position: 661\n",
      "Gap Count: 490\n",
      "Mapped Position: 171\n",
      "Conserved Position: 662\n",
      "Gap Count: 490\n",
      "Mapped Position: 172\n",
      "Conserved Position: 663\n",
      "Gap Count: 490\n",
      "Mapped Position: 173\n",
      "Conserved Position: 664\n",
      "Gap Count: 490\n",
      "Mapped Position: 174\n",
      "Conserved Position: 665\n",
      "Gap Count: 490\n",
      "Mapped Position: 175\n",
      "Conserved Position: 668\n",
      "Gap Count: 490\n",
      "Mapped Position: 178\n",
      "Conserved Position: 669\n",
      "Gap Count: 490\n",
      "Mapped Position: 179\n",
      "Conserved Position: 670\n",
      "Gap Count: 490\n",
      "Mapped Position: 180\n",
      "Conserved Position: 671\n",
      "Gap Count: 490\n",
      "Mapped Position: 181\n",
      "Conserved Position: 672\n",
      "Gap Count: 490\n",
      "Mapped Position: 182\n",
      "Conserved Position: 673\n",
      "Gap Count: 490\n",
      "Mapped Position: 183\n",
      "Conserved Position: 674\n",
      "Gap Count: 490\n",
      "Mapped Position: 184\n",
      "Conserved Position: 675\n",
      "Gap Count: 490\n",
      "Mapped Position: 185\n",
      "Conserved Position: 676\n",
      "Gap Count: 490\n",
      "Mapped Position: 186\n",
      "Conserved Position: 678\n",
      "Gap Count: 490\n",
      "Mapped Position: 188\n",
      "Conserved Position: 680\n",
      "Gap Count: 490\n",
      "Mapped Position: 190\n",
      "Conserved Position: 681\n",
      "Gap Count: 490\n",
      "Mapped Position: 191\n",
      "Conserved Position: 682\n",
      "Gap Count: 490\n",
      "Mapped Position: 192\n",
      "Conserved Position: 684\n",
      "Gap Count: 490\n",
      "Mapped Position: 194\n",
      "Conserved Position: 688\n",
      "Gap Count: 490\n",
      "Mapped Position: 198\n",
      "Conserved Position: 691\n",
      "Gap Count: 490\n",
      "Mapped Position: 201\n",
      "Conserved Position: 693\n",
      "Gap Count: 490\n",
      "Mapped Position: 203\n",
      "Conserved Position: 703\n",
      "Gap Count: 496\n",
      "Mapped Position: 207\n",
      "Conserved Position: 716\n",
      "Gap Count: 502\n",
      "Mapped Position: 214\n",
      "Conserved Position: 717\n",
      "Gap Count: 502\n",
      "Mapped Position: 215\n",
      "Conserved Position: 719\n",
      "Gap Count: 502\n",
      "Mapped Position: 217\n",
      "Conserved Position: 742\n",
      "Gap Count: 521\n",
      "Mapped Position: 221\n",
      "Conserved Position: 743\n",
      "Gap Count: 521\n",
      "Mapped Position: 222\n",
      "Conserved Position: 744\n",
      "Gap Count: 521\n",
      "Mapped Position: 223\n",
      "Conserved Position: 745\n",
      "Gap Count: 521\n",
      "Mapped Position: 224\n",
      "Conserved Position: 746\n",
      "Gap Count: 521\n",
      "Mapped Position: 225\n",
      "Conserved Position: 747\n",
      "Gap Count: 521\n",
      "Mapped Position: 226\n",
      "Conserved Position: 748\n",
      "Gap Count: 521\n",
      "Mapped Position: 227\n",
      "Conserved Position: 749\n",
      "Gap Count: 521\n",
      "Mapped Position: 228\n",
      "Conserved Position: 750\n",
      "Gap Count: 521\n",
      "Mapped Position: 229\n",
      "Conserved Position: 751\n",
      "Gap Count: 521\n",
      "Mapped Position: 230\n",
      "Conserved Position: 752\n",
      "Gap Count: 521\n",
      "Mapped Position: 231\n",
      "Conserved Position: 753\n",
      "Gap Count: 521\n",
      "Mapped Position: 232\n",
      "Conserved Position: 755\n",
      "Gap Count: 521\n",
      "Mapped Position: 234\n",
      "Conserved Position: 757\n",
      "Gap Count: 521\n",
      "Mapped Position: 236\n",
      "Conserved Position: 758\n",
      "Gap Count: 521\n",
      "Mapped Position: 237\n",
      "Conserved Position: 759\n",
      "Gap Count: 521\n",
      "Mapped Position: 238\n",
      "Conserved Position: 760\n",
      "Gap Count: 521\n",
      "Mapped Position: 239\n",
      "Conserved Position: 761\n",
      "Gap Count: 521\n",
      "Mapped Position: 240\n",
      "Conserved Position: 764\n",
      "Gap Count: 522\n",
      "Mapped Position: 242\n",
      "Conserved Position: 765\n",
      "Gap Count: 522\n",
      "Mapped Position: 243\n",
      "Conserved Position: 767\n",
      "Gap Count: 522\n",
      "Mapped Position: 245\n",
      "Conserved Position: 768\n",
      "Gap Count: 522\n",
      "Mapped Position: 246\n",
      "Conserved Position: 769\n",
      "Gap Count: 522\n",
      "Mapped Position: 247\n",
      "Conserved Position: 771\n",
      "Gap Count: 522\n",
      "Mapped Position: 249\n",
      "Conserved Position: 803\n",
      "Gap Count: 552\n",
      "Mapped Position: 251\n",
      "Conserved Position: 804\n",
      "Gap Count: 552\n",
      "Mapped Position: 252\n",
      "Conserved Position: 805\n",
      "Gap Count: 552\n",
      "Mapped Position: 253\n",
      "Conserved Position: 806\n",
      "Gap Count: 552\n",
      "Mapped Position: 254\n",
      "Conserved Position: 807\n",
      "Gap Count: 552\n",
      "Mapped Position: 255\n",
      "Conserved Position: 808\n",
      "Gap Count: 552\n",
      "Mapped Position: 256\n",
      "Conserved Position: 810\n",
      "Gap Count: 553\n",
      "Mapped Position: 257\n",
      "Conserved Position: 811\n",
      "Gap Count: 553\n",
      "Mapped Position: 258\n",
      "Conserved Position: 812\n",
      "Gap Count: 553\n",
      "Mapped Position: 259\n",
      "Conserved Position: 813\n",
      "Gap Count: 553\n",
      "Mapped Position: 260\n",
      "Conserved Position: 815\n",
      "Gap Count: 553\n",
      "Mapped Position: 262\n",
      "Conserved Position: 818\n",
      "Gap Count: 553\n",
      "Mapped Position: 265\n",
      "Conserved Position: 823\n",
      "Gap Count: 557\n",
      "Mapped Position: 266\n",
      "Conserved Position: 824\n",
      "Gap Count: 557\n",
      "Mapped Position: 267\n",
      "Conserved Position: 826\n",
      "Gap Count: 557\n",
      "Mapped Position: 269\n",
      "Conserved Position: 827\n",
      "Gap Count: 557\n",
      "Mapped Position: 270\n",
      "Conserved Position: 830\n",
      "Gap Count: 558\n",
      "Mapped Position: 272\n",
      "Conserved Position: 851\n",
      "Gap Count: 578\n",
      "Mapped Position: 273\n",
      "Conserved Position: 853\n",
      "Gap Count: 578\n",
      "Mapped Position: 275\n",
      "Conserved Position: 854\n",
      "Gap Count: 578\n",
      "Mapped Position: 276\n",
      "Conserved Position: 855\n",
      "Gap Count: 578\n",
      "Mapped Position: 277\n",
      "Conserved Position: 856\n",
      "Gap Count: 578\n",
      "Mapped Position: 278\n",
      "Conserved Position: 858\n",
      "Gap Count: 578\n",
      "Mapped Position: 280\n",
      "Conserved Position: 863\n",
      "Gap Count: 582\n",
      "Mapped Position: 281\n",
      "Conserved Position: 864\n",
      "Gap Count: 582\n",
      "Mapped Position: 282\n",
      "Conserved Position: 866\n",
      "Gap Count: 583\n",
      "Mapped Position: 283\n",
      "Conserved Position: 867\n",
      "Gap Count: 583\n",
      "Mapped Position: 284\n",
      "Conserved Position: 868\n",
      "Gap Count: 583\n",
      "Mapped Position: 285\n",
      "Conserved Position: 869\n",
      "Gap Count: 583\n",
      "Mapped Position: 286\n",
      "Conserved Position: 870\n",
      "Gap Count: 583\n",
      "Mapped Position: 287\n",
      "Conserved Position: 871\n",
      "Gap Count: 583\n",
      "Mapped Position: 288\n",
      "Conserved Position: 872\n",
      "Gap Count: 583\n",
      "Mapped Position: 289\n",
      "Conserved Position: 873\n",
      "Gap Count: 583\n",
      "Mapped Position: 290\n",
      "Conserved Position: 874\n",
      "Gap Count: 583\n",
      "Mapped Position: 291\n",
      "Conserved Position: 875\n",
      "Gap Count: 583\n",
      "Mapped Position: 292\n",
      "Conserved Position: 876\n",
      "Gap Count: 583\n",
      "Mapped Position: 293\n",
      "Conserved Position: 877\n",
      "Gap Count: 583\n",
      "Mapped Position: 294\n",
      "Conserved Position: 878\n",
      "Gap Count: 583\n",
      "Mapped Position: 295\n",
      "Conserved Position: 880\n",
      "Gap Count: 583\n",
      "Mapped Position: 297\n",
      "Conserved Position: 881\n",
      "Gap Count: 583\n",
      "Mapped Position: 298\n",
      "Conserved Position: 906\n",
      "Gap Count: 598\n",
      "Mapped Position: 308\n",
      "Conserved Position: 907\n",
      "Gap Count: 598\n",
      "Mapped Position: 309\n",
      "Conserved Position: 908\n",
      "Gap Count: 598\n",
      "Mapped Position: 310\n",
      "Conserved Position: 909\n",
      "Gap Count: 598\n",
      "Mapped Position: 311\n",
      "Conserved Position: 910\n",
      "Gap Count: 598\n",
      "Mapped Position: 312\n",
      "Conserved Position: 911\n",
      "Gap Count: 598\n",
      "Mapped Position: 313\n",
      "Conserved Position: 913\n",
      "Gap Count: 598\n",
      "Mapped Position: 315\n",
      "Conserved Position: 914\n",
      "Gap Count: 598\n",
      "Mapped Position: 316\n",
      "Conserved Position: 915\n",
      "Gap Count: 598\n",
      "Mapped Position: 317\n",
      "Conserved Position: 916\n",
      "Gap Count: 598\n",
      "Mapped Position: 318\n",
      "Conserved Position: 917\n",
      "Gap Count: 598\n",
      "Mapped Position: 319\n",
      "Conserved Position: 946\n",
      "Gap Count: 598\n",
      "Mapped Position: 348\n",
      "Conserved Position: 947\n",
      "Gap Count: 598\n",
      "Mapped Position: 349\n",
      "Conserved Position: 948\n",
      "Gap Count: 598\n",
      "Mapped Position: 350\n",
      "Conserved Position: 953\n",
      "Gap Count: 601\n",
      "Mapped Position: 352\n",
      "Conserved Position: 954\n",
      "Gap Count: 601\n",
      "Mapped Position: 353\n",
      "Conserved Position: 955\n",
      "Gap Count: 601\n",
      "Mapped Position: 354\n",
      "Conserved Position: 956\n",
      "Gap Count: 601\n",
      "Mapped Position: 355\n",
      "Conserved Position: 957\n",
      "Gap Count: 601\n",
      "Mapped Position: 356\n",
      "Conserved Position: 958\n",
      "Gap Count: 601\n",
      "Mapped Position: 357\n",
      "Conserved Position: 959\n",
      "Gap Count: 601\n",
      "Mapped Position: 358\n",
      "Conserved Position: 963\n",
      "Gap Count: 603\n",
      "Mapped Position: 360\n",
      "Conserved Position: 964\n",
      "Gap Count: 603\n",
      "Mapped Position: 361\n",
      "Conserved Position: 965\n",
      "Gap Count: 603\n",
      "Mapped Position: 362\n",
      "Conserved Position: 973\n",
      "Gap Count: 609\n",
      "Mapped Position: 364\n",
      "Conserved Position: 974\n",
      "Gap Count: 609\n",
      "Mapped Position: 365\n",
      "Conserved Position: 1008\n",
      "Gap Count: 611\n",
      "Mapped Position: 397\n",
      "Conserved Position: 1010\n",
      "Gap Count: 611\n",
      "Mapped Position: 399\n",
      "Conserved Position: 1011\n",
      "Gap Count: 611\n",
      "Mapped Position: 400\n",
      "Conserved Position: 1012\n",
      "Gap Count: 611\n",
      "Mapped Position: 401\n",
      "Conserved Position: 1019\n",
      "Gap Count: 615\n",
      "Mapped Position: 404\n",
      "Conserved Position: 1022\n",
      "Gap Count: 615\n",
      "Mapped Position: 407\n",
      "Conserved Position: 1029\n",
      "Gap Count: 618\n",
      "Mapped Position: 411\n"
     ]
    }
   ],
   "source": [
    "a = map_conserved_sites(\"fdh/data/fdh_seq.fasta\", x, \"fdh/results/fdh_MSA_cluster0.fasta\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create DataFrame\n",
    "df_conserved = pd.DataFrame(a, columns=['Residue', 'Position', 'conservation'])\n",
    "\n",
    "    # Write DataFrame to a text file\n",
    "#df_conserved.to_csv(args.output, sep='\\t', index=False)  # Change separator and file format as needed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    Residue  Position        conservation\n",
      "0         K         3  0.9132636910414689\n",
      "1         V         4  0.5701171256726812\n",
      "2         L         5  0.6771130104463438\n",
      "3         V         7  0.8376068376068376\n",
      "4         L         8  0.8217790440012662\n",
      "..      ...       ...                 ...\n",
      "243       I       400  0.9810066476733144\n",
      "244       V       401  0.9167458056346945\n",
      "245       G       404    0.98005698005698\n",
      "246       A       407  0.8315922760367205\n",
      "247       Y       411  0.9860715416270972\n",
      "\n",
      "[248 rows x 3 columns]\n"
     ]
    }
   ],
   "source": [
    "print(df_conserved)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Adding OG seq to each cluster file\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "import argparse\n",
    "from Bio import SeqIO\n",
    "from Bio.Align.Applications import ClustalOmegaCommandline\n",
    "\n",
    "def add_header(input_file, output_file, header_file):\n",
    "    \"\"\"\n",
    "    Function to add a header sequence to a fasta file if it's missing.\n",
    "    \n",
    "    Parameters:\n",
    "        input_file (str): Path to the input fasta file.\n",
    "        output_file (str): Path to the output fasta file.\n",
    "        header_file (str): Path to the fasta file containing the header sequence.\n",
    "    \"\"\"\n",
    "    # Read the header sequence from the header file\n",
    "    header_record = SeqIO.read(header_file, \"fasta\")\n",
    "\n",
    "    # Check if the header exists in the input file\n",
    "    header_exists = False\n",
    "    for record in SeqIO.parse(input_file, \"fasta\"):\n",
    "        if record.description == header_record.description:\n",
    "            header_exists = True\n",
    "            # Check if the sequence matches as well\n",
    "            if str(record.seq) == str(header_record.seq):\n",
    "                print(\"Header and sequence already present in the input file.\")\n",
    "            else:\n",
    "                print(\"Header found in the input file, but with a different sequence.\")\n",
    "            break\n",
    "    \n",
    "    # If header doesn't exist, add it to the beginning of the file\n",
    "    if not header_exists:\n",
    "        SeqIO.write(header_record, output_file, \"fasta\")\n",
    "        with open(input_file, \"r\") as f:\n",
    "            with open(output_file, \"a\") as f_out:\n",
    "                for line in f:\n",
    "                    f_out.write(line)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "input_file = \"data/dummy_data.fasta\"\n",
    "output_file = \"data/seq_added.fasta\"\n",
    "header_file = \"data/dummy_seq.fasta\"\n",
    "\n",
    "add_header(input_file, output_file, header_file)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.18"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
