{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Opening test json files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('companies.json', 'r') as f:\n",
    "    test = json.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('FDH_search_result.json', 'r') as f:\n",
    "    data = json.load(f) # Real interpro output\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Visualizing json files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'signature': {'accession': 'G3DSA:3.40.50.720',\n",
       "   'name': '',\n",
       "   'description': None,\n",
       "   'signatureLibraryRelease': {'library': 'GENE3D', 'version': '4.3.0'},\n",
       "   'entry': None},\n",
       "  'locations': [{'start': 2,\n",
       "    'end': 382,\n",
       "    'representative': False,\n",
       "    'hmmStart': 1,\n",
       "    'hmmEnd': 381,\n",
       "    'hmmLength': 210,\n",
       "    'hmmBounds': 'N_TERMINAL_COMPLETE',\n",
       "    'evalue': 0.0,\n",
       "    'score': 750.9,\n",
       "    'envelopeStart': 2,\n",
       "    'envelopeEnd': 384,\n",
       "    'postProcessed': True,\n",
       "    'location-fragments': [{'start': 2,\n",
       "      'end': 146,\n",
       "      'dc-status': 'C_TERMINAL_DISC'},\n",
       "     {'start': 339, 'end': 382, 'dc-status': 'N_TERMINAL_DISC'}]}],\n",
       "  'evalue': 0.0,\n",
       "  'score': 751.1,\n",
       "  'model-ac': '2gsdA01'},\n",
       " {'signature': {'accession': 'PS00670',\n",
       "   'name': 'D_2_HYDROXYACID_DH_2',\n",
       "   'description': 'D-isomer specific 2-hydroxyacid dehydrogenases signature 2.',\n",
       "   'signatureLibraryRelease': {'library': 'PROSITE_PATTERNS',\n",
       "    'version': '2023_05'},\n",
       "   'entry': {'accession': 'IPR029753',\n",
       "    'name': 'D-isomer_DH_CS',\n",
       "    'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain conserved site',\n",
       "    'type': 'CONSERVED_SITE',\n",
       "    'goXRefs': [],\n",
       "    'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8238'},\n",
       "     {'name': 'zwittermicin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7694'},\n",
       "     {'name': '(-)-dehydrodiconiferyl alcohol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7557'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7331'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, cytosol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7383'},\n",
       "     {'name': 'stearate biosynthesis I (animals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5972'},\n",
       "     {'name': 'autoinducer CAI-1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7621'},\n",
       "     {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6973'},\n",
       "     {'name': 'septacidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8366'},\n",
       "     {'name': 'bisphenol A degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7757'},\n",
       "     {'name': 'pyruvate fermentation to (R)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8274'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-389661'},\n",
       "     {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7610'},\n",
       "     {'name': 'photorespiration III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8363'},\n",
       "     {'name': 'sphingofungin B-D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8464'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3899300'},\n",
       "     {'name': 'butanol and isobutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7396'},\n",
       "     {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5652'},\n",
       "     {'name': 'pyruvate fermentation to butanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6583'},\n",
       "     {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7340'},\n",
       "     {'name': 'D-galactose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6693'},\n",
       "     {'name': 'sitosterol degradation to androstenedione',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6948'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-977347'},\n",
       "     {'name': '1,2-propanediol biosynthesis from lactate (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7541'},\n",
       "     {'name': '(S)-camphor degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6989'},\n",
       "     {'name': 'meso-butanediol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6392'},\n",
       "     {'name': 'daunorubicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7352'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'},\n",
       "     {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8010'},\n",
       "     {'name': 'L-isoleucine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5101'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8410'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-389661'},\n",
       "     {'name': 'kanosamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5978'},\n",
       "     {'name': 'pyrethrin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6926'},\n",
       "     {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5882'},\n",
       "     {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7301'},\n",
       "     {'name': 'reductive TCA cycle II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5392'},\n",
       "     {'name': 'phosphinothricin tripeptide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6322'},\n",
       "     {'name': 'sulfolactate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6637'},\n",
       "     {'name': 'L-tyrosine degradation V (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8016'},\n",
       "     {'name': 'dTDP-beta-L-digitoxose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7657'},\n",
       "     {'name': 'ephedrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5883'},\n",
       "     {'name': 'oryzalexin A, B, and C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7489'},\n",
       "     {'name': 'L-galactose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7568'},\n",
       "     {'name': 'Signaling by TCF7L2 mutants',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5339700'},\n",
       "     {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7606'},\n",
       "     {'name': 'hygromycin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8367'},\n",
       "     {'name': 'L-ascorbate degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6704'},\n",
       "     {'name': 'dTDP-beta-L-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6976'},\n",
       "     {'name': 'superpathway of nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7342'},\n",
       "     {'name': 'terrequinone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7520'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3769402'},\n",
       "     {'name': 'CDP-6-deoxy-D-gulose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8139'},\n",
       "     {'name': 'L-homomethionine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1186'},\n",
       "     {'name': 'L-canavanine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8419'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7338'},\n",
       "     {'name': 'methylaspartate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6728'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3899300'},\n",
       "     {'name': 'tropane alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5317'},\n",
       "     {'name': 'TCA cycle V (2-oxoglutarate synthase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6969'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'CDP-ascarylose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5830'},\n",
       "     {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7615'},\n",
       "     {'name': '3-sulfopropanediol degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6634'},\n",
       "     {'name': 'nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5316'},\n",
       "     {'name': 'vancomycin resistance I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6454'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-4641265'},\n",
       "     {'name': '3-sulfopropanediol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8214'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3769402'},\n",
       "     {'name': 'tetramethylpyrazine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7403'},\n",
       "     {'name': 'pinitol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6738'},\n",
       "     {'name': 'guadinomine B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7693'},\n",
       "     {'name': '6-diazo-5-oxo-L-norleucine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8297'},\n",
       "     {'name': 'detoxification of reactive carbonyls in chloroplasts',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6786'},\n",
       "     {'name': 'isethionate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2601'},\n",
       "     {'name': '1,3-propanediol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7385'},\n",
       "     {'name': 'L-tryptophan degradation V (side chain pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3162'},\n",
       "     {'name': 'arabidopyrone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7682'},\n",
       "     {'name': 'dTDP-beta-L-olivose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6974'},\n",
       "     {'name': 'sporopollenin precursors biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6733'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3899300'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3899300'},\n",
       "     {'name': 'propane degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7774'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7337'},\n",
       "     {'name': 'androstenedione degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8152'},\n",
       "     {'name': 'mupirocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8012'},\n",
       "     {'name': 'antimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8374'},\n",
       "     {'name': 'erinacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8408'},\n",
       "     {'name': 'jasmonic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-735'},\n",
       "     {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7300'},\n",
       "     {'name': 'pyruvate fermentation to hexanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6863'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7115'},\n",
       "     {'name': 'L-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7787'},\n",
       "     {'name': 'suberin monomers biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1121'},\n",
       "     {'name': 'cis-geranyl-CoA degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6672'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8411'},\n",
       "     {'name': 'aurofusarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7695'},\n",
       "     {'name': 'bile acid biosynthesis, neutral pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6061'},\n",
       "     {'name': 'phenylethanol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5751'},\n",
       "     {'name': 'mithramycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7045'},\n",
       "     {'name': 'arachidonate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8397'},\n",
       "     {'name': 'aclacinomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7354'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-4641265'},\n",
       "     {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6946'},\n",
       "     {'name': 'heme d1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7554'},\n",
       "     {'name': 'methyl tert-butyl ether degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7779'},\n",
       "     {'name': 'fusicoccin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6659'},\n",
       "     {'name': '(Z)-phenylmethanethial S-oxide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6539'},\n",
       "     {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6318'},\n",
       "     {'name': 'myxochelin A and B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8383'},\n",
       "     {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8132'},\n",
       "     {'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-977347'},\n",
       "     {'name': '11-cis-3-hydroxyretinal biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7043'},\n",
       "     {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8014'},\n",
       "     {'name': 'TCA cycle II (plants and fungi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5690'},\n",
       "     {'name': 'leporin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8370'},\n",
       "     {'name': 'superpathway of glucose and xylose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6901'},\n",
       "     {'name': 'photorespiration II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8362'},\n",
       "     {'name': 'abscisic acid degradation to phaseic acid',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5271'},\n",
       "     {'name': 'L-glucose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7130'},\n",
       "     {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7304'},\n",
       "     {'name': 'bikaverin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7692'},\n",
       "     {'name': '6-methylpretetramide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7811'},\n",
       "     {'name': 'phytosterol biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2541'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3769402'},\n",
       "     {'name': 'photorespiration I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-181'},\n",
       "     {'name': '6-gingerol analog biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6920'},\n",
       "     {'name': 'dihydroconiferyl alcohol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6302'},\n",
       "     {'name': 'phosalacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7769'},\n",
       "     {'name': 'phenolphthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7742'},\n",
       "     {'name': 'mycolate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWYG-321'},\n",
       "     {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7613'},\n",
       "     {'name': 'L-alanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8188'},\n",
       "     {'name': 'propane degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7775'},\n",
       "     {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8086'},\n",
       "     {'name': 'L-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7295'},\n",
       "     {'name': 'apicidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7667'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-977347'},\n",
       "     {'name': '8-O-methylfusarubin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7689'},\n",
       "     {'name': '4-ethylphenol degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6080'},\n",
       "     {'name': 'Sensory processing of sound by inner hair cells of the cochlea',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9662360'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3769402'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-4641265'},\n",
       "     {'name': 't-anethole biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5867'},\n",
       "     {'name': 'the visual cycle (insects)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7042'},\n",
       "     {'name': '2-deoxy-D-ribose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8058'},\n",
       "     {'name': 'D-xylose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7294'},\n",
       "     {'name': 'quinoxaline-2-carboxylate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7734'},\n",
       "     {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3722'},\n",
       "     {'name': 'sulfolactate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6638'},\n",
       "     {'name': '3-hydroxy-L-homotyrosine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7649'},\n",
       "     {'name': 'mannojirimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7531'},\n",
       "     {'name': 'cinchona alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5848'},\n",
       "     {'name': 'L-glutamate degradation XI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8190'},\n",
       "     {'name': 'dTDP-L-daunosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7814'},\n",
       "     {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5754'},\n",
       "     {'name': 'p-cymene degradation to p-cumate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-741'},\n",
       "     {'name': 'limonene degradation IV (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8029'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-977347'},\n",
       "     {'name': 'salinosporamide A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6627'},\n",
       "     {'name': 'erythritol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7788'},\n",
       "     {'name': 'dTDP-beta-L-megosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7104'},\n",
       "     {'name': '2-keto-L-gulonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5782'},\n",
       "     {'name': 'UDP-beta-L-rhamnose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3261'},\n",
       "     {'name': 'androstenedione degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6944'},\n",
       "     {'name': 'validamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5818'},\n",
       "     {'name': 'phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7741'},\n",
       "     {'name': 'superpathway of glyoxylate cycle and fatty acid degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-561'},\n",
       "     {'name': 'sulfolactate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6616'},\n",
       "     {'name': 'myo-inositol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7241'},\n",
       "     {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6945'},\n",
       "     {'name': 'testosterone degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8155'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7726'},\n",
       "     {'name': 'alpha-solanine/alpha-chaconine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5666'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-4641265'},\n",
       "     {'name': 'dehydrophos biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6682'},\n",
       "     {'name': 'pyruvate fermentation to butanol II (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6883'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-4641265'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7333'},\n",
       "     {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6053'},\n",
       "     {'name': 'juglone degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7825'},\n",
       "     {'name': '4-oxopentanoate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7948'},\n",
       "     {'name': '(2R)-methoxymalonyl-[acp] biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8426'},\n",
       "     {'name': 'patulin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7490'},\n",
       "     {'name': 'cyathin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8409'},\n",
       "     {'name': 'cutin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-321'},\n",
       "     {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7288'},\n",
       "     {'name': 'ecdysteroid metabolism (arthropods)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7321'},\n",
       "     {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7440'},\n",
       "     {'name': 'meso-butanediol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6391'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7339'},\n",
       "     {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6626'},\n",
       "     {'name': 'prodigiosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7547'},\n",
       "     {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8017'},\n",
       "     {'name': 'tryptoquialanine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7660'},\n",
       "     {'name': 'formaldehyde assimilation I (serine pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1622'},\n",
       "     {'name': 'L-serine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8011'}]}},\n",
       "  'locations': [{'start': 245,\n",
       "    'end': 267,\n",
       "    'representative': False,\n",
       "    'level': 'STRONG',\n",
       "    'cigarAlignment': '2M2I4M1I2M1I1M4I1M2I1M1I1M',\n",
       "    'alignment': 'MYpvCDVVtLNiPlhspTehMiN',\n",
       "    'location-fragments': [{'start': 245,\n",
       "      'end': 267,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'model-ac': 'PS00670'},\n",
       " {'signature': {'accession': 'cd05302',\n",
       "   'name': 'FDH',\n",
       "   'description': 'FDH',\n",
       "   'signatureLibraryRelease': {'library': 'CDD', 'version': '3.20'},\n",
       "   'entry': {'accession': 'IPR033689',\n",
       "    'name': 'FDH_NAD-dep',\n",
       "    'description': 'NAD-dependent formate dehydrogenase',\n",
       "    'type': 'FAMILY',\n",
       "    'goXRefs': [{'name': 'formate dehydrogenase (NAD+) activity',\n",
       "      'databaseName': 'GO',\n",
       "      'category': 'MOLECULAR_FUNCTION',\n",
       "      'id': 'GO:0008863'}],\n",
       "    'pathwayXRefs': [{'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'}]}},\n",
       "  'locations': [{'start': 2,\n",
       "    'end': 378,\n",
       "    'representative': True,\n",
       "    'sites': [{'description': 'NAD binding site',\n",
       "      'numLocations': 25,\n",
       "      'siteLocations': [{'start': 283, 'end': 283, 'residue': 'T'},\n",
       "       {'start': 310, 'end': 310, 'residue': 'V'},\n",
       "       {'start': 262, 'end': 262, 'residue': 'T'},\n",
       "       {'start': 148, 'end': 148, 'residue': 'S'},\n",
       "       {'start': 151, 'end': 151, 'residue': 'V'},\n",
       "       {'start': 336, 'end': 336, 'residue': 'G'},\n",
       "       {'start': 203, 'end': 203, 'residue': 'I'},\n",
       "       {'start': 333, 'end': 333, 'residue': 'H'},\n",
       "       {'start': 223, 'end': 223, 'residue': 'R'},\n",
       "       {'start': 256, 'end': 256, 'residue': 'I'},\n",
       "       {'start': 202, 'end': 202, 'residue': 'R'},\n",
       "       {'start': 259, 'end': 259, 'residue': 'H'},\n",
       "       {'start': 255, 'end': 255, 'residue': 'N'},\n",
       "       {'start': 201, 'end': 201, 'residue': 'G'},\n",
       "       {'start': 222, 'end': 222, 'residue': 'D'},\n",
       "       {'start': 123, 'end': 123, 'residue': 'I'},\n",
       "       {'start': 99, 'end': 99, 'residue': 'F'},\n",
       "       {'start': 147, 'end': 147, 'residue': 'N'},\n",
       "       {'start': 224, 'end': 224, 'residue': 'H'},\n",
       "       {'start': 335, 'end': 335, 'residue': 'S'},\n",
       "       {'start': 257, 'end': 257, 'residue': 'P'},\n",
       "       {'start': 199, 'end': 199, 'residue': 'A'},\n",
       "       {'start': 284, 'end': 284, 'residue': 'A'},\n",
       "       {'start': 309, 'end': 309, 'residue': 'D'},\n",
       "       {'start': 285, 'end': 285, 'residue': 'R'}]},\n",
       "     {'description': 'dimerization interface',\n",
       "      'numLocations': 62,\n",
       "      'siteLocations': [{'start': 337, 'end': 337, 'residue': 'T'},\n",
       "       {'start': 193, 'end': 193, 'residue': 'M'},\n",
       "       {'start': 176, 'end': 176, 'residue': 'G'},\n",
       "       {'start': 181, 'end': 181, 'residue': 'A'},\n",
       "       {'start': 171, 'end': 171, 'residue': 'E'},\n",
       "       {'start': 50, 'end': 50, 'residue': 'L'},\n",
       "       {'start': 153, 'end': 153, 'residue': 'E'},\n",
       "       {'start': 167, 'end': 167, 'residue': 'L'},\n",
       "       {'start': 150, 'end': 150, 'residue': 'S'},\n",
       "       {'start': 214, 'end': 214, 'residue': 'F'},\n",
       "       {'start': 323, 'end': 323, 'residue': 'R'},\n",
       "       {'start': 161, 'end': 161, 'residue': 'S'},\n",
       "       {'start': 318, 'end': 318, 'residue': 'K'},\n",
       "       {'start': 213, 'end': 213, 'residue': 'P'},\n",
       "       {'start': 331, 'end': 331, 'residue': 'T'},\n",
       "       {'start': 209, 'end': 209, 'residue': 'R'},\n",
       "       {'start': 190, 'end': 190, 'residue': 'L'},\n",
       "       {'start': 9, 'end': 9, 'residue': 'Y'},\n",
       "       {'start': 330, 'end': 330, 'residue': 'M'},\n",
       "       {'start': 11, 'end': 11, 'residue': 'D'},\n",
       "       {'start': 179, 'end': 179, 'residue': 'N'},\n",
       "       {'start': 342, 'end': 342, 'residue': 'Q'},\n",
       "       {'start': 191, 'end': 191, 'residue': 'E'},\n",
       "       {'start': 178, 'end': 178, 'residue': 'W'},\n",
       "       {'start': 324, 'end': 324, 'residue': 'T'},\n",
       "       {'start': 316, 'end': 316, 'residue': 'A'},\n",
       "       {'start': 332, 'end': 332, 'residue': 'P'},\n",
       "       {'start': 177, 'end': 177, 'residue': 'G'},\n",
       "       {'start': 154, 'end': 154, 'residue': 'H'},\n",
       "       {'start': 169, 'end': 169, 'residue': 'S'},\n",
       "       {'start': 315, 'end': 315, 'residue': 'P'},\n",
       "       {'start': 333, 'end': 333, 'residue': 'H'},\n",
       "       {'start': 188, 'end': 188, 'residue': 'Y'},\n",
       "       {'start': 344, 'end': 344, 'residue': 'R'},\n",
       "       {'start': 172, 'end': 172, 'residue': 'W'},\n",
       "       {'start': 329, 'end': 329, 'residue': 'G'},\n",
       "       {'start': 311, 'end': 311, 'residue': 'W'},\n",
       "       {'start': 174, 'end': 174, 'residue': 'R'},\n",
       "       {'start': 340, 'end': 340, 'residue': 'T'},\n",
       "       {'start': 338, 'end': 338, 'residue': 'T'},\n",
       "       {'start': 166, 'end': 166, 'residue': 'Y'},\n",
       "       {'start': 314, 'end': 314, 'residue': 'Q'},\n",
       "       {'start': 184, 'end': 184, 'residue': 'V'},\n",
       "       {'start': 173, 'end': 173, 'residue': 'A'},\n",
       "       {'start': 334, 'end': 334, 'residue': 'I'},\n",
       "       {'start': 187, 'end': 187, 'residue': 'A'},\n",
       "       {'start': 157, 'end': 157, 'residue': 'M'},\n",
       "       {'start': 185, 'end': 185, 'residue': 'S'},\n",
       "       {'start': 170, 'end': 170, 'residue': 'H'},\n",
       "       {'start': 160, 'end': 160, 'residue': 'L'},\n",
       "       {'start': 156, 'end': 156, 'residue': 'V'},\n",
       "       {'start': 164, 'end': 164, 'residue': 'R'},\n",
       "       {'start': 339, 'end': 339, 'residue': 'L'},\n",
       "       {'start': 189, 'end': 189, 'residue': 'D'},\n",
       "       {'start': 182, 'end': 182, 'residue': 'D'},\n",
       "       {'start': 180, 'end': 180, 'residue': 'I'},\n",
       "       {'start': 149, 'end': 149, 'residue': 'I'},\n",
       "       {'start': 328, 'end': 328, 'residue': 'N'},\n",
       "       {'start': 10, 'end': 10, 'residue': 'D'},\n",
       "       {'start': 335, 'end': 335, 'residue': 'S'},\n",
       "       {'start': 183, 'end': 183, 'residue': 'C'},\n",
       "       {'start': 210, 'end': 210, 'residue': 'R'}]}],\n",
       "    'evalue': 0.0,\n",
       "    'score': 692.915,\n",
       "    'location-fragments': [{'start': 2,\n",
       "      'end': 378,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'model-ac': 'cd05302'},\n",
       " {'signature': {'accession': 'PTHR42938',\n",
       "   'name': 'FORMATE DEHYDROGENASE 1',\n",
       "   'description': None,\n",
       "   'signatureLibraryRelease': {'library': 'PANTHER', 'version': '18.0'},\n",
       "   'entry': None},\n",
       "  'locations': [{'start': 72,\n",
       "    'end': 360,\n",
       "    'representative': False,\n",
       "    'hmmStart': 30,\n",
       "    'hmmEnd': 311,\n",
       "    'hmmLength': 0,\n",
       "    'hmmBounds': 'INCOMPLETE',\n",
       "    'envelopeStart': 51,\n",
       "    'envelopeEnd': 365,\n",
       "    'location-fragments': [{'start': 72,\n",
       "      'end': 360,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'accession': 'PTHR42938:SF9',\n",
       "  'name': 'FORMATE DEHYDROGENASE 1',\n",
       "  'evalue': 4.1e-63,\n",
       "  'score': 225.2,\n",
       "  'proteinClass': 'PC00092',\n",
       "  'graftPoint': 'PTN002873291',\n",
       "  'goXRefs': [{'name': 'formate dehydrogenase (NAD+) activity',\n",
       "    'databaseName': 'GO',\n",
       "    'category': 'MOLECULAR_FUNCTION',\n",
       "    'id': 'GO:0008863'}],\n",
       "  'model-ac': 'PTHR42938:SF9'},\n",
       " {'signature': {'accession': 'G3DSA:3.40.50.720',\n",
       "   'name': '',\n",
       "   'description': None,\n",
       "   'signatureLibraryRelease': {'library': 'GENE3D', 'version': '4.3.0'},\n",
       "   'entry': None},\n",
       "  'locations': [{'start': 147,\n",
       "    'end': 338,\n",
       "    'representative': False,\n",
       "    'hmmStart': 1,\n",
       "    'hmmEnd': 381,\n",
       "    'hmmLength': 192,\n",
       "    'hmmBounds': 'INCOMPLETE',\n",
       "    'evalue': 0.0,\n",
       "    'score': 750.9,\n",
       "    'envelopeStart': 2,\n",
       "    'envelopeEnd': 384,\n",
       "    'postProcessed': True,\n",
       "    'location-fragments': [{'start': 147,\n",
       "      'end': 338,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'evalue': 0.0,\n",
       "  'score': 751.1,\n",
       "  'model-ac': '2gsdA02'},\n",
       " {'signature': {'accession': 'PF00389',\n",
       "   'name': '2-Hacid_dh',\n",
       "   'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, catalytic domain',\n",
       "   'signatureLibraryRelease': {'library': 'PFAM', 'version': '36.0'},\n",
       "   'entry': {'accession': 'IPR006139',\n",
       "    'name': 'D-isomer_2_OHA_DH_cat_dom',\n",
       "    'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, catalytic domain',\n",
       "    'type': 'DOMAIN',\n",
       "    'goXRefs': [{'name': 'oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor',\n",
       "      'databaseName': 'GO',\n",
       "      'category': 'MOLECULAR_FUNCTION',\n",
       "      'id': 'GO:0016616'},\n",
       "     {'name': 'NAD binding',\n",
       "      'databaseName': 'GO',\n",
       "      'category': 'MOLECULAR_FUNCTION',\n",
       "      'id': 'GO:0051287'}],\n",
       "    'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8238'},\n",
       "     {'name': 'zwittermicin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7694'},\n",
       "     {'name': '(-)-dehydrodiconiferyl alcohol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7557'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7331'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, cytosol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7383'},\n",
       "     {'name': 'stearate biosynthesis I (animals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5972'},\n",
       "     {'name': 'autoinducer CAI-1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7621'},\n",
       "     {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6973'},\n",
       "     {'name': 'septacidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8366'},\n",
       "     {'name': 'bisphenol A degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7757'},\n",
       "     {'name': 'pyruvate fermentation to (R)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8274'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-389661'},\n",
       "     {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7610'},\n",
       "     {'name': 'photorespiration III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8363'},\n",
       "     {'name': 'sphingofungin B-D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8464'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3899300'},\n",
       "     {'name': 'butanol and isobutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7396'},\n",
       "     {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5652'},\n",
       "     {'name': 'pyruvate fermentation to butanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6583'},\n",
       "     {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7340'},\n",
       "     {'name': 'D-galactose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6693'},\n",
       "     {'name': 'sitosterol degradation to androstenedione',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6948'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-977347'},\n",
       "     {'name': '1,2-propanediol biosynthesis from lactate (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7541'},\n",
       "     {'name': '(S)-camphor degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6989'},\n",
       "     {'name': 'meso-butanediol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6392'},\n",
       "     {'name': 'daunorubicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7352'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'},\n",
       "     {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8010'},\n",
       "     {'name': 'L-isoleucine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5101'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8410'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-389661'},\n",
       "     {'name': 'kanosamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5978'},\n",
       "     {'name': 'pyrethrin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6926'},\n",
       "     {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5882'},\n",
       "     {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7301'},\n",
       "     {'name': 'reductive TCA cycle II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5392'},\n",
       "     {'name': 'phosphinothricin tripeptide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6322'},\n",
       "     {'name': 'sulfolactate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6637'},\n",
       "     {'name': 'L-tyrosine degradation V (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8016'},\n",
       "     {'name': 'dTDP-beta-L-digitoxose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7657'},\n",
       "     {'name': 'ephedrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5883'},\n",
       "     {'name': 'rosmarinic acid biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5049'},\n",
       "     {'name': 'oryzalexin A, B, and C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7489'},\n",
       "     {'name': 'L-galactose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7568'},\n",
       "     {'name': 'Signaling by TCF7L2 mutants',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5339700'},\n",
       "     {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7606'},\n",
       "     {'name': 'hygromycin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8367'},\n",
       "     {'name': 'L-ascorbate degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6704'},\n",
       "     {'name': 'dTDP-beta-L-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6976'},\n",
       "     {'name': 'superpathway of nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7342'},\n",
       "     {'name': 'terrequinone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7520'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3769402'},\n",
       "     {'name': 'CDP-6-deoxy-D-gulose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8139'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-4641265'},\n",
       "     {'name': 'L-homomethionine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1186'},\n",
       "     {'name': 'L-canavanine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8419'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7338'},\n",
       "     {'name': 'methylaspartate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6728'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3899300'},\n",
       "     {'name': 'tropane alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5317'},\n",
       "     {'name': 'TCA cycle V (2-oxoglutarate synthase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6969'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'CDP-ascarylose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5830'},\n",
       "     {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7615'},\n",
       "     {'name': '3-sulfopropanediol degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6634'},\n",
       "     {'name': 'nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5316'},\n",
       "     {'name': 'vancomycin resistance I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6454'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-4641265'},\n",
       "     {'name': '3-sulfopropanediol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8214'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3769402'},\n",
       "     {'name': 'tetramethylpyrazine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7403'},\n",
       "     {'name': 'pinitol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6738'},\n",
       "     {'name': 'guadinomine B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7693'},\n",
       "     {'name': '6-diazo-5-oxo-L-norleucine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8297'},\n",
       "     {'name': 'detoxification of reactive carbonyls in chloroplasts',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6786'},\n",
       "     {'name': 'isethionate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2601'},\n",
       "     {'name': '1,3-propanediol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7385'},\n",
       "     {'name': 'L-tryptophan degradation V (side chain pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3162'},\n",
       "     {'name': 'arabidopyrone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7682'},\n",
       "     {'name': 'dTDP-beta-L-olivose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6974'},\n",
       "     {'name': 'sporopollenin precursors biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6733'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3899300'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3899300'},\n",
       "     {'name': 'propane degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7774'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7337'},\n",
       "     {'name': 'androstenedione degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8152'},\n",
       "     {'name': 'rosmarinic acid biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5048'},\n",
       "     {'name': 'mupirocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8012'},\n",
       "     {'name': 'antimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8374'},\n",
       "     {'name': 'erinacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8408'},\n",
       "     {'name': 'jasmonic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-735'},\n",
       "     {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7300'},\n",
       "     {'name': 'pyruvate fermentation to hexanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6863'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7115'},\n",
       "     {'name': 'L-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7787'},\n",
       "     {'name': 'suberin monomers biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1121'},\n",
       "     {'name': 'cis-geranyl-CoA degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6672'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8411'},\n",
       "     {'name': 'aurofusarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7695'},\n",
       "     {'name': 'bile acid biosynthesis, neutral pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6061'},\n",
       "     {'name': 'phenylethanol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5751'},\n",
       "     {'name': 'mithramycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7045'},\n",
       "     {'name': 'arachidonate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8397'},\n",
       "     {'name': 'aclacinomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7354'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-4641265'},\n",
       "     {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6946'},\n",
       "     {'name': 'heme d1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7554'},\n",
       "     {'name': 'methyl tert-butyl ether degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7779'},\n",
       "     {'name': 'fusicoccin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6659'},\n",
       "     {'name': '(Z)-phenylmethanethial S-oxide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6539'},\n",
       "     {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6318'},\n",
       "     {'name': 'myxochelin A and B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8383'},\n",
       "     {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8132'},\n",
       "     {'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-977347'},\n",
       "     {'name': '11-cis-3-hydroxyretinal biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7043'},\n",
       "     {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8014'},\n",
       "     {'name': 'TCA cycle II (plants and fungi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5690'},\n",
       "     {'name': 'leporin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8370'},\n",
       "     {'name': 'superpathway of glucose and xylose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6901'},\n",
       "     {'name': 'photorespiration II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8362'},\n",
       "     {'name': 'abscisic acid degradation to phaseic acid',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5271'},\n",
       "     {'name': 'L-glucose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7130'},\n",
       "     {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7304'},\n",
       "     {'name': 'bikaverin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7692'},\n",
       "     {'name': '6-methylpretetramide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7811'},\n",
       "     {'name': 'phytosterol biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2541'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3769402'},\n",
       "     {'name': 'photorespiration I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-181'},\n",
       "     {'name': '6-gingerol analog biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6920'},\n",
       "     {'name': 'dihydroconiferyl alcohol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6302'},\n",
       "     {'name': 'phosalacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7769'},\n",
       "     {'name': 'phenolphthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7742'},\n",
       "     {'name': 'mycolate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWYG-321'},\n",
       "     {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7613'},\n",
       "     {'name': 'L-alanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8188'},\n",
       "     {'name': 'propane degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7775'},\n",
       "     {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8086'},\n",
       "     {'name': 'L-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7295'},\n",
       "     {'name': 'apicidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7667'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-977347'},\n",
       "     {'name': '8-O-methylfusarubin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7689'},\n",
       "     {'name': '4-ethylphenol degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6080'},\n",
       "     {'name': 'Sensory processing of sound by inner hair cells of the cochlea',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9662360'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3769402'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-4641265'},\n",
       "     {'name': 't-anethole biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5867'},\n",
       "     {'name': 'the visual cycle (insects)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7042'},\n",
       "     {'name': '2-deoxy-D-ribose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8058'},\n",
       "     {'name': 'D-xylose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7294'},\n",
       "     {'name': 'quinoxaline-2-carboxylate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7734'},\n",
       "     {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3722'},\n",
       "     {'name': 'sulfolactate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6638'},\n",
       "     {'name': '3-hydroxy-L-homotyrosine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7649'},\n",
       "     {'name': 'mannojirimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7531'},\n",
       "     {'name': 'cinchona alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5848'},\n",
       "     {'name': 'L-glutamate degradation XI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8190'},\n",
       "     {'name': 'dTDP-L-daunosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7814'},\n",
       "     {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5754'},\n",
       "     {'name': 'p-cymene degradation to p-cumate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-741'},\n",
       "     {'name': 'limonene degradation IV (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8029'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-977347'},\n",
       "     {'name': 'salinosporamide A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6627'},\n",
       "     {'name': 'erythritol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7788'},\n",
       "     {'name': 'dTDP-beta-L-megosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7104'},\n",
       "     {'name': '2-keto-L-gulonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5782'},\n",
       "     {'name': 'UDP-beta-L-rhamnose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3261'},\n",
       "     {'name': 'androstenedione degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6944'},\n",
       "     {'name': 'validamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5818'},\n",
       "     {'name': 'phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7741'},\n",
       "     {'name': 'superpathway of glyoxylate cycle and fatty acid degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-561'},\n",
       "     {'name': 'sulfolactate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6616'},\n",
       "     {'name': 'myo-inositol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7241'},\n",
       "     {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6945'},\n",
       "     {'name': 'testosterone degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8155'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7726'},\n",
       "     {'name': 'alpha-solanine/alpha-chaconine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5666'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-4641265'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-3769402'},\n",
       "     {'name': 'dehydrophos biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6682'},\n",
       "     {'name': 'pyruvate fermentation to butanol II (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6883'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-4641265'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7333'},\n",
       "     {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6053'},\n",
       "     {'name': 'juglone degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7825'},\n",
       "     {'name': '4-oxopentanoate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7948'},\n",
       "     {'name': '(2R)-methoxymalonyl-[acp] biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8426'},\n",
       "     {'name': 'patulin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7490'},\n",
       "     {'name': 'cyathin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8409'},\n",
       "     {'name': 'cutin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-321'},\n",
       "     {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7288'},\n",
       "     {'name': 'ecdysteroid metabolism (arthropods)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7321'},\n",
       "     {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7440'},\n",
       "     {'name': 'meso-butanediol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6391'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7339'},\n",
       "     {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6626'},\n",
       "     {'name': 'prodigiosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7547'},\n",
       "     {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8017'},\n",
       "     {'name': 'tryptoquialanine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7660'},\n",
       "     {'name': 'formaldehyde assimilation I (serine pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1622'},\n",
       "     {'name': 'L-serine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8011'}]}},\n",
       "  'locations': [{'start': 62,\n",
       "    'end': 363,\n",
       "    'representative': False,\n",
       "    'hmmStart': 13,\n",
       "    'hmmEnd': 130,\n",
       "    'hmmLength': 134,\n",
       "    'hmmBounds': 'INCOMPLETE',\n",
       "    'evalue': 1.4e-11,\n",
       "    'score': 56.0,\n",
       "    'envelopeStart': 55,\n",
       "    'envelopeEnd': 366,\n",
       "    'postProcessed': True,\n",
       "    'location-fragments': [{'start': 62,\n",
       "      'end': 155,\n",
       "      'dc-status': 'C_TERMINAL_DISC'},\n",
       "     {'start': 336, 'end': 363, 'dc-status': 'N_TERMINAL_DISC'}]}],\n",
       "  'evalue': 1e-11,\n",
       "  'score': 56.0,\n",
       "  'model-ac': 'PF00389'},\n",
       " {'signature': {'accession': 'PS00671',\n",
       "   'name': 'D_2_HYDROXYACID_DH_3',\n",
       "   'description': 'D-isomer specific 2-hydroxyacid dehydrogenases signature 3.',\n",
       "   'signatureLibraryRelease': {'library': 'PROSITE_PATTERNS',\n",
       "    'version': '2023_05'},\n",
       "   'entry': {'accession': 'IPR029753',\n",
       "    'name': 'D-isomer_DH_CS',\n",
       "    'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain conserved site',\n",
       "    'type': 'CONSERVED_SITE',\n",
       "    'goXRefs': [],\n",
       "    'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8238'},\n",
       "     {'name': 'zwittermicin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7694'},\n",
       "     {'name': '(-)-dehydrodiconiferyl alcohol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7557'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7331'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, cytosol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7383'},\n",
       "     {'name': 'stearate biosynthesis I (animals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5972'},\n",
       "     {'name': 'autoinducer CAI-1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7621'},\n",
       "     {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6973'},\n",
       "     {'name': 'septacidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8366'},\n",
       "     {'name': 'bisphenol A degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7757'},\n",
       "     {'name': 'pyruvate fermentation to (R)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8274'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-389661'},\n",
       "     {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7610'},\n",
       "     {'name': 'photorespiration III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8363'},\n",
       "     {'name': 'sphingofungin B-D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8464'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3899300'},\n",
       "     {'name': 'butanol and isobutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7396'},\n",
       "     {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5652'},\n",
       "     {'name': 'pyruvate fermentation to butanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6583'},\n",
       "     {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7340'},\n",
       "     {'name': 'D-galactose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6693'},\n",
       "     {'name': 'sitosterol degradation to androstenedione',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6948'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-977347'},\n",
       "     {'name': '1,2-propanediol biosynthesis from lactate (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7541'},\n",
       "     {'name': '(S)-camphor degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6989'},\n",
       "     {'name': 'meso-butanediol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6392'},\n",
       "     {'name': 'daunorubicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7352'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'},\n",
       "     {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8010'},\n",
       "     {'name': 'L-isoleucine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5101'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8410'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-389661'},\n",
       "     {'name': 'kanosamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5978'},\n",
       "     {'name': 'pyrethrin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6926'},\n",
       "     {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5882'},\n",
       "     {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7301'},\n",
       "     {'name': 'reductive TCA cycle II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5392'},\n",
       "     {'name': 'phosphinothricin tripeptide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6322'},\n",
       "     {'name': 'sulfolactate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6637'},\n",
       "     {'name': 'L-tyrosine degradation V (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8016'},\n",
       "     {'name': 'dTDP-beta-L-digitoxose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7657'},\n",
       "     {'name': 'ephedrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5883'},\n",
       "     {'name': 'oryzalexin A, B, and C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7489'},\n",
       "     {'name': 'L-galactose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7568'},\n",
       "     {'name': 'Signaling by TCF7L2 mutants',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5339700'},\n",
       "     {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7606'},\n",
       "     {'name': 'hygromycin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8367'},\n",
       "     {'name': 'L-ascorbate degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6704'},\n",
       "     {'name': 'dTDP-beta-L-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6976'},\n",
       "     {'name': 'superpathway of nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7342'},\n",
       "     {'name': 'terrequinone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7520'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3769402'},\n",
       "     {'name': 'CDP-6-deoxy-D-gulose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8139'},\n",
       "     {'name': 'L-homomethionine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1186'},\n",
       "     {'name': 'L-canavanine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8419'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7338'},\n",
       "     {'name': 'methylaspartate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6728'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3899300'},\n",
       "     {'name': 'tropane alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5317'},\n",
       "     {'name': 'TCA cycle V (2-oxoglutarate synthase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6969'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'CDP-ascarylose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5830'},\n",
       "     {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7615'},\n",
       "     {'name': '3-sulfopropanediol degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6634'},\n",
       "     {'name': 'nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5316'},\n",
       "     {'name': 'vancomycin resistance I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6454'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-4641265'},\n",
       "     {'name': '3-sulfopropanediol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8214'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3769402'},\n",
       "     {'name': 'tetramethylpyrazine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7403'},\n",
       "     {'name': 'pinitol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6738'},\n",
       "     {'name': 'guadinomine B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7693'},\n",
       "     {'name': '6-diazo-5-oxo-L-norleucine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8297'},\n",
       "     {'name': 'detoxification of reactive carbonyls in chloroplasts',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6786'},\n",
       "     {'name': 'isethionate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2601'},\n",
       "     {'name': '1,3-propanediol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7385'},\n",
       "     {'name': 'L-tryptophan degradation V (side chain pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3162'},\n",
       "     {'name': 'arabidopyrone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7682'},\n",
       "     {'name': 'dTDP-beta-L-olivose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6974'},\n",
       "     {'name': 'sporopollenin precursors biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6733'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3899300'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3899300'},\n",
       "     {'name': 'propane degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7774'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7337'},\n",
       "     {'name': 'androstenedione degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8152'},\n",
       "     {'name': 'mupirocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8012'},\n",
       "     {'name': 'antimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8374'},\n",
       "     {'name': 'erinacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8408'},\n",
       "     {'name': 'jasmonic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-735'},\n",
       "     {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7300'},\n",
       "     {'name': 'pyruvate fermentation to hexanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6863'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7115'},\n",
       "     {'name': 'L-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7787'},\n",
       "     {'name': 'suberin monomers biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1121'},\n",
       "     {'name': 'cis-geranyl-CoA degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6672'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8411'},\n",
       "     {'name': 'aurofusarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7695'},\n",
       "     {'name': 'bile acid biosynthesis, neutral pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6061'},\n",
       "     {'name': 'phenylethanol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5751'},\n",
       "     {'name': 'mithramycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7045'},\n",
       "     {'name': 'arachidonate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8397'},\n",
       "     {'name': 'aclacinomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7354'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-4641265'},\n",
       "     {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6946'},\n",
       "     {'name': 'heme d1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7554'},\n",
       "     {'name': 'methyl tert-butyl ether degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7779'},\n",
       "     {'name': 'fusicoccin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6659'},\n",
       "     {'name': '(Z)-phenylmethanethial S-oxide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6539'},\n",
       "     {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6318'},\n",
       "     {'name': 'myxochelin A and B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8383'},\n",
       "     {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8132'},\n",
       "     {'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-977347'},\n",
       "     {'name': '11-cis-3-hydroxyretinal biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7043'},\n",
       "     {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8014'},\n",
       "     {'name': 'TCA cycle II (plants and fungi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5690'},\n",
       "     {'name': 'leporin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8370'},\n",
       "     {'name': 'superpathway of glucose and xylose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6901'},\n",
       "     {'name': 'photorespiration II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8362'},\n",
       "     {'name': 'abscisic acid degradation to phaseic acid',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5271'},\n",
       "     {'name': 'L-glucose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7130'},\n",
       "     {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7304'},\n",
       "     {'name': 'bikaverin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7692'},\n",
       "     {'name': '6-methylpretetramide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7811'},\n",
       "     {'name': 'phytosterol biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2541'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3769402'},\n",
       "     {'name': 'photorespiration I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-181'},\n",
       "     {'name': '6-gingerol analog biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6920'},\n",
       "     {'name': 'dihydroconiferyl alcohol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6302'},\n",
       "     {'name': 'phosalacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7769'},\n",
       "     {'name': 'phenolphthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7742'},\n",
       "     {'name': 'mycolate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWYG-321'},\n",
       "     {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7613'},\n",
       "     {'name': 'L-alanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8188'},\n",
       "     {'name': 'propane degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7775'},\n",
       "     {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8086'},\n",
       "     {'name': 'L-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7295'},\n",
       "     {'name': 'apicidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7667'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-977347'},\n",
       "     {'name': '8-O-methylfusarubin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7689'},\n",
       "     {'name': '4-ethylphenol degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6080'},\n",
       "     {'name': 'Sensory processing of sound by inner hair cells of the cochlea',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9662360'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3769402'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-4641265'},\n",
       "     {'name': 't-anethole biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5867'},\n",
       "     {'name': 'the visual cycle (insects)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7042'},\n",
       "     {'name': '2-deoxy-D-ribose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8058'},\n",
       "     {'name': 'D-xylose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7294'},\n",
       "     {'name': 'quinoxaline-2-carboxylate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7734'},\n",
       "     {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3722'},\n",
       "     {'name': 'sulfolactate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6638'},\n",
       "     {'name': '3-hydroxy-L-homotyrosine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7649'},\n",
       "     {'name': 'mannojirimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7531'},\n",
       "     {'name': 'cinchona alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5848'},\n",
       "     {'name': 'L-glutamate degradation XI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8190'},\n",
       "     {'name': 'dTDP-L-daunosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7814'},\n",
       "     {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5754'},\n",
       "     {'name': 'p-cymene degradation to p-cumate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-741'},\n",
       "     {'name': 'limonene degradation IV (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8029'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-977347'},\n",
       "     {'name': 'salinosporamide A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6627'},\n",
       "     {'name': 'erythritol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7788'},\n",
       "     {'name': 'dTDP-beta-L-megosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7104'},\n",
       "     {'name': '2-keto-L-gulonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5782'},\n",
       "     {'name': 'UDP-beta-L-rhamnose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3261'},\n",
       "     {'name': 'androstenedione degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6944'},\n",
       "     {'name': 'validamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5818'},\n",
       "     {'name': 'phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7741'},\n",
       "     {'name': 'superpathway of glyoxylate cycle and fatty acid degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-561'},\n",
       "     {'name': 'sulfolactate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6616'},\n",
       "     {'name': 'myo-inositol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7241'},\n",
       "     {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6945'},\n",
       "     {'name': 'testosterone degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8155'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7726'},\n",
       "     {'name': 'alpha-solanine/alpha-chaconine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5666'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-4641265'},\n",
       "     {'name': 'dehydrophos biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6682'},\n",
       "     {'name': 'pyruvate fermentation to butanol II (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6883'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-4641265'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7333'},\n",
       "     {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6053'},\n",
       "     {'name': 'juglone degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7825'},\n",
       "     {'name': '4-oxopentanoate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7948'},\n",
       "     {'name': '(2R)-methoxymalonyl-[acp] biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8426'},\n",
       "     {'name': 'patulin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7490'},\n",
       "     {'name': 'cyathin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8409'},\n",
       "     {'name': 'cutin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-321'},\n",
       "     {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7288'},\n",
       "     {'name': 'ecdysteroid metabolism (arthropods)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7321'},\n",
       "     {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7440'},\n",
       "     {'name': 'meso-butanediol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6391'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7339'},\n",
       "     {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6626'},\n",
       "     {'name': 'prodigiosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7547'},\n",
       "     {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8017'},\n",
       "     {'name': 'tryptoquialanine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7660'},\n",
       "     {'name': 'formaldehyde assimilation I (serine pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1622'},\n",
       "     {'name': 'L-serine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8011'}]}},\n",
       "  'locations': [{'start': 274,\n",
       "    'end': 290,\n",
       "    'representative': False,\n",
       "    'level': 'STRONG',\n",
       "    'cigarAlignment': '2M1I1M1I4M1I3M1I3M',\n",
       "    'alignment': 'FKrGaYIVNtARGkLCD',\n",
       "    'location-fragments': [{'start': 274,\n",
       "      'end': 290,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'model-ac': 'PS00671'},\n",
       " {'signature': {'accession': 'PF02826',\n",
       "   'name': '2-Hacid_dh_C',\n",
       "   'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD binding domain',\n",
       "   'signatureLibraryRelease': {'library': 'PFAM', 'version': '36.0'},\n",
       "   'entry': {'accession': 'IPR006140',\n",
       "    'name': 'D-isomer_DH_NAD-bd',\n",
       "    'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain',\n",
       "    'type': 'DOMAIN',\n",
       "    'goXRefs': [{'name': 'NAD binding',\n",
       "      'databaseName': 'GO',\n",
       "      'category': 'MOLECULAR_FUNCTION',\n",
       "      'id': 'GO:0051287'}],\n",
       "    'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8238'},\n",
       "     {'name': 'zwittermicin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7694'},\n",
       "     {'name': '(-)-dehydrodiconiferyl alcohol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7557'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7331'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, cytosol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7383'},\n",
       "     {'name': 'stearate biosynthesis I (animals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5972'},\n",
       "     {'name': 'autoinducer CAI-1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7621'},\n",
       "     {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6973'},\n",
       "     {'name': 'septacidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8366'},\n",
       "     {'name': 'bisphenol A degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7757'},\n",
       "     {'name': 'pyruvate fermentation to (R)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8274'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-389661'},\n",
       "     {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7610'},\n",
       "     {'name': 'photorespiration III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8363'},\n",
       "     {'name': 'sphingofungin B-D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8464'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3899300'},\n",
       "     {'name': 'butanol and isobutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7396'},\n",
       "     {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5652'},\n",
       "     {'name': 'pyruvate fermentation to butanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6583'},\n",
       "     {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7340'},\n",
       "     {'name': 'D-galactose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6693'},\n",
       "     {'name': 'sitosterol degradation to androstenedione',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6948'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-977347'},\n",
       "     {'name': '1,2-propanediol biosynthesis from lactate (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7541'},\n",
       "     {'name': '(S)-camphor degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6989'},\n",
       "     {'name': 'meso-butanediol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6392'},\n",
       "     {'name': 'daunorubicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7352'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'},\n",
       "     {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8010'},\n",
       "     {'name': 'L-isoleucine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5101'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8410'},\n",
       "     {'name': \"inosine-5'-phosphate biosynthesis III\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7234'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-389661'},\n",
       "     {'name': 'kanosamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5978'},\n",
       "     {'name': 'pyrethrin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6926'},\n",
       "     {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5882'},\n",
       "     {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7301'},\n",
       "     {'name': 'reductive TCA cycle II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5392'},\n",
       "     {'name': 'phosphinothricin tripeptide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6322'},\n",
       "     {'name': 'sulfolactate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6637'},\n",
       "     {'name': 'L-tyrosine degradation V (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8016'},\n",
       "     {'name': 'dTDP-beta-L-digitoxose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7657'},\n",
       "     {'name': 'ephedrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5883'},\n",
       "     {'name': 'rosmarinic acid biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5049'},\n",
       "     {'name': 'oryzalexin A, B, and C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7489'},\n",
       "     {'name': 'L-galactose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7568'},\n",
       "     {'name': 'Signaling by TCF7L2 mutants',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5339700'},\n",
       "     {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7606'},\n",
       "     {'name': 'hygromycin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8367'},\n",
       "     {'name': 'L-ascorbate degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6704'},\n",
       "     {'name': 'dTDP-beta-L-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6976'},\n",
       "     {'name': 'superpathway of nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7342'},\n",
       "     {'name': 'terrequinone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7520'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3769402'},\n",
       "     {'name': 'CDP-6-deoxy-D-gulose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8139'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-4641265'},\n",
       "     {'name': 'L-homomethionine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1186'},\n",
       "     {'name': 'superpathway of 5-aminoimidazole ribonucleotide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6277'},\n",
       "     {'name': 'L-canavanine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8419'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7338'},\n",
       "     {'name': 'methylaspartate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6728'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3899300'},\n",
       "     {'name': 'tropane alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5317'},\n",
       "     {'name': 'TCA cycle V (2-oxoglutarate synthase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6969'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'CDP-ascarylose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5830'},\n",
       "     {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7615'},\n",
       "     {'name': '3-sulfopropanediol degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6634'},\n",
       "     {'name': 'nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5316'},\n",
       "     {'name': 'vancomycin resistance I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6454'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-4641265'},\n",
       "     {'name': '3-sulfopropanediol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8214'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3769402'},\n",
       "     {'name': 'tetramethylpyrazine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7403'},\n",
       "     {'name': 'pinitol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6738'},\n",
       "     {'name': 'guadinomine B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7693'},\n",
       "     {'name': '6-diazo-5-oxo-L-norleucine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8297'},\n",
       "     {'name': 'detoxification of reactive carbonyls in chloroplasts',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6786'},\n",
       "     {'name': 'isethionate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2601'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, mitochondrial)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7384'},\n",
       "     {'name': '1,3-propanediol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7385'},\n",
       "     {'name': 'L-tryptophan degradation V (side chain pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3162'},\n",
       "     {'name': 'arabidopyrone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7682'},\n",
       "     {'name': 'dTDP-beta-L-olivose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6974'},\n",
       "     {'name': 'sporopollenin precursors biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6733'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3899300'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3899300'},\n",
       "     {'name': 'propane degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7774'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7337'},\n",
       "     {'name': 'androstenedione degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8152'},\n",
       "     {'name': 'rosmarinic acid biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5048'},\n",
       "     {'name': 'mupirocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8012'},\n",
       "     {'name': 'antimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8374'},\n",
       "     {'name': 'erinacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8408'},\n",
       "     {'name': 'jasmonic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-735'},\n",
       "     {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7300'},\n",
       "     {'name': 'pyruvate fermentation to hexanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6863'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7115'},\n",
       "     {'name': 'L-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7787'},\n",
       "     {'name': 'suberin monomers biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1121'},\n",
       "     {'name': 'cis-geranyl-CoA degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6672'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8411'},\n",
       "     {'name': 'aurofusarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7695'},\n",
       "     {'name': 'bile acid biosynthesis, neutral pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6061'},\n",
       "     {'name': 'phenylethanol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5751'},\n",
       "     {'name': 'mithramycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7045'},\n",
       "     {'name': 'arachidonate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8397'},\n",
       "     {'name': 'aclacinomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7354'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-4641265'},\n",
       "     {'name': '5-aminoimidazole ribonucleotide biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6122'},\n",
       "     {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6946'},\n",
       "     {'name': 'heme d1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7554'},\n",
       "     {'name': 'methyl tert-butyl ether degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7779'},\n",
       "     {'name': 'fusicoccin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6659'},\n",
       "     {'name': '(Z)-phenylmethanethial S-oxide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6539'},\n",
       "     {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6318'},\n",
       "     {'name': 'myxochelin A and B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8383'},\n",
       "     {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8132'},\n",
       "     {'name': 'L-malate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7686'},\n",
       "     {'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-977347'},\n",
       "     {'name': '11-cis-3-hydroxyretinal biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7043'},\n",
       "     {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8014'},\n",
       "     {'name': 'TCA cycle II (plants and fungi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5690'},\n",
       "     {'name': 'leporin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8370'},\n",
       "     {'name': 'superpathway of glucose and xylose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6901'},\n",
       "     {'name': 'photorespiration II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8362'},\n",
       "     {'name': 'abscisic acid degradation to phaseic acid',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5271'},\n",
       "     {'name': 'L-glucose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7130'},\n",
       "     {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7304'},\n",
       "     {'name': 'bikaverin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7692'},\n",
       "     {'name': '6-methylpretetramide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7811'},\n",
       "     {'name': 'phytosterol biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2541'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3769402'},\n",
       "     {'name': 'photorespiration I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-181'},\n",
       "     {'name': '6-gingerol analog biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6920'},\n",
       "     {'name': 'dihydroconiferyl alcohol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6302'},\n",
       "     {'name': 'phosalacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7769'},\n",
       "     {'name': 'phenolphthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7742'},\n",
       "     {'name': 'mycolate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWYG-321'},\n",
       "     {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7613'},\n",
       "     {'name': 'L-alanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8188'},\n",
       "     {'name': 'propane degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7775'},\n",
       "     {'name': \"inosine-5'-phosphate biosynthesis I\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6123'},\n",
       "     {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8086'},\n",
       "     {'name': 'L-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7295'},\n",
       "     {'name': 'apicidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7667'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-977347'},\n",
       "     {'name': '8-O-methylfusarubin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7689'},\n",
       "     {'name': '4-ethylphenol degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6080'},\n",
       "     {'name': 'Sensory processing of sound by inner hair cells of the cochlea',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9662360'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3769402'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-4641265'},\n",
       "     {'name': 't-anethole biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5867'},\n",
       "     {'name': 'the visual cycle (insects)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7042'},\n",
       "     {'name': '2-deoxy-D-ribose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8058'},\n",
       "     {'name': 'D-xylose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7294'},\n",
       "     {'name': 'quinoxaline-2-carboxylate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7734'},\n",
       "     {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3722'},\n",
       "     {'name': 'sulfolactate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6638'},\n",
       "     {'name': '3-hydroxy-L-homotyrosine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7649'},\n",
       "     {'name': 'mannojirimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7531'},\n",
       "     {'name': 'cinchona alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5848'},\n",
       "     {'name': 'L-glutamate degradation XI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8190'},\n",
       "     {'name': 'dTDP-L-daunosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7814'},\n",
       "     {'name': 'chitin deacetylation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7118'},\n",
       "     {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5754'},\n",
       "     {'name': 'p-cymene degradation to p-cumate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-741'},\n",
       "     {'name': 'limonene degradation IV (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8029'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-977347'},\n",
       "     {'name': 'salinosporamide A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6627'},\n",
       "     {'name': 'erythritol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7788'},\n",
       "     {'name': 'dTDP-beta-L-megosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7104'},\n",
       "     {'name': '2-keto-L-gulonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5782'},\n",
       "     {'name': 'UDP-beta-L-rhamnose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3261'},\n",
       "     {'name': 'androstenedione degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6944'},\n",
       "     {'name': 'validamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5818'},\n",
       "     {'name': 'phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7741'},\n",
       "     {'name': 'superpathway of glyoxylate cycle and fatty acid degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-561'},\n",
       "     {'name': 'sulfolactate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6616'},\n",
       "     {'name': 'myo-inositol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7241'},\n",
       "     {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6945'},\n",
       "     {'name': 'testosterone degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8155'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7726'},\n",
       "     {'name': 'alpha-solanine/alpha-chaconine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5666'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-4641265'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-3769402'},\n",
       "     {'name': 'dehydrophos biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6682'},\n",
       "     {'name': 'pyruvate fermentation to butanol II (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6883'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-4641265'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7333'},\n",
       "     {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6053'},\n",
       "     {'name': 'juglone degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7825'},\n",
       "     {'name': '4-oxopentanoate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7948'},\n",
       "     {'name': '(2R)-methoxymalonyl-[acp] biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8426'},\n",
       "     {'name': 'patulin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7490'},\n",
       "     {'name': 'cyathin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8409'},\n",
       "     {'name': 'cutin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-321'},\n",
       "     {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7288'},\n",
       "     {'name': 'ecdysteroid metabolism (arthropods)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7321'},\n",
       "     {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7440'},\n",
       "     {'name': 'meso-butanediol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6391'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7339'},\n",
       "     {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6626'},\n",
       "     {'name': 'prodigiosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7547'},\n",
       "     {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8017'},\n",
       "     {'name': 'tryptoquialanine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7660'},\n",
       "     {'name': 'formaldehyde assimilation I (serine pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1622'},\n",
       "     {'name': 'L-serine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8011'},\n",
       "     {'name': 'L-carnitine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3641'}]}},\n",
       "  'locations': [{'start': 156,\n",
       "    'end': 335,\n",
       "    'representative': False,\n",
       "    'hmmStart': 1,\n",
       "    'hmmEnd': 178,\n",
       "    'hmmLength': 178,\n",
       "    'hmmBounds': 'COMPLETE',\n",
       "    'evalue': 2.5e-44,\n",
       "    'score': 162.3,\n",
       "    'envelopeStart': 156,\n",
       "    'envelopeEnd': 335,\n",
       "    'postProcessed': True,\n",
       "    'location-fragments': [{'start': 156,\n",
       "      'end': 335,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'evalue': 1.8e-44,\n",
       "  'score': 162.7,\n",
       "  'model-ac': 'PF02826'},\n",
       " {'signature': {'accession': 'MF_03210',\n",
       "   'name': 'Formate_dehydrogenase',\n",
       "   'description': 'Formate dehydrogenase, mitochondrial.',\n",
       "   'signatureLibraryRelease': {'library': 'HAMAP', 'version': '2023_05'},\n",
       "   'entry': {'accession': 'IPR033689',\n",
       "    'name': 'FDH_NAD-dep',\n",
       "    'description': 'NAD-dependent formate dehydrogenase',\n",
       "    'type': 'FAMILY',\n",
       "    'goXRefs': [{'name': 'formate dehydrogenase (NAD+) activity',\n",
       "      'databaseName': 'GO',\n",
       "      'category': 'MOLECULAR_FUNCTION',\n",
       "      'id': 'GO:0008863'}],\n",
       "    'pathwayXRefs': [{'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'}]}},\n",
       "  'locations': [{'start': 2,\n",
       "    'end': 383,\n",
       "    'representative': False,\n",
       "    'score': 56.903183,\n",
       "    'alignment': 'Not available',\n",
       "    'location-fragments': [{'start': 2,\n",
       "      'end': 383,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'model-ac': 'MF_03210'},\n",
       " {'signature': {'accession': 'SSF52283',\n",
       "   'name': 'Formate/glycerate dehydrogenase catalytic domain-like',\n",
       "   'description': None,\n",
       "   'signatureLibraryRelease': {'library': 'SUPERFAMILY', 'version': '1.75'},\n",
       "   'entry': None},\n",
       "  'locations': [{'start': 3,\n",
       "    'end': 173,\n",
       "    'representative': False,\n",
       "    'hmmLength': 187,\n",
       "    'location-fragments': [{'start': 3,\n",
       "      'end': 173,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'evalue': 9.26e-65,\n",
       "  'model-ac': '0045380'},\n",
       " {'signature': {'accession': 'SSF51735',\n",
       "   'name': 'NAD(P)-binding Rossmann-fold domains',\n",
       "   'description': None,\n",
       "   'signatureLibraryRelease': {'library': 'SUPERFAMILY', 'version': '1.75'},\n",
       "   'entry': {'accession': 'IPR036291',\n",
       "    'name': 'NAD(P)-bd_dom_sf',\n",
       "    'description': 'NAD(P)-binding domain superfamily',\n",
       "    'type': 'HOMOLOGOUS_SUPERFAMILY',\n",
       "    'goXRefs': [],\n",
       "    'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8238'},\n",
       "     {'name': 'Glucocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-194002'},\n",
       "     {'name': 'salvigenin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7325'},\n",
       "     {'name': 'quercetin glucoside degradation (Allium)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7133'},\n",
       "     {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6973'},\n",
       "     {'name': 'icosapentaenoate biosynthesis II (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7049'},\n",
       "     {'name': 'Lysine catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-71064'},\n",
       "     {'name': 'nicotinate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5055'},\n",
       "     {'name': 'propene degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5534'},\n",
       "     {'name': 'Escherichia coli serotype O:157/Salmonella enterica serotype O:30 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8224'},\n",
       "     {'name': 'Phenylalanine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-8964208'},\n",
       "     {'name': 'UDP-beta-L-arabinose biosynthesis I (from UDP-alpha-D-xylose)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-63'},\n",
       "     {'name': '2,5-xylenol and 3,5-xylenol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7698'},\n",
       "     {'name': 'salidroside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6802'},\n",
       "     {'name': 'Androgen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-193048'},\n",
       "     {'name': 'Synthesis of bile acids and bile salts via 24-hydroxycholesterol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-193775'},\n",
       "     {'name': 'clorobiocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8368'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-977347'},\n",
       "     {'name': 'ascorbate glutathione cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2261'},\n",
       "     {'name': 'Phase I - Functionalization of compounds',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-211945'},\n",
       "     {'name': 'TP53 Regulates Metabolic Genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-5628897'},\n",
       "     {'name': 'fructan degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-862'},\n",
       "     {'name': 'meso-butanediol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6392'},\n",
       "     {'name': 'dipicolinate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8088'},\n",
       "     {'name': 'glyoxylate assimilation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5744'},\n",
       "     {'name': \"3,3'-thiodipropanoate degradation\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7465'},\n",
       "     {'name': 'arachidonate biosynthesis V (8-detaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7725'},\n",
       "     {'name': 'gondoate biosynthesis (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7663'},\n",
       "     {'name': 'Lipid particle organization',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-8964572'},\n",
       "     {'name': 'hypoglycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5826'},\n",
       "     {'name': 'alpha-linolenic acid (ALA) metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-2046106'},\n",
       "     {'name': 'Association of TriC/CCT with target proteins during biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-390471'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-mannosaminouronate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7335'},\n",
       "     {'name': 'Fructose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-5652227'},\n",
       "     {'name': 'palmitoleate biosynthesis I (from (5Z)-dodec-5-enoate)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6282'},\n",
       "     {'name': 'Glucocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-194002'},\n",
       "     {'name': 'D-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8331'},\n",
       "     {'name': 'Respiratory electron transport',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-611105'},\n",
       "     {'name': 'harzianum A and trichodermin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7714'},\n",
       "     {'name': 'limonene degradation III (to perillate)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6526'},\n",
       "     {'name': 'L-dopa degradation II (bacterial)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8110'},\n",
       "     {'name': 'Ion homeostasis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-5578775'},\n",
       "     {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7606'},\n",
       "     {'name': 'sorgoleone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5987'},\n",
       "     {'name': 'tetrapyrrole biosynthesis I (from glutamate)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5188'},\n",
       "     {'name': 'Ion homeostasis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5578775'},\n",
       "     {'name': 'D-carnitine degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7471'},\n",
       "     {'name': 'cytosolic NADPH production (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7268'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-70370'},\n",
       "     {'name': '3,6-anhydro-alpha-L-galactopyranose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7562'},\n",
       "     {'name': 'anandamide biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8053'},\n",
       "     {'name': 'CMP-pseudaminate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6143'},\n",
       "     {'name': 'pyruvate fermentation to isobutanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7111'},\n",
       "     {'name': '(4S)-carveol and (4S)-dihydrocarveol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5927'},\n",
       "     {'name': 'Bicarbonate transporters',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-425381'},\n",
       "     {'name': 'pederin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8049'},\n",
       "     {'name': 'indole-3-acetate biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-581'},\n",
       "     {'name': 'pyruvate-derived opine biosynthesis II (marine invertebrates)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7351'},\n",
       "     {'name': 'Androgen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CFA-193048'},\n",
       "     {'name': 'CDP-ascarylose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5830'},\n",
       "     {'name': '2-aminoethylphosphonate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6832'},\n",
       "     {'name': 'chaetoglobosin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7612'},\n",
       "     {'name': 'Wax biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-9640463'},\n",
       "     {'name': 'histamine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6181'},\n",
       "     {'name': '3-sulfopropanediol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8214'},\n",
       "     {'name': 'the visual cycle I (vertebrates)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6861'},\n",
       "     {'name': 'Streptococcus pneumoniae serotype 4 capsular polysaccharide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8461'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-3769402'},\n",
       "     {'name': 'Fructose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-5652227'},\n",
       "     {'name': '3-chlorobenzoate degradation I (via chlorocatechol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6088'},\n",
       "     {'name': 'Formation of the active cofactor, UDP-glucuronate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-173599'},\n",
       "     {'name': 'L-arabinose degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5517'},\n",
       "     {'name': '(4R)-4-[(E)-2-butenyl]-4-methyl-L-threonine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8441'},\n",
       "     {'name': 'eNOS activation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-203615'},\n",
       "     {'name': 'alizarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5802'},\n",
       "     {'name': 'Estrogen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-193144'},\n",
       "     {'name': 'CMP-legionaminate biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6749'},\n",
       "     {'name': 'L-arginine degradation XII',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7523'},\n",
       "     {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7184'},\n",
       "     {'name': 'Pyruvate metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-70268'},\n",
       "     {'name': 'virginiae butanolide type gamma-butyrolactones biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7881'},\n",
       "     {'name': 'p-cumate degradation to 2-hydroxypentadienoate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5163'},\n",
       "     {'name': 'Interconversion of nucleotide di- and triphosphates',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-499943'},\n",
       "     {'name': 'myo-inositol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2301'},\n",
       "     {'name': 'Neutrophil degranulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-6798695'},\n",
       "     {'name': 'sulfoacetaldehyde degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6718'},\n",
       "     {'name': 'Citric acid cycle (TCA cycle)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-71403'},\n",
       "     {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7300'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-CEL-70171'},\n",
       "     {'name': '3-methylarginine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6511'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7115'},\n",
       "     {'name': 'palmitate biosynthesis III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8279'},\n",
       "     {'name': 'docosahexaenoate biosynthesis II (bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7585'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8411'},\n",
       "     {'name': 'aurofusarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7695'},\n",
       "     {'name': 'alazopeptin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8293'},\n",
       "     {'name': 'Vitamin B5 (pantothenate) metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-199220'},\n",
       "     {'name': 'neurosporaxanthin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6681'},\n",
       "     {'name': 'arachidonate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8397'},\n",
       "     {'name': '1,2-dichlorobenzene degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6090'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-5365859'},\n",
       "     {'name': 'kanamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7000'},\n",
       "     {'name': 'arachidonate biosynthesis I (6-desaturase, lower eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5353'},\n",
       "     {'name': 'selenate reduction',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6932'},\n",
       "     {'name': 'D-galactose degradation I (Leloir pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6317'},\n",
       "     {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-8866907'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-RNO-70171'},\n",
       "     {'name': 'apicidin F biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7668'},\n",
       "     {'name': 'benzoate biosynthesis III (CoA-dependent, non-beta-oxidative)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6446'},\n",
       "     {'name': 'L-threonate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7874'},\n",
       "     {'name': 'pyruvate fermentation to (R)-acetoin I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5938'},\n",
       "     {'name': 'palmitate biosynthesis II (type II fatty acid synthase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5971'},\n",
       "     {'name': 'L-isoleucine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5078'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-70370'},\n",
       "     {'name': 'pyrimidine deoxyribonucleotide phosphorylation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7197'},\n",
       "     {'name': 'citronellol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6670'},\n",
       "     {'name': 'L-lysine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2941'},\n",
       "     {'name': 'peptidoglycan cross-bridge biosynthesis II (E. faecium)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6461'},\n",
       "     {'name': 'Abacavir metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2161541'},\n",
       "     {'name': 'Wax biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9640463'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-70370'},\n",
       "     {'name': 'gibberellin biosynthesis IV (Gibberella fujikuroi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5047'},\n",
       "     {'name': 'Branched-chain amino acid catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-70895'},\n",
       "     {'name': 'ultra-long-chain fatty acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8041'},\n",
       "     {'name': 'Abacavir metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-2161541'},\n",
       "     {'name': 'wighteone and luteone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4502'},\n",
       "     {'name': 'eupatolitin 3-O-glucoside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7157'},\n",
       "     {'name': \"UDP-N,N'-diacetylbacillosamine biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7028'},\n",
       "     {'name': 'Ub-specific processing proteases',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-5689880'},\n",
       "     {'name': 'chloramphenicol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8032'},\n",
       "     {'name': 'photorespiration I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-181'},\n",
       "     {'name': 'NiFe(CO)(CN)2 cofactor biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8319'},\n",
       "     {'name': 'dihydroconiferyl alcohol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6302'},\n",
       "     {'name': 'homophytochelatin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6841'},\n",
       "     {'name': 'Synthesis of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77111'},\n",
       "     {'name': 'IM-2 type gamma-butyrolactones biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7882'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-5365859'},\n",
       "     {'name': '(+)-camphor biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6990'},\n",
       "     {'name': 'triethylamine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7085'},\n",
       "     {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-SPO-70635'},\n",
       "     {'name': 'monolignol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-361'},\n",
       "     {'name': 'L-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7295'},\n",
       "     {'name': 'galloylated catechin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7448'},\n",
       "     {'name': 'ChREBP activates metabolic gene expression',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-163765'},\n",
       "     {'name': 'apicidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7667'},\n",
       "     {'name': 'alkane biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7033'},\n",
       "     {'name': 'Ca2+ activated K+ channels',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-1296052'},\n",
       "     {'name': 'leinamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8343'},\n",
       "     {'name': 'limonene degradation I (D-limonene)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5923'},\n",
       "     {'name': 'acetyl-CoA fermentation to butanoate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5676'},\n",
       "     {'name': '5-N-acetylardeemin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7543'},\n",
       "     {'name': 'quinoxaline-2-carboxylate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7734'},\n",
       "     {'name': 'adenosine ribonucleotides de novo biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7219'},\n",
       "     {'name': 'Gluconeogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-70263'},\n",
       "     {'name': 'limonene degradation IV (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8029'},\n",
       "     {'name': 'UDP-yelosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7842'},\n",
       "     {'name': '2-keto-L-gulonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5782'},\n",
       "     {'name': 'cholesterol biosynthesis (algae, late side-chain reductase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8191'},\n",
       "     {'name': 'Platelet degranulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-114608'},\n",
       "     {'name': 'Synthesis of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-77111'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7334'},\n",
       "     {'name': 'vanchrobactin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7987'},\n",
       "     {'name': 'Biosynthesis of E-series 18(S)-resolvins',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9018896'},\n",
       "     {'name': 'L-isoleucine biosynthesis III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5103'},\n",
       "     {'name': 'The canonical retinoid cycle in rods (twilight vision)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2453902'},\n",
       "     {'name': '(+)-pisatin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2467'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-70370'},\n",
       "     {'name': 'Bicarbonate transporters',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-425381'},\n",
       "     {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-2162123'},\n",
       "     {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-8866904'},\n",
       "     {'name': 'bacilysin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7626'},\n",
       "     {'name': 'TP53 regulates transcription of several additional cell death genes whose specific roles in p53-dependent apoptosis remain uncertain',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-6803205'},\n",
       "     {'name': 'momilactone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7477'},\n",
       "     {'name': 'Ethanol oxidation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-71384'},\n",
       "     {'name': 'ethene biosynthesis IV (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7126'},\n",
       "     {'name': 'norspermidine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6562'},\n",
       "     {'name': '2-allylmalonyl-CoA biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8427'},\n",
       "     {'name': 'pyrimidine deoxyribonucleotides biosynthesis from CTP',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7210'},\n",
       "     {'name': 'bacteriochlorophyll e biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7760'},\n",
       "     {'name': 'prodigiosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7547'},\n",
       "     {'name': 'heptadecane biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6622'},\n",
       "     {'name': 'Beta-oxidation of very long chain fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-390247'},\n",
       "     {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8017'},\n",
       "     {'name': 'tryptoquialanine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7660'},\n",
       "     {'name': '3beta-hydroxysesquiterpene lactone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7497'},\n",
       "     {'name': 'hypotaurine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7387'},\n",
       "     {'name': 'cyanidin diglucoside biosynthesis (acyl-glucose dependent)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7256'},\n",
       "     {'name': 'L-pyrrolysine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6994'},\n",
       "     {'name': 'rutin degradation (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7134'},\n",
       "     {'name': 'steviol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7070'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, cytosol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7383'},\n",
       "     {'name': 'stearate biosynthesis I (animals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5972'},\n",
       "     {'name': 'paromomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7018'},\n",
       "     {'name': 'septacidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8366'},\n",
       "     {'name': 'Acyl chain remodeling of CL',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-1482798'},\n",
       "     {'name': 'FR-900098 and FR-33289 antibiotics biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7419'},\n",
       "     {'name': 'even iso-branched-chain fatty acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8175'},\n",
       "     {'name': 'wax esters biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5885'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-RNO-156581'},\n",
       "     {'name': 'photorespiration III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8363'},\n",
       "     {'name': 'Phenylalanine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-8964208'},\n",
       "     {'name': 'astaxanthin biosynthesis (flowering plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7636'},\n",
       "     {'name': 'Acyl chain remodeling of CL',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-1482798'},\n",
       "     {'name': 'ceramide and sphingolipid recycling and degradation (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7119'},\n",
       "     {'name': '3-chlorotoluene degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6103'},\n",
       "     {'name': 'Synthesis of IP2, IP, and Ins in the cytosol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-1855183'},\n",
       "     {'name': 'pyoluteorin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7930'},\n",
       "     {'name': '3-chlorotoluene degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6104'},\n",
       "     {'name': 'Fatty acyl-CoA biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-75105'},\n",
       "     {'name': 'glutathione-mediated detoxification II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6842'},\n",
       "     {'name': 'hyperxanthone E biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8442'},\n",
       "     {'name': 'Estrogen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-193144'},\n",
       "     {'name': \"6'-deoxychalcone metabolism\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5161'},\n",
       "     {'name': 'Prednisone ADME',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-9757110'},\n",
       "     {'name': 'arginomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7569'},\n",
       "     {'name': 'L-isoleucine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5101'},\n",
       "     {'name': 'coumestrol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6332'},\n",
       "     {'name': 'plasmalogen biosynthesis II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8335'},\n",
       "     {'name': 'peptidoglycan cross-bridge biosynthesis IV (Weissella viridescens)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6463'},\n",
       "     {'name': 'Synthesis of bile acids and bile salts via 27-hydroxycholesterol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-193807'},\n",
       "     {'name': 'Utilization of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-77108'},\n",
       "     {'name': \"6,7,4'-trihydroxyisoflavone biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5061'},\n",
       "     {'name': 'pyrethrin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6926'},\n",
       "     {'name': '(8E,10E)-dodeca-8,10-dienol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7654'},\n",
       "     {'name': 'cinnamate and 3-hydroxycinnamate degradation to 2-hydroxypentadienoate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6690'},\n",
       "     {'name': 'quinate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6416'},\n",
       "     {'name': 'phosphopantothenate biosynthesis III (archaea)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6654'},\n",
       "     {'name': 'vestitol and sativan biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5729'},\n",
       "     {'name': 'all trans undecaprenyl diphosphate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5816'},\n",
       "     {'name': 'glycolysis VI (from fructose)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8404'},\n",
       "     {'name': 'bacteriochlorophyll c biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7759'},\n",
       "     {'name': 'Metabolism of folate and pterines',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-196757'},\n",
       "     {'name': 'Shigella boydii serotype 6 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8232'},\n",
       "     {'name': 'type I lipoteichoic acid biosynthesis (S. aureus)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7817'},\n",
       "     {'name': 'sulfolactate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6637'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-HSA-70171'},\n",
       "     {'name': 'maysin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5793'},\n",
       "     {'name': 'Signaling by TCF7L2 mutants',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5339700'},\n",
       "     {'name': 'L-ascorbate degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6704'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3769402'},\n",
       "     {'name': 'dechlorogriseofulvin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7655'},\n",
       "     {'name': 'meleagrin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7607'},\n",
       "     {'name': 'CDP-6-deoxy-D-gulose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8139'},\n",
       "     {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-77352'},\n",
       "     {'name': '(R)- and (S)-3-hydroxybutanoate biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7216'},\n",
       "     {'name': 'putrescine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-0'},\n",
       "     {'name': 'Cholesterol biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-191273'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NADP-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-241'},\n",
       "     {'name': 'GDP-fucose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-6787639'},\n",
       "     {'name': '1,5-anhydrofructose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6992'},\n",
       "     {'name': 'RHOD GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9013405'},\n",
       "     {'name': 'hypericin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5780'},\n",
       "     {'name': 'geraniol and nerol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6678'},\n",
       "     {'name': '7-dehydroporiferasterol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7155'},\n",
       "     {'name': 'Complex I biogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-6799198'},\n",
       "     {'name': 'PPARA activates gene expression',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-1989781'},\n",
       "     {'name': 'superpathway of pterocarpan biosynthesis (via formononetin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2229'},\n",
       "     {'name': 'The canonical retinoid cycle in rods (twilight vision)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-2453902'},\n",
       "     {'name': '(-)-maackiain biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2464'},\n",
       "     {'name': 'fumagillin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8407'},\n",
       "     {'name': 'noscapine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7138'},\n",
       "     {'name': 'protein SAMPylation and SAMP-mediated thiolation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7887'},\n",
       "     {'name': 'GDP-fucose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-6787639'},\n",
       "     {'name': 'Sperm Motility And Taxes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-1300642'},\n",
       "     {'name': 'L-methionine biosynthesis IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7977'},\n",
       "     {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-8866907'},\n",
       "     {'name': 'phaselate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6320'},\n",
       "     {'name': 'cGMP effects',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-418457'},\n",
       "     {'name': 'pyruvate fermentation to hexanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6863'},\n",
       "     {'name': 'yersiniabactin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6407'},\n",
       "     {'name': 'retinol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6857'},\n",
       "     {'name': 'gibberellin inactivation III (epoxidation)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6494'},\n",
       "     {'name': 'TP53 Regulates Metabolic Genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-5628897'},\n",
       "     {'name': 'chitin degradation I (archaea)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6855'},\n",
       "     {'name': 'Abacavir metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-2161541'},\n",
       "     {'name': 'Complex I biogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-6799198'},\n",
       "     {'name': 'Sulfur amino acid metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-1614635'},\n",
       "     {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-2162123'},\n",
       "     {'name': '3-hydroxypropanoate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5743'},\n",
       "     {'name': 'Utilization of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77108'},\n",
       "     {'name': 'L-lysine biosynthesis V',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3081'},\n",
       "     {'name': 'heme b biosynthesis III (from siroheme)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7552'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-MMU-156581'},\n",
       "     {'name': 'furaneol and mesifurane biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5975'},\n",
       "     {'name': 'Sulfur amino acid metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-1614635'},\n",
       "     {'name': 'Glutamate and glutamine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-8964539'},\n",
       "     {'name': 'tylosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7415'},\n",
       "     {'name': 'papaverine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7363'},\n",
       "     {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6946'},\n",
       "     {'name': 'methyl tert-butyl ether degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7779'},\n",
       "     {'name': 'D-erythronate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7873'},\n",
       "     {'name': 'beta-alanine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8120'},\n",
       "     {'name': 'Androgen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-193048'},\n",
       "     {'name': 'Phase I - Functionalization of compounds',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-211945'},\n",
       "     {'name': 'aloin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6312'},\n",
       "     {'name': 'CDP-diacylglycerol biosynthesis III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5981'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-977347'},\n",
       "     {'name': 'gliotoxin inactivation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7534'},\n",
       "     {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8014'},\n",
       "     {'name': 'neopentalenoketolactone and pentalenate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6919'},\n",
       "     {'name': 'CDP-diacylglycerol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5667'},\n",
       "     {'name': 'heme degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5874'},\n",
       "     {'name': 'aureobasidin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7665'},\n",
       "     {'name': 'Biosynthesis of D-series resolvins',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9018676'},\n",
       "     {'name': 'indole-3-acetate activation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1921'},\n",
       "     {'name': 'benzene degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5450'},\n",
       "     {'name': 'glycine lipid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8082'},\n",
       "     {'name': 'Cytoprotection by HMOX1',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9707564'},\n",
       "     {'name': 'Citric acid cycle (TCA cycle)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-71403'},\n",
       "     {'name': 'mannitol cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6531'},\n",
       "     {'name': 'Biosynthesis of D-series resolvins',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-9018676'},\n",
       "     {'name': 'aspyridone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7565'},\n",
       "     {'name': 'phosalacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7769'},\n",
       "     {'name': 'hinokiresinol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7632'},\n",
       "     {'name': 'Escherichia coli serotype O:117 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8220'},\n",
       "     {'name': 'luteolinidin 5-O-glucoside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7252'},\n",
       "     {'name': 'juvenile hormone III biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6575'},\n",
       "     {'name': 'kievitone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4681'},\n",
       "     {'name': 'GDP-fucose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-6787639'},\n",
       "     {'name': 'luteolin triglucuronide degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7445'},\n",
       "     {'name': 'propane degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7775'},\n",
       "     {'name': 'GDP-fucose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-6787639'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-SCE-156581'},\n",
       "     {'name': 'Glutamate and glutamine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-8964539'},\n",
       "     {'name': 'sterigmatocystin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5956'},\n",
       "     {'name': 'bacillibactin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5903'},\n",
       "     {'name': 'naringenin biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7397'},\n",
       "     {'name': '8-O-methylfusarubin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7689'},\n",
       "     {'name': '4-ethylphenol degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6080'},\n",
       "     {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7187'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3769402'},\n",
       "     {'name': 'D-xylose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7294'},\n",
       "     {'name': 'Sulfur amino acid metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-1614635'},\n",
       "     {'name': 'bacteriochlorophyll d biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7758'},\n",
       "     {'name': 'UDP-N-acetyl-D-galactosamine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5514'},\n",
       "     {'name': 'dTDP-beta-D-fucofuranose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7312'},\n",
       "     {'name': '3-hydroxy-L-homotyrosine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7649'},\n",
       "     {'name': \"myxol-2' fucoside biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6279'},\n",
       "     {'name': 'glycerol degradation to butanol',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7003'},\n",
       "     {'name': 'Beta-oxidation of pristanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-389887'},\n",
       "     {'name': 'farnesylcysteine salvage pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6577'},\n",
       "     {'name': 'Fructose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-5652227'},\n",
       "     {'name': 'CLEC7A (Dectin-1) induces NFAT activation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5607763'},\n",
       "     {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-77348'},\n",
       "     {'name': 'L-methionine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5082'},\n",
       "     {'name': 'sorbitol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5530'},\n",
       "     {'name': 'superpathway of glyoxylate cycle and fatty acid degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-561'},\n",
       "     {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6945'},\n",
       "     {'name': 'cyanidin 3,7-diglucoside polyacylation biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7464'},\n",
       "     {'name': 'homofuraneol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7910'},\n",
       "     {'name': 'GDP-L-fucose biosynthesis I (from GDP-D-mannose)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-66'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7726'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-3769402'},\n",
       "     {'name': 'ascorbate recycling (cytosolic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6370'},\n",
       "     {'name': 'brassinosteroids inactivation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6546'},\n",
       "     {'name': 'Synthesis of bile acids and bile salts via 7alpha-hydroxycholesterol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-193368'},\n",
       "     {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-77352'},\n",
       "     {'name': 'mannitol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3861'},\n",
       "     {'name': 'aloenin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6310'},\n",
       "     {'name': 'patulin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7490'},\n",
       "     {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7440'},\n",
       "     {'name': 'ecdysteroid metabolism (arthropods)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7321'},\n",
       "     {'name': 'ternatin C3 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7280'},\n",
       "     {'name': 'adipate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8354'},\n",
       "     {'name': 'D-arabinose degradation V',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8334'},\n",
       "     {'name': 'soybean saponin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5203'},\n",
       "     {'name': 'oleandomycin activation/inactivation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6972'},\n",
       "     {'name': '2,3-dihydroxybenzoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5901'},\n",
       "     {'name': 'UDP-alpha-D-galactose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7344'},\n",
       "     {'name': 'capsiconiate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6027'},\n",
       "     {'name': 'formaldehyde assimilation I (serine pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1622'},\n",
       "     {'name': 'naphthalene degradation (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5427'},\n",
       "     {'name': 'tabtoxinine-beta-lactam biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7954'},\n",
       "     {'name': 'neoxaline biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7608'},\n",
       "     {'name': 'paromamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7014'},\n",
       "     {'name': 'Prednisone ADME',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-9757110'},\n",
       "     {'name': 'di-homo-gamma-linolenate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8396'},\n",
       "     {'name': 'divinyl ether biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5409'},\n",
       "     {'name': '(-)-dehydrodiconiferyl alcohol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7557'},\n",
       "     {'name': 'Defective AHCY causes HMAHCHD',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5578997'},\n",
       "     {'name': 'Peroxisomal protein import',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-9033241'},\n",
       "     {'name': 'shikimate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6419'},\n",
       "     {'name': 'Ethanol oxidation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-71384'},\n",
       "     {'name': 'pyruvate fermentation to acetate V',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5537'},\n",
       "     {'name': 'Pyruvate metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-70268'},\n",
       "     {'name': 'A-factor gamma-butyrolactone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7878'},\n",
       "     {'name': 'Transferrin endocytosis and recycling',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-917977'},\n",
       "     {'name': 'GDP-L-colitose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5740'},\n",
       "     {'name': 'fatty acid beta-oxidation III (unsaturated, odd number)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5137'},\n",
       "     {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7610'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3899300'},\n",
       "     {'name': 'Ethanol oxidation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-71384'},\n",
       "     {'name': 'guanosine ribonucleotides de novo biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7221'},\n",
       "     {'name': 'dTDP-alpha-D-forosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6808'},\n",
       "     {'name': 'daunorubicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7352'},\n",
       "     {'name': 'Sphingolipid de novo biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-1660661'},\n",
       "     {'name': 'stearate biosynthesis IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8280'},\n",
       "     {'name': 'aureothin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8033'},\n",
       "     {'name': 'Beta-oxidation of pristanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-389887'},\n",
       "     {'name': 'UDP-2-acetamido-4-amino-2,4,6-trideoxy-alpha-D-galactose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7467'},\n",
       "     {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-8866904'},\n",
       "     {'name': 'Lysine catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-71064'},\n",
       "     {'name': 'trans-zeatin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2681'},\n",
       "     {'name': 'L-tyrosine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3461'},\n",
       "     {'name': 'L-arabinose degradation V',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8329'},\n",
       "     {'name': 'Formation of the active cofactor, UDP-glucuronate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-173599'},\n",
       "     {'name': 'glycolysis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-1042'},\n",
       "     {'name': 'dTDP-6-deoxy-alpha-D-allose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7413'},\n",
       "     {'name': 'chlorophyll cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5068'},\n",
       "     {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5882'},\n",
       "     {'name': 'Neutrophil degranulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-6798695'},\n",
       "     {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7301'},\n",
       "     {'name': 'adenosine deoxyribonucleotides de novo biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7220'},\n",
       "     {'name': 'Synthesis of PA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-1483166'},\n",
       "     {'name': 'bacteriochlorophyll a biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5526'},\n",
       "     {'name': 'ochratoxin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8116'},\n",
       "     {'name': 'UTP and CTP de novo biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7176'},\n",
       "     {'name': 'Regulation of insulin secretion',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-422356'},\n",
       "     {'name': 'Wax biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-9640463'},\n",
       "     {'name': 'terrequinone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7520'},\n",
       "     {'name': 'Tetrahydrobiopterin (BH4) synthesis, recycling, salvage and regulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-1474151'},\n",
       "     {'name': 'Phenylalanine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-8964208'},\n",
       "     {'name': 'folate transformations I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2201'},\n",
       "     {'name': '(-)-medicarpin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2463'},\n",
       "     {'name': 'ascomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8424'},\n",
       "     {'name': 'linustatin bioactivation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7091'},\n",
       "     {'name': 'staphyloferrin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8008'},\n",
       "     {'name': 'xanthohumol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5135'},\n",
       "     {'name': 'cob(II)yrinate a,c-diamide biosynthesis I (early cobalt insertion)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7377'},\n",
       "     {'name': 'stephacidin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7603'},\n",
       "     {'name': 'Heme degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-189483'},\n",
       "     {'name': 'bacterial bioluminescence',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7723'},\n",
       "     {'name': 'Beta oxidation of myristoyl-CoA to lauroyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77285'},\n",
       "     {'name': 'Complex I biogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-6799198'},\n",
       "     {'name': 'L-glutamate degradation X',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5766'},\n",
       "     {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7615'},\n",
       "     {'name': 'Neutrophil degranulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-6798695'},\n",
       "     {'name': 'Beta-oxidation of pristanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-389887'},\n",
       "     {'name': 'pectin degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7248'},\n",
       "     {'name': 'acrylate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6373'},\n",
       "     {'name': 'bixin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5305'},\n",
       "     {'name': 'L-ascorbate biosynthesis I (plants, L-galactose pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-882'},\n",
       "     {'name': 'Pentose phosphate pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-71336'},\n",
       "     {'name': 'brassinolide biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-699'},\n",
       "     {'name': '2-methylpropene degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7778'},\n",
       "     {'name': 'sphingolipid biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5129'},\n",
       "     {'name': 'Formation of xylulose-5-phosphate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-5661270'},\n",
       "     {'name': 'Peroxisomal protein import',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9033241'},\n",
       "     {'name': 'hydroxycinnamate sugar acid ester biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7461'},\n",
       "     {'name': 'alginate biosynthesis I (algal)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6073'},\n",
       "     {'name': 'D-apionate degradation I (xylose isomerase family decarboxylase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8091'},\n",
       "     {'name': 'arabidopyrone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7682'},\n",
       "     {'name': 'Transcriptional activation of mitochondrial biogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-2151201'},\n",
       "     {'name': 'Synthesis of Lipoxins (LX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2142700'},\n",
       "     {'name': 'griseofulvin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7653'},\n",
       "     {'name': 'homotaurine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6321'},\n",
       "     {'name': 'nepetalactone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8069'},\n",
       "     {'name': 'mupirocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8012'},\n",
       "     {'name': 'acyl-[acyl-carrier protein] thioesterase pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5142'},\n",
       "     {'name': 'Ub-specific processing proteases',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-5689880'},\n",
       "     {'name': 'phosphatidylinositol biosynthesis I (bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6580'},\n",
       "     {'name': 'erinacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8408'},\n",
       "     {'name': 'jasmonic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-735'},\n",
       "     {'name': 'flavonol acylglucoside biosynthesis II - isorhamnetin derivatives',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7171'},\n",
       "     {'name': 'Prednisone ADME',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9757110'},\n",
       "     {'name': 'Amaryllidacea alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7826'},\n",
       "     {'name': 'pyruvate fermentation to (S)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5481'},\n",
       "     {'name': 'Biosynthesis of E-series 18(S)-resolvins',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-9018896'},\n",
       "     {'name': '2-amino-3-hydroxycyclopent-2-enone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7536'},\n",
       "     {'name': 'Beta oxidation of palmitoyl-CoA to myristoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-77305'},\n",
       "     {'name': 'acridone alkaloid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5958'},\n",
       "     {'name': 'anthocyanidin acylglucoside and acylsambubioside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7679'},\n",
       "     {'name': 'ferulate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6343'},\n",
       "     {'name': 'fusicoccin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6659'},\n",
       "     {'name': 'lovastatin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7535'},\n",
       "     {'name': 'icosapentaenoate biosynthesis I (lower eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6958'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-DME-70171'},\n",
       "     {'name': 'gallate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6707'},\n",
       "     {'name': 'formaldehyde oxidation VI (H4MPT pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1723'},\n",
       "     {'name': 'Branched-chain amino acid catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-70895'},\n",
       "     {'name': 'L-arginine biosynthesis IV (archaea)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7400'},\n",
       "     {'name': 'quebrachitol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8078'},\n",
       "     {'name': 'actinomycin D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7718'},\n",
       "     {'name': '2,5-dichlorotoluene degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6191'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-5365859'},\n",
       "     {'name': 'pentagalloylglucose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5475'},\n",
       "     {'name': 'calystegine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5318'},\n",
       "     {'name': 'Utilization of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-77108'},\n",
       "     {'name': 'Sulfur amino acid metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-1614635'},\n",
       "     {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7304'},\n",
       "     {'name': 'bile acid 7beta-dehydroxylation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8134'},\n",
       "     {'name': 'TYSND1 cleaves peroxisomal proteins',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9033500'},\n",
       "     {'name': 'Branched-chain amino acid catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-70895'},\n",
       "     {'name': 'S-benzyl-L-cysteine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8418'},\n",
       "     {'name': '3-methylbutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6871'},\n",
       "     {'name': 'siroheme biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5194'},\n",
       "     {'name': 'Gluconeogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-70263'},\n",
       "     {'name': 'gibberellin biosynthesis III (early C-13 hydroxylation)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5035'},\n",
       "     {'name': '3-dehydroquinate biosynthesis II (archaea)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6160'},\n",
       "     {'name': '3-hydroxyquinaldate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7733'},\n",
       "     {'name': 'Ub-specific processing proteases',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-5689880'},\n",
       "     {'name': 'dTDP-alpha-D-ravidosamine and dTDP-4-acetyl-alpha-D-ravidosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7688'},\n",
       "     {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7613'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-77346'},\n",
       "     {'name': 'nivalenol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7713'},\n",
       "     {'name': 'mycinamicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7412'},\n",
       "     {'name': 'CDP-ribitol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8416'},\n",
       "     {'name': 'gamma-coniciene and coniine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5748'},\n",
       "     {'name': 'equisetin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7669'},\n",
       "     {'name': 'Synthesis of PA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-XTR-1483166'},\n",
       "     {'name': 'docosahexaenoate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8400'},\n",
       "     {'name': 'L-threonine degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5436'},\n",
       "     {'name': 'ethylmalonyl-CoA pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5741'},\n",
       "     {'name': 'glycerol-3-phosphate shuttle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6118'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-DRE-156581'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-5365859'},\n",
       "     {'name': 'acylceramide biosynthesis and processing',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8042'},\n",
       "     {'name': 'Sulfur amino acid metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-1614635'},\n",
       "     {'name': 'mannojirimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7531'},\n",
       "     {'name': 'sulfolactate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6638'},\n",
       "     {'name': 'L-glutamate degradation XI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8190'},\n",
       "     {'name': 'dTDP-L-daunosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7814'},\n",
       "     {'name': 'Pyruvate metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-70268'},\n",
       "     {'name': 'phosphatidylcholine biosynthesis VII',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7470'},\n",
       "     {'name': 'erythritol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7788'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-GGA-70171'},\n",
       "     {'name': 'alpha-linolenic acid (ALA) metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2046106'},\n",
       "     {'name': 'androstenedione degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6944'},\n",
       "     {'name': 'RHOF GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9035034'},\n",
       "     {'name': 'tenellin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7561'},\n",
       "     {'name': 'icosapentaenoate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8399'},\n",
       "     {'name': 'dTDP-D-desosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6942'},\n",
       "     {'name': 'Formation of the active cofactor, UDP-glucuronate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-173599'},\n",
       "     {'name': 'D-glucuronate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5525'},\n",
       "     {'name': 'viridicatumtoxin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7659'},\n",
       "     {'name': 'resveratrol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-84'},\n",
       "     {'name': 'dehydrophos biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6682'},\n",
       "     {'name': 'Ca2+ activated K+ channels',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-1296052'},\n",
       "     {'name': 'aspoquinolone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8388'},\n",
       "     {'name': 'myo-, chiro- and scyllo-inositol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7237'},\n",
       "     {'name': 'Synthesis of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-77111'},\n",
       "     {'name': 'Mycothiol biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MTU-879299'},\n",
       "     {'name': 'crotonyl-CoA/ethylmalonyl-CoA/hydroxybutyryl-CoA cycle (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7854'},\n",
       "     {'name': 'vibriobactin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6374'},\n",
       "     {'name': 'diazepinomicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8382'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-77346'},\n",
       "     {'name': '4-oxopentanoate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7948'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-77346'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosa-4,7,10,13,16-pentaenoate biosynthesis II (4-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7728'},\n",
       "     {'name': 'CMP phosphorylation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7205'},\n",
       "     {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6626'},\n",
       "     {'name': 'fatty acid biosynthesis initiation (type II)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4381'},\n",
       "     {'name': 'Escherichia coli serotype O:128 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8222'},\n",
       "     {'name': 'oleate beta-oxidation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7307'},\n",
       "     {'name': 'sanguinarine and macarpine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5287'},\n",
       "     {'name': 'Pentose phosphate pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-71336'},\n",
       "     {'name': 'mycobacterial sulfolipid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7746'},\n",
       "     {'name': 'cyclohexane-1-carboxylate degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-301'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-70370'},\n",
       "     {'name': 'Beta-oxidation of very long chain fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-390247'},\n",
       "     {'name': 'Peroxisomal protein import',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-9033241'},\n",
       "     {'name': 'CDP-4-dehydro-3,6-dideoxy-D-glucose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5833'},\n",
       "     {'name': 'cholesterol degradation to androstenedione III (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8151'},\n",
       "     {'name': 'methylerythritol phosphate pathway II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7560'},\n",
       "     {'name': '8-amino-7-oxononanoate biosynthesis IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8203'},\n",
       "     {'name': 'tetrahydrofolate salvage from 5,10-methenyltetrahydrofolate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6613'},\n",
       "     {'name': '3-hydroxypropanoate/4-hydroxybutanate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5789'},\n",
       "     {'name': 'cuticular wax biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-282'},\n",
       "     {'name': 'docosahexaenoate biosynthesis I (lower eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7053'},\n",
       "     {'name': 'benzoate degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2503'},\n",
       "     {'name': 'Formation of the active cofactor, UDP-glucuronate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-173599'},\n",
       "     {'name': 'Beta-oxidation of very long chain fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-390247'},\n",
       "     {'name': 'ChREBP activates metabolic gene expression',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-163765'},\n",
       "     {'name': 'propanoate fermentation to 2-methylbutanoate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5109'},\n",
       "     {'name': '1,3,6,7-tetrahydroxyxanthone biosynthesis (from benzoyl-CoA)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5001'},\n",
       "     {'name': 'Ethanol oxidation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-71384'},\n",
       "     {'name': 'cichoriin interconversion',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7057'},\n",
       "     {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7340'},\n",
       "     {'name': 'Platelet degranulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-114608'},\n",
       "     {'name': '9-lipoxygenase and 9-hydroperoxide lyase pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5408'},\n",
       "     {'name': 'TP53 Regulates Metabolic Genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-5628897'},\n",
       "     {'name': '(S)-camphor degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6989'},\n",
       "     {'name': 'RHOF GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9035034'},\n",
       "     {'name': 'L-arabinose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5515'},\n",
       "     {'name': 'Peroxisomal protein import',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-9033241'},\n",
       "     {'name': 'adenosine deoxyribonucleotides de novo biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7227'},\n",
       "     {'name': 'butirosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7019'},\n",
       "     {'name': 'valproate beta-oxidation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8182'},\n",
       "     {'name': 'seleno-amino acid biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6936'},\n",
       "     {'name': 'ergotamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6495'},\n",
       "     {'name': 'L-lysine degradation V',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5283'},\n",
       "     {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-77310'},\n",
       "     {'name': 'raspberry ketone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5393'},\n",
       "     {'name': 'succinate fermentation to butanoate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5677'},\n",
       "     {'name': 'D-glucuronate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6501'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-DDI-70171'},\n",
       "     {'name': 'colanic acid (Escherichia coli K12) biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8243'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-SPO-156581'},\n",
       "     {'name': 'cinnamoyl-CoA biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6457'},\n",
       "     {'name': 'Metabolism of ingested SeMet, Sec, MeSec into H2Se',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2408508'},\n",
       "     {'name': 'Gluconeogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-70263'},\n",
       "     {'name': 'Platelet degranulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-114608'},\n",
       "     {'name': 'poly(ribitol phosphate) wall teichoic acid biosynthesis II (S. aureus)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7816'},\n",
       "     {'name': 'cylindrospermopsin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8045'},\n",
       "     {'name': 'L-arginine biosynthesis III (via N-acetyl-L-citrulline)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5154'},\n",
       "     {'name': 'ferrichrome A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7571'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-389661'},\n",
       "     {'name': 'linoleate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8395'},\n",
       "     {'name': 'sterol biosynthesis (methylotrophs)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8026'},\n",
       "     {'name': 'cis-vaccenate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5973'},\n",
       "     {'name': 'CMP-diacetamido-8-epilegionaminic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7719'},\n",
       "     {'name': 'holomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7690'},\n",
       "     {'name': 'marneral biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6005'},\n",
       "     {'name': 'The canonical retinoid cycle in rods (twilight vision)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-2453902'},\n",
       "     {'name': 'ferulate and sinapate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5168'},\n",
       "     {'name': 'reductive TCA cycle II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5392'},\n",
       "     {'name': 'adipate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8347'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-70370'},\n",
       "     {'name': 'TP53 Regulates Metabolic Genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-5628897'},\n",
       "     {'name': 'D-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7786'},\n",
       "     {'name': 'oryzalexin A, B, and C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7489'},\n",
       "     {'name': 'taxol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5660'},\n",
       "     {'name': 'versicolorin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5955'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-SSC-70171'},\n",
       "     {'name': 'dTDP-beta-L-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6976'},\n",
       "     {'name': 'citreoisocoumarin and bikisocoumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7696'},\n",
       "     {'name': 'superpathway of nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7342'},\n",
       "     {'name': 'Peroxisomal protein import',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9033241'},\n",
       "     {'name': 'chorismate biosynthesis from 3-dehydroquinate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6163'},\n",
       "     {'name': 'Formation of xylulose-5-phosphate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-5661270'},\n",
       "     {'name': 'Sphingolipid de novo biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-1660661'},\n",
       "     {'name': \"abscisic acid degradation to 7'-hydroxyabscisate\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7642'},\n",
       "     {'name': 'Synthesis of Ketone Bodies',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-77111'},\n",
       "     {'name': 'Citric acid cycle (TCA cycle)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-71403'},\n",
       "     {'name': 'Ethanol oxidation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-71384'},\n",
       "     {'name': 'pyruvate fermentation to acetate VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5538'},\n",
       "     {'name': 'sciadonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6598'},\n",
       "     {'name': 'cyclosporin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8443'},\n",
       "     {'name': 'guanosine deoxyribonucleotides de novo biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7226'},\n",
       "     {'name': 'Citric acid cycle (TCA cycle)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-71403'},\n",
       "     {'name': 'flavonoid biosynthesis (in equisetum)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6787'},\n",
       "     {'name': 'L-rhamnose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6713'},\n",
       "     {'name': '3-sulfopropanediol degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6634'},\n",
       "     {'name': 'beta-D-mannosyl phosphomycoketide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7740'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-77346'},\n",
       "     {'name': 'L-lysine biosynthesis VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5097'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-4641265'},\n",
       "     {'name': 'Escherichia coli serotype O:169 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8233'},\n",
       "     {'name': 'Androgen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-193048'},\n",
       "     {'name': 'arachidonate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7592'},\n",
       "     {'name': 'Escherichia coli serotype O:55/Salmonella enterica serotype O:50 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8217'},\n",
       "     {'name': 'pinitol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6738'},\n",
       "     {'name': 'Glutamate and glutamine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-8964539'},\n",
       "     {'name': '6-diazo-5-oxo-L-norleucine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8297'},\n",
       "     {'name': 'Ub-specific processing proteases',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5689880'},\n",
       "     {'name': 'Salmonella enterica serotype O:18 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8206'},\n",
       "     {'name': 'detoxification of reactive carbonyls in chloroplasts',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6786'},\n",
       "     {'name': 'L-tryptophan degradation V (side chain pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3162'},\n",
       "     {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77352'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3899300'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': 'testosterone and androsterone degradation to androstendione (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6943'},\n",
       "     {'name': 'Defective GALE causes EDG',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5609977'},\n",
       "     {'name': 'androstenedione degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8152'},\n",
       "     {'name': 'Metabolism of folate and pterines',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-196757'},\n",
       "     {'name': 'echinocandin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7650'},\n",
       "     {'name': 'orientin and isoorientin biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7188'},\n",
       "     {'name': 'tetrahydropteridine recycling',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8099'},\n",
       "     {'name': 'L-fucose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8444'},\n",
       "     {'name': 'Beta oxidation of palmitoyl-CoA to myristoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-77305'},\n",
       "     {'name': 'L-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7787'},\n",
       "     {'name': 'blasticidin S biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7570'},\n",
       "     {'name': 'neolinustatin bioactivation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7092'},\n",
       "     {'name': 'L-phenylalanine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3462'},\n",
       "     {'name': 'rifamycin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5984'},\n",
       "     {'name': 'spermidine hydroxycinnamic acid conjugates biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6442'},\n",
       "     {'name': 'bile acid biosynthesis, neutral pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6061'},\n",
       "     {'name': 'Escherichia coli serotype O:149/Shigella boydii serotype O1 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8253'},\n",
       "     {'name': 'serotonin degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6313'},\n",
       "     {'name': 'Bicarbonate transporters',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-425381'},\n",
       "     {'name': 'heme d1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7554'},\n",
       "     {'name': 'methyl ketone biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7007'},\n",
       "     {'name': 'Escherichia coli serotype O:104 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7530'},\n",
       "     {'name': 'GDP-mycosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7573'},\n",
       "     {'name': 'podophyllotoxin glucosides metabolism',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7491'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-BTA-156581'},\n",
       "     {'name': 'grayanate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8326'},\n",
       "     {'name': 'Retinoid metabolism and transport',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-975634'},\n",
       "     {'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'GDP-L-galactose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5115'},\n",
       "     {'name': 'Lipid particle organization',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-8964572'},\n",
       "     {'name': 'L-leucine degradation IV (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7767'},\n",
       "     {'name': 'reductive glycine pathway of autotrophic CO2 fixation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8303'},\n",
       "     {'name': 'caffeoylglucarate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6673'},\n",
       "     {'name': 'Cytoprotection by HMOX1',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9707564'},\n",
       "     {'name': 'fatty acid biosynthesis initiation (plant mitochondria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6799'},\n",
       "     {'name': 'Ion homeostasis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-5578775'},\n",
       "     {'name': 'Formation of xylulose-5-phosphate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-5661270'},\n",
       "     {'name': 'FCGR3A-mediated IL10 synthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9664323'},\n",
       "     {'name': 'L-glucose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7130'},\n",
       "     {'name': 'sulfazecin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8381'},\n",
       "     {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77348'},\n",
       "     {'name': 'Estrogen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-193144'},\n",
       "     {'name': 'L-proline biosynthesis III (from L-ornithine)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3341'},\n",
       "     {'name': 'benzoyl-CoA biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6458'},\n",
       "     {'name': 'petroselinate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5367'},\n",
       "     {'name': 'TP53 Regulates Transcription of Genes Involved in Cytochrome C Release',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-6803204'},\n",
       "     {'name': 'Prednisone ADME',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-9757110'},\n",
       "     {'name': 'yeast mannan degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8435'},\n",
       "     {'name': 'Metabolism of folate and pterines',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-196757'},\n",
       "     {'name': 'Tetrahydrobiopterin (BH4) synthesis, recycling, salvage and regulation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-1474151'},\n",
       "     {'name': 'phenolphthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7742'},\n",
       "     {'name': 'L-arginine degradation XIV (oxidative Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6344'},\n",
       "     {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-8866907'},\n",
       "     {'name': 'carminate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8447'},\n",
       "     {'name': 'dapdiamides biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7706'},\n",
       "     {'name': '2-deoxy-D-ribose degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8060'},\n",
       "     {'name': 'dTDP-3-acetamido-3,6-dideoxy-alpha-D-glucose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7318'},\n",
       "     {'name': 'anguibactin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7986'},\n",
       "     {'name': 'chitin derivatives degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6906'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-977347'},\n",
       "     {'name': 'UDP-alpha-D-galacturonate biosynthesis I (from UDP-D-glucuronate)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4861'},\n",
       "     {'name': 'Lipid particle organization',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-8964572'},\n",
       "     {'name': 'Wax biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-9640463'},\n",
       "     {'name': 'superpathway of mycolate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6113'},\n",
       "     {'name': 'benzoyl-CoA degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1361'},\n",
       "     {'name': 'Synthesis of IP2, IP, and Ins in the cytosol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-PFA-1855183'},\n",
       "     {'name': 'Pentose phosphate pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-PFA-71336'},\n",
       "     {'name': 't-anethole biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5867'},\n",
       "     {'name': 'D-galacturonate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6486'},\n",
       "     {'name': 'chitin deacetylation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7118'},\n",
       "     {'name': 'aurachin A, B, C and D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7407'},\n",
       "     {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-77310'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-977347'},\n",
       "     {'name': 'L-tryptophan degradation XI (mammalian, via kynurenine)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6309'},\n",
       "     {'name': 'gamma-butyrobetaine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8339'},\n",
       "     {'name': 'D-xylose degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6760'},\n",
       "     {'name': 'Glutamate and glutamine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-8964539'},\n",
       "     {'name': 'glucosylglycerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7902'},\n",
       "     {'name': 'validamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5818'},\n",
       "     {'name': 'Sphingolipid de novo biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-1660661'},\n",
       "     {'name': 'stearate biosynthesis II (bacteria and plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5989'},\n",
       "     {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-BTA-70635'},\n",
       "     {'name': 'poly(ribitol phosphate) wall teichoic acid biosynthesis I (B. subtilis)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7815'},\n",
       "     {'name': 'starch degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-842'},\n",
       "     {'name': 'Role of phospholipids in phagocytosis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2029485'},\n",
       "     {'name': 'secologanin and strictosidine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5290'},\n",
       "     {'name': 'superpathway of dTDP-glucose-derived O-antigen building blocks biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7317'},\n",
       "     {'name': 'superpathway of UDP-N-acetylglucosamine-derived O-antigen building blocks biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7332'},\n",
       "     {'name': 'paxilline and diprenylpaxilline biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7493'},\n",
       "     {'name': 'malonate degradation I (biotin-independent)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5794'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7333'},\n",
       "     {'name': '4-hydroxy-4-methyl-L-glutamate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7701'},\n",
       "     {'name': 'lotaustralin degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6002'},\n",
       "     {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6053'},\n",
       "     {'name': 'The retinoid cycle in cones (daylight vision)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-2187335'},\n",
       "     {'name': 'juglone degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7825'},\n",
       "     {'name': 'Mineralocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-193993'},\n",
       "     {'name': 'RHOQ GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-XTR-9013406'},\n",
       "     {'name': 'Dimycocersyl phthiocerol biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MTU-9635470'},\n",
       "     {'name': 'Synthesis of IP2, IP, and Ins in the cytosol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-1855183'},\n",
       "     {'name': 'Sphingolipid de novo biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-1660661'},\n",
       "     {'name': 'deoxynivalenol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7712'},\n",
       "     {'name': 'ajmaline and sarpagine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5301'},\n",
       "     {'name': 'Lysine catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-71064'},\n",
       "     {'name': 'L-proline biosynthesis II (from arginine)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4981'},\n",
       "     {'name': 'CDP-tyvelose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5834'},\n",
       "     {'name': 'dTDP-alpha-D-mycaminose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7414'},\n",
       "     {'name': 'L-leucine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5076'},\n",
       "     {'name': 'matairesinol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5466'},\n",
       "     {'name': 'cucurbitacin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8434'},\n",
       "     {'name': 'Pyruvate metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-GGA-373920'},\n",
       "     {'name': 'lipoate biosynthesis and incorporation IV (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7382'},\n",
       "     {'name': 'Entner-Doudoroff pathway I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8004'},\n",
       "     {'name': 'streptovaricin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8035'},\n",
       "     {'name': 'naphthalene degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7620'},\n",
       "     {'name': 'arachidonate biosynthesis II (bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7583'},\n",
       "     {'name': 'bisphenol A degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7757'},\n",
       "     {'name': 'Phase I - Functionalization of compounds',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-211945'},\n",
       "     {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-70635'},\n",
       "     {'name': '2-O-acetyl-3-O-trans-coutarate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7460'},\n",
       "     {'name': 'ChREBP activates metabolic gene expression',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-163765'},\n",
       "     {'name': 'sphingofungin B-D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8464'},\n",
       "     {'name': 'UDP-3-acetamido-2-amino-2,3-dideoxy-alpha-D-glucopyranose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8273'},\n",
       "     {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-77350'},\n",
       "     {'name': 'abscisic acid degradation to neophaseic acid',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7640'},\n",
       "     {'name': 'pyruvate fermentation to butanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6583'},\n",
       "     {'name': 'Glucocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-194002'},\n",
       "     {'name': 'brassinolide biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2582'},\n",
       "     {'name': 'anandamide biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8051'},\n",
       "     {'name': 'RHOD GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9013405'},\n",
       "     {'name': '(-)-microperfuranone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7556'},\n",
       "     {'name': 'Phenylalanine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-8964208'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7330'},\n",
       "     {'name': \"(1'S,5'S)-averufin biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5954'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-5365859'},\n",
       "     {'name': 'Lysine catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-71064'},\n",
       "     {'name': 'cyclohexane-1-carboxyl-CoA biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8044'},\n",
       "     {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-77288'},\n",
       "     {'name': 'Respiratory electron transport',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-611105'},\n",
       "     {'name': 'fumiquinazoline D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7542'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8410'},\n",
       "     {'name': 'glycolysis II (from fructose 6-phosphate)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5484'},\n",
       "     {'name': 'reductive acetyl coenzyme A pathway II (autotrophic methanogens)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7784'},\n",
       "     {'name': 'proanthocyanidins biosynthesis from flavanols',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-641'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-70370'},\n",
       "     {'name': 'Synthesis of bile acids and bile salts via 7alpha-hydroxycholesterol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-193368'},\n",
       "     {'name': 'kanosamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5978'},\n",
       "     {'name': 'L-proline biosynthesis IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4281'},\n",
       "     {'name': 'rRNA processing in the mitochondrion',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-8868766'},\n",
       "     {'name': 'pyruvate-derived opine biosynthesis I (bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8450'},\n",
       "     {'name': 'Pyruvate metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-70268'},\n",
       "     {'name': 'Escherichia coli serotype O:127 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8231'},\n",
       "     {'name': '1D-myo-inositol hexakisphosphate biosynthesis IV (Dictyostelium)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6372'},\n",
       "     {'name': 'Synthesis of bile acids and bile salts via 24-hydroxycholesterol',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-193775'},\n",
       "     {'name': 'UDP-alpha-D-glucuronate biosynthesis (from UDP-glucose)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7346'},\n",
       "     {'name': 'teichuronic acid biosynthesis (B. subtilis 168)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7820'},\n",
       "     {'name': 'L-tyrosine degradation V (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8016'},\n",
       "     {'name': 'dTDP-beta-L-digitoxose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7657'},\n",
       "     {'name': 'ephedrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5883'},\n",
       "     {'name': 'D-apiose degradation II (to D-apionate)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8093'},\n",
       "     {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-DME-156581'},\n",
       "     {'name': '(5Z)-dodecenoate biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7858'},\n",
       "     {'name': 'Glutamate and glutamine metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-8964539'},\n",
       "     {'name': 'Fatty acyl-CoA biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-75105'},\n",
       "     {'name': 'menthol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3061'},\n",
       "     {'name': 'Lysine catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-71064'},\n",
       "     {'name': 'Biosynthesis of D-series resolvins',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-9018676'},\n",
       "     {'name': 'Formation of the active cofactor, UDP-glucuronate',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-173599'},\n",
       "     {'name': 'toluene degradation to benzoyl-CoA (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-81'},\n",
       "     {'name': 'acetylaszonalenin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7532'},\n",
       "     {'name': 'L-canavanine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8419'},\n",
       "     {'name': 'methylaspartate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6728'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7338'},\n",
       "     {'name': 'D-arginine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6422'},\n",
       "     {'name': 'D-xylose degradation to ethylene glycol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7178'},\n",
       "     {'name': 'tropane alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5317'},\n",
       "     {'name': 'pinosylvin metabolism',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5045'},\n",
       "     {'name': '2-chloroacrylate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7425'},\n",
       "     {'name': 'cardenolide glucosides biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6036'},\n",
       "     {'name': 'acetaldehyde biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6333'},\n",
       "     {'name': 'penigequinolone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8389'},\n",
       "     {'name': '2-hydroxyparaconate and itatartarate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8019'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-XTR-77346'},\n",
       "     {'name': 'Mineralocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-193993'},\n",
       "     {'name': 'D-fructuronate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7242'},\n",
       "     {'name': 'nocardicin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7797'},\n",
       "     {'name': 'Escherichia coli serotype O:107 O antigen biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8219'},\n",
       "     {'name': 'fatty acid salvage',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7094'},\n",
       "     {'name': 'roquefortine C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7605'},\n",
       "     {'name': 'saframycin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7671'},\n",
       "     {'name': 'guadinomine B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7693'},\n",
       "     {'name': 'Mineralocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-193993'},\n",
       "     {'name': 'Estrogen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-193144'},\n",
       "     {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-77288'},\n",
       "     {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-2162123'},\n",
       "     {'name': 'emodin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8166'},\n",
       "     {'name': 'Galactose catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-70370'},\n",
       "     {'name': 'propane degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7774'},\n",
       "     {'name': 'icosapentaenoate biosynthesis III (8-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7724'},\n",
       "     {'name': 'RHOD GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-9013405'},\n",
       "     {'name': 'kauralexin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6887'},\n",
       "     {'name': 'aloesone and aloesin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-4801'},\n",
       "     {'name': 'acinetoferrin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7989'},\n",
       "     {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6545'},\n",
       "     {'name': 'rosamicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7677'},\n",
       "     {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77288'},\n",
       "     {'name': 'anthocyanidin 3-malylglucoside biosynthesis (acyl-glucose dependent)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7261'},\n",
       "     {'name': 'Beta-oxidation of pristanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-389887'},\n",
       "     {'name': 'Cytoprotection by HMOX1',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-9707564'},\n",
       "     {'name': 'Wax biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-9640463'},\n",
       "     {'name': 'phenylethanol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5751'},\n",
       "     {'name': 'Wax biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9640463'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-4641265'},\n",
       "     {'name': 'dimethyl sulfoxide degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6050'},\n",
       "     {'name': 'Beta-oxidation of very long chain fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-390247'},\n",
       "     {'name': 'Mineralocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CFA-193993'},\n",
       "     {'name': 'gossypol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5773'},\n",
       "     {'name': 'wax esters biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5884'},\n",
       "     {'name': 'Complex I biogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-6799198'},\n",
       "     {'name': 'sch210971 and sch210972 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7702'},\n",
       "     {'name': '6-methoxypodophyllotoxin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5479'},\n",
       "     {'name': 'isopimaric acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5422'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77346'},\n",
       "     {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8132'},\n",
       "     {'name': 'aromatic biogenic amine degradation (bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7431'},\n",
       "     {'name': 'apigeninidin 5-O-glucoside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7253'},\n",
       "     {'name': 'Estrogen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-193144'},\n",
       "     {'name': 'Synthesis of Leukotrienes (LT) and Eoxins (EX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2142691'},\n",
       "     {'name': 'chlorophyll a biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5064'},\n",
       "     {'name': 'naringenin C-glucosylation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6602'},\n",
       "     {'name': 'CDC42 GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9013148'},\n",
       "     {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-77350'},\n",
       "     {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-MMU-70635'},\n",
       "     {'name': 'TCA cycle II (plants and fungi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5690'},\n",
       "     {'name': 'The canonical retinoid cycle in rods (twilight vision)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-2453902'},\n",
       "     {'name': 'superpathway of glucose and xylose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6901'},\n",
       "     {'name': 'juniperonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7619'},\n",
       "     {'name': 'dhurrin degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5976'},\n",
       "     {'name': 'erythritol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8373'},\n",
       "     {'name': '2-methyl-branched fatty acid beta-oxidation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8181'},\n",
       "     {'name': 'dimethylsulfoniopropanoate degradation II (cleavage)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6056'},\n",
       "     {'name': '3-dehydroquinate biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6164'},\n",
       "     {'name': 'strychnine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8216'},\n",
       "     {'name': 'cyanidin dimalonylglucoside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7452'},\n",
       "     {'name': 'Fructose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-5652227'},\n",
       "     {'name': 'NR1H2 & NR1H3 regulate gene expression linked to lipogenesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9029558'},\n",
       "     {'name': 'glutaryl-CoA degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5177'},\n",
       "     {'name': 'Vitamin B5 (pantothenate) metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-199220'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-5365859'},\n",
       "     {'name': 'TP53 Regulates Metabolic Genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-5628897'},\n",
       "     {'name': 'naphthomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8038'},\n",
       "     {'name': 'gamma-butyrobetaine degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8337'},\n",
       "     {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-BTA-70171'},\n",
       "     {'name': 'thiocoraline biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7737'},\n",
       "     {'name': 'L-alanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8188'},\n",
       "     {'name': 'Androgen biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-193048'},\n",
       "     {'name': 'saponin biosynthesis III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5759'},\n",
       "     {'name': 'violdelphin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7458'},\n",
       "     {'name': 'tetrahydrofolate biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6614'},\n",
       "     {'name': 'fusaric acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7672'},\n",
       "     {'name': 'bile acids 7-O epimerization',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8254'},\n",
       "     {'name': 'Cholesterol biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-191273'},\n",
       "     {'name': 'CMP-N-acetyl-7-O-acetylneuraminate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7529'},\n",
       "     {'name': 'daidzin and daidzein degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6996'},\n",
       "     {'name': 'pyruvate fermentation to ethanol II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5486'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-77346'},\n",
       "     {'name': 'Sensory processing of sound by inner hair cells of the cochlea',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9662360'},\n",
       "     {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-2162123'},\n",
       "     {'name': 'poly-hydroxy fatty acids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6710'},\n",
       "     {'name': 'CDP-paratose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5832'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-77346'},\n",
       "     {'name': 'sulfoacetate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6593'},\n",
       "     {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-2162123'},\n",
       "     {'name': 'alkane oxidation', 'databaseName': 'MetaCyc', 'id': 'PWY-2724'},\n",
       "     {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-DDI-70635'},\n",
       "     {'name': 'rubber biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5815'},\n",
       "     {'name': 'clavulanate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5679'},\n",
       "     {'name': 'RHOJ GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9013409'},\n",
       "     {'name': 'The canonical retinoid cycle in rods (twilight vision)',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-2453902'},\n",
       "     {'name': 'L-arginine degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8187'},\n",
       "     {'name': 'archaeosine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7923'},\n",
       "     {'name': 'Respiratory electron transport',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-611105'},\n",
       "     {'name': 'dimycocerosyl phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7744'},\n",
       "     {'name': 'phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7741'},\n",
       "     {'name': 'Threonine catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-8849175'},\n",
       "     {'name': 'Branched-chain amino acid catabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-70895'},\n",
       "     {'name': 'sulfolactate degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6616'},\n",
       "     {'name': 'myo-inositol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7241'},\n",
       "     {'name': 'galactolipid biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7666'},\n",
       "     {'name': 'Abacavir metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-2161541'},\n",
       "     {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-77350'},\n",
       "     {'name': '(Kdo)2-lipid A biosynthesis (P. putida)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8075'},\n",
       "     {'name': 'tunicamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7821'},\n",
       "     {'name': 'Beta oxidation of myristoyl-CoA to lauroyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-77285'},\n",
       "     {'name': 'dTDP-3-acetamido-alpha-D-fucose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6953'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-galactosaminuronate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7336'},\n",
       "     {'name': 'Pentose phosphate pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SCE-71336'},\n",
       "     {'name': 'GDP-fucose biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-6787639'},\n",
       "     {'name': 'VEGFR2 mediated cell proliferation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5218921'},\n",
       "     {'name': 'aminopropanol phosphate biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7378'},\n",
       "     {'name': 'cutin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-321'},\n",
       "     {'name': 'petrobactin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6289'},\n",
       "     {'name': 'kappa-carrageenan degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6821'},\n",
       "     {'name': 'RA biosynthesis pathway',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-5365859'},\n",
       "     {'name': 'pyruvate fermentation to ethanol III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6587'},\n",
       "     {'name': 'fatty acid beta-oxidation V (unsaturated, odd number, di-isomerase-dependent)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6837'},\n",
       "     {'name': 'chelerythrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7507'},\n",
       "     {'name': '1,4-dimethylbenzene degradation to 4-methylbenzoate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5429'},\n",
       "     {'name': 'DAG and IP3 signaling',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-1489509'},\n",
       "     {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77350'},\n",
       "     {'name': 'D-xylose degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8330'},\n",
       "     {'name': 'sangivamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6721'},\n",
       "     {'name': 'D-altritol and galactitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7862'},\n",
       "     {'name': 'bombykol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7423'},\n",
       "     {'name': 'lampranthin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7265'},\n",
       "     {'name': 'asperlicin E biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7549'},\n",
       "     {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DDI-77346'},\n",
       "     {'name': 'formaldehyde oxidation VII (THF pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7909'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7331'},\n",
       "     {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-77310'},\n",
       "     {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-8866904'},\n",
       "     {'name': 'Sulfur amino acid metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SPO-1614635'},\n",
       "     {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-8866904'},\n",
       "     {'name': 'RHOQ GTPase cycle',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-9013406'},\n",
       "     {'name': 'L-rhamnose degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6714'},\n",
       "     {'name': 'Metabolism of folate and pterines',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-196757'},\n",
       "     {'name': '1,2,4-trichlorobenzene degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6091'},\n",
       "     {'name': 'type IV lipoteichoic acid biosynthesis (S. pneumoniae)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7818'},\n",
       "     {'name': 'pyruvate fermentation to (R)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8274'},\n",
       "     {'name': 'Pyruvate metabolism',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-70268'},\n",
       "     {'name': 'Glyoxylate metabolism and glycine degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-389661'},\n",
       "     {'name': 'nystatin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7624'},\n",
       "     {'name': 'mycolyl-arabinogalactan-peptidoglycan complex biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6397'},\n",
       "     {'name': 'Essential pentosuria',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5662853'},\n",
       "     {'name': 'D-galactose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6693'},\n",
       "     {'name': 'arachidonate biosynthesis IV (8-detaturase, lower eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7601'},\n",
       "     {'name': 'grixazone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7153'},\n",
       "     {'name': 'Beta-oxidation of very long chain fatty acids',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-390247'},\n",
       "     {'name': 'Glucocorticoid biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-194002'},\n",
       "     ...]}},\n",
       "  'locations': [{'start': 150,\n",
       "    'end': 336,\n",
       "    'representative': False,\n",
       "    'hmmLength': 188,\n",
       "    'location-fragments': [{'start': 150,\n",
       "      'end': 336,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'evalue': 1.05e-50,\n",
       "  'model-ac': '0053195'},\n",
       " {'signature': {'accession': 'PS00065',\n",
       "   'name': 'D_2_HYDROXYACID_DH_1',\n",
       "   'description': 'D-isomer specific 2-hydroxyacid dehydrogenases NAD-binding signature.',\n",
       "   'signatureLibraryRelease': {'library': 'PROSITE_PATTERNS',\n",
       "    'version': '2023_05'},\n",
       "   'entry': {'accession': 'IPR029752',\n",
       "    'name': 'D-isomer_DH_CS1',\n",
       "    'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain conserved site 1',\n",
       "    'type': 'CONSERVED_SITE',\n",
       "    'goXRefs': [],\n",
       "    'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8238'},\n",
       "     {'name': 'zwittermicin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7694'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-XTR-983168'},\n",
       "     {'name': '(-)-dehydrodiconiferyl alcohol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7557'},\n",
       "     {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7331'},\n",
       "     {'name': 'anaerobic energy metabolism (invertebrates, cytosol)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7383'},\n",
       "     {'name': 'stearate biosynthesis I (animals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5972'},\n",
       "     {'name': 'autoinducer CAI-1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7621'},\n",
       "     {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6973'},\n",
       "     {'name': 'septacidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8366'},\n",
       "     {'name': 'bisphenol A degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7757'},\n",
       "     {'name': 'pyruvate fermentation to (R)-lactate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8274'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-983168'},\n",
       "     {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7610'},\n",
       "     {'name': 'photorespiration III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8363'},\n",
       "     {'name': 'salidroside biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6802'},\n",
       "     {'name': 'sphingofungin B-D biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8464'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3899300'},\n",
       "     {'name': 'butanol and isobutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7396'},\n",
       "     {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5652'},\n",
       "     {'name': 'pyruvate fermentation to butanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6583'},\n",
       "     {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7340'},\n",
       "     {'name': 'D-galactose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6693'},\n",
       "     {'name': 'sitosterol degradation to androstenedione',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6948'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-977347'},\n",
       "     {'name': '1,2-propanediol biosynthesis from lactate (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7541'},\n",
       "     {'name': '(S)-camphor degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6989'},\n",
       "     {'name': 'pyruvate fermentation to ethanol I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5480'},\n",
       "     {'name': 'meso-butanediol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6392'},\n",
       "     {'name': 'daunorubicin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7352'},\n",
       "     {'name': 'oxalate degradation VI',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7985'},\n",
       "     {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8010'},\n",
       "     {'name': 'L-isoleucine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5101'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8410'},\n",
       "     {'name': 'kanosamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5978'},\n",
       "     {'name': 'pyrethrin I biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6926'},\n",
       "     {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5882'},\n",
       "     {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7301'},\n",
       "     {'name': 'reductive TCA cycle II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5392'},\n",
       "     {'name': 'phosphinothricin tripeptide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6322'},\n",
       "     {'name': 'sulfolactate degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6637'},\n",
       "     {'name': 'L-tyrosine degradation V (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8016'},\n",
       "     {'name': 'dTDP-beta-L-digitoxose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7657'},\n",
       "     {'name': 'ephedrine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5883'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-983168'},\n",
       "     {'name': 'rosmarinic acid biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5049'},\n",
       "     {'name': 'oryzalexin A, B, and C biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7489'},\n",
       "     {'name': 'L-galactose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7568'},\n",
       "     {'name': 'Signaling by TCF7L2 mutants',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-5339700'},\n",
       "     {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7606'},\n",
       "     {'name': 'hygromycin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8367'},\n",
       "     {'name': 'L-ascorbate degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6704'},\n",
       "     {'name': 'dTDP-beta-L-mycarose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6976'},\n",
       "     {'name': 'superpathway of nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7342'},\n",
       "     {'name': 'terrequinone A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7520'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3769402'},\n",
       "     {'name': 'CDP-6-deoxy-D-gulose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8139'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-4641265'},\n",
       "     {'name': 'L-homomethionine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1186'},\n",
       "     {'name': 'pyruvate fermentation to isobutanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7111'},\n",
       "     {'name': 'L-canavanine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8419'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7338'},\n",
       "     {'name': 'methylaspartate cycle',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6728'},\n",
       "     {'name': 'D-xylose degradation to ethylene glycol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7178'},\n",
       "     {'name': 'tropane alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5317'},\n",
       "     {'name': 'TCA cycle V (2-oxoglutarate synthase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6969'},\n",
       "     {'name': 'acetaldehyde biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6333'},\n",
       "     {'name': 'oxalate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6696'},\n",
       "     {'name': 'CDP-ascarylose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5830'},\n",
       "     {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7615'},\n",
       "     {'name': '3-sulfopropanediol degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6634'},\n",
       "     {'name': 'traumatin and (Z)-3-hexen-1-yl acetate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5410'},\n",
       "     {'name': 'noradrenaline and adrenaline degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6342'},\n",
       "     {'name': 'nicotine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5316'},\n",
       "     {'name': 'L-valine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5057'},\n",
       "     {'name': 'vancomycin resistance I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6454'},\n",
       "     {'name': '3-sulfopropanediol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8214'},\n",
       "     {'name': 'tetramethylpyrazine degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7403'},\n",
       "     {'name': 'pinitol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6738'},\n",
       "     {'name': 'guadinomine B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7693'},\n",
       "     {'name': '6-diazo-5-oxo-L-norleucine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8297'},\n",
       "     {'name': 'detoxification of reactive carbonyls in chloroplasts',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6786'},\n",
       "     {'name': 'isethionate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2601'},\n",
       "     {'name': '1,3-propanediol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7385'},\n",
       "     {'name': 'L-tryptophan degradation V (side chain pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3162'},\n",
       "     {'name': 'arabidopyrone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7682'},\n",
       "     {'name': 'dTDP-beta-L-olivose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6974'},\n",
       "     {'name': 'sporopollenin precursors biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6733'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-3899300'},\n",
       "     {'name': 'SUMOylation of transcription cofactors',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3899300'},\n",
       "     {'name': 'propane degradation I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7774'},\n",
       "     {'name': 'formate oxidation to CO2',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1881'},\n",
       "     {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7337'},\n",
       "     {'name': 'androstenedione degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8152'},\n",
       "     {'name': 'rosmarinic acid biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5048'},\n",
       "     {'name': 'mupirocin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8012'},\n",
       "     {'name': 'antimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8374'},\n",
       "     {'name': 'erinacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8408'},\n",
       "     {'name': 'jasmonic acid biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-735'},\n",
       "     {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7300'},\n",
       "     {'name': 'pyruvate fermentation to hexanol (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6863'},\n",
       "     {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7115'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-983168'},\n",
       "     {'name': 'L-threitol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7787'},\n",
       "     {'name': 'suberin monomers biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1121'},\n",
       "     {'name': 'cis-geranyl-CoA degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6672'},\n",
       "     {'name': 'representative whole plasma arachidonate-derived icosanoids II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8411'},\n",
       "     {'name': 'aurofusarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7695'},\n",
       "     {'name': 'bile acid biosynthesis, neutral pathway',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6061'},\n",
       "     {'name': 'phenylethanol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5751'},\n",
       "     {'name': 'mithramycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7045'},\n",
       "     {'name': 'arachidonate metabolites biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8397'},\n",
       "     {'name': 'kanamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7000'},\n",
       "     {'name': 'aclacinomycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7354'},\n",
       "     {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6946'},\n",
       "     {'name': 'serotonin degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6313'},\n",
       "     {'name': 'heme d1 biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7554'},\n",
       "     {'name': 'methyl tert-butyl ether degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7779'},\n",
       "     {'name': 'fusicoccin A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6659'},\n",
       "     {'name': '(Z)-phenylmethanethial S-oxide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6539'},\n",
       "     {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6318'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-BTA-983168'},\n",
       "     {'name': 'myxochelin A and B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8383'},\n",
       "     {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8132'},\n",
       "     {'name': 'purine nucleobases degradation II (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5497'},\n",
       "     {'name': 'L-isoleucine degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5078'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-977347'},\n",
       "     {'name': '11-cis-3-hydroxyretinal biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7043'},\n",
       "     {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8014'},\n",
       "     {'name': 'TCA cycle II (plants and fungi)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5690'},\n",
       "     {'name': 'leporin B biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8370'},\n",
       "     {'name': 'superpathway of glucose and xylose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6901'},\n",
       "     {'name': 'photorespiration II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8362'},\n",
       "     {'name': 'abscisic acid degradation to phaseic acid',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5271'},\n",
       "     {'name': 'L-glucose degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7130'},\n",
       "     {'name': 'erythritol biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8373'},\n",
       "     {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7304'},\n",
       "     {'name': 'bikaverin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7692'},\n",
       "     {'name': '6-methylpretetramide biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7811'},\n",
       "     {'name': '3-methylbutanol biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6871'},\n",
       "     {'name': 'phytosterol biosynthesis (plants)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-2541'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-3769402'},\n",
       "     {'name': 'photorespiration I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-181'},\n",
       "     {'name': '6-gingerol analog biosynthesis (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6920'},\n",
       "     {'name': 'dihydroconiferyl alcohol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6302'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DRE-983168'},\n",
       "     {'name': 'phosalacine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7769'},\n",
       "     {'name': 'phenolphthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7742'},\n",
       "     {'name': 'mycolate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWYG-321'},\n",
       "     {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7613'},\n",
       "     {'name': 'L-alanine degradation VI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8188'},\n",
       "     {'name': 'propane degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7775'},\n",
       "     {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8086'},\n",
       "     {'name': 'monolignol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-361'},\n",
       "     {'name': 'L-arabinose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7295'},\n",
       "     {'name': 'apicidin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7667'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-977347'},\n",
       "     {'name': '8-O-methylfusarubin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7689'},\n",
       "     {'name': 'pyruvate fermentation to ethanol II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5486'},\n",
       "     {'name': '4-ethylphenol degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6080'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-3769402'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-RNO-4641265'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-DME-983168'},\n",
       "     {'name': 't-anethole biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5867'},\n",
       "     {'name': 'the visual cycle (insects)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7042'},\n",
       "     {'name': '2-deoxy-D-ribose degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8058'},\n",
       "     {'name': 'D-xylose degradation IV',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7294'},\n",
       "     {'name': 'quinoxaline-2-carboxylate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7734'},\n",
       "     {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3722'},\n",
       "     {'name': 'sulfolactate degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6638'},\n",
       "     {'name': '3-hydroxy-L-homotyrosine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7649'},\n",
       "     {'name': 'mannojirimycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7531'},\n",
       "     {'name': 'cinchona alkaloids biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5848'},\n",
       "     {'name': 'L-glutamate degradation XI (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8190'},\n",
       "     {'name': 'dTDP-L-daunosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7814'},\n",
       "     {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5754'},\n",
       "     {'name': 'p-cymene degradation to p-cumate',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-741'},\n",
       "     {'name': 'limonene degradation IV (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8029'},\n",
       "     {'name': 'Serine biosynthesis',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-SSC-977347'},\n",
       "     {'name': 'salinosporamide A biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6627'},\n",
       "     {'name': 'erythritol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7788'},\n",
       "     {'name': 'dTDP-beta-L-megosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7104'},\n",
       "     {'name': '2-keto-L-gulonate biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5782'},\n",
       "     {'name': 'UDP-beta-L-rhamnose biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-3261'},\n",
       "     {'name': 'androstenedione degradation I (aerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6944'},\n",
       "     {'name': 'validamycin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5818'},\n",
       "     {'name': 'L-phenylalanine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5079'},\n",
       "     {'name': 'L-methionine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5082'},\n",
       "     {'name': 'phthiocerol biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7741'},\n",
       "     {'name': 'superpathway of glyoxylate cycle and fatty acid degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-561'},\n",
       "     {'name': 'myo-inositol degradation II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7241'},\n",
       "     {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6945'},\n",
       "     {'name': 'testosterone degradation (anaerobic)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8155'},\n",
       "     {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7726'},\n",
       "     {'name': 'alpha-solanine/alpha-chaconine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5666'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-4641265'},\n",
       "     {'name': 'Deactivation of the beta-catenin transactivating complex',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-CEL-3769402'},\n",
       "     {'name': 'dehydrophos biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6682'},\n",
       "     {'name': 'pyruvate fermentation to butanol II (engineered)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6883'},\n",
       "     {'name': '(S)-propane-1,2-diol degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7013'},\n",
       "     {'name': 'Antigen processing: Ubiquitination & Proteasome degradation',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-HSA-983168'},\n",
       "     {'name': 'Repression of WNT target genes',\n",
       "      'databaseName': 'Reactome',\n",
       "      'id': 'R-MMU-4641265'},\n",
       "     {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7333'},\n",
       "     {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6053'},\n",
       "     {'name': 'juglone degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7825'},\n",
       "     {'name': '4-oxopentanoate degradation',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7948'},\n",
       "     {'name': '(2R)-methoxymalonyl-[acp] biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8426'},\n",
       "     {'name': 'patulin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7490'},\n",
       "     {'name': 'cyathin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8409'},\n",
       "     {'name': 'cutin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-321'},\n",
       "     {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7288'},\n",
       "     {'name': 'ecdysteroid metabolism (arthropods)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7321'},\n",
       "     {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7440'},\n",
       "     {'name': 'pyruvate fermentation to ethanol III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6587'},\n",
       "     {'name': 'meso-butanediol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6391'},\n",
       "     {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7339'},\n",
       "     {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\",\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-6626'},\n",
       "     {'name': 'prodigiosin biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7547'},\n",
       "     {'name': 'erythritol biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8372'},\n",
       "     {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8017'},\n",
       "     {'name': 'tryptoquialanine biosynthesis',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7660'},\n",
       "     {'name': 'formaldehyde assimilation I (serine pathway)',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-1622'},\n",
       "     {'name': 'L-serine biosynthesis II',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-8011'},\n",
       "     {'name': 'L-leucine degradation III',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-5076'},\n",
       "     {'name': 'paromamine biosynthesis I',\n",
       "      'databaseName': 'MetaCyc',\n",
       "      'id': 'PWY-7014'}]}},\n",
       "  'locations': [{'start': 195,\n",
       "    'end': 222,\n",
       "    'representative': False,\n",
       "    'level': 'STRONG',\n",
       "    'cigarAlignment': '5M1I4M13I2M1I2M',\n",
       "    'alignment': 'VGTIAaGRIGlavlrrlapfdvhLHyTD',\n",
       "    'location-fragments': [{'start': 195,\n",
       "      'end': 222,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'model-ac': 'PS00065'},\n",
       " {'signature': {'accession': 'G3DSA:3.40.50.720:FF:000057',\n",
       "   'name': None,\n",
       "   'description': 'Formate dehydrogenase',\n",
       "   'signatureLibraryRelease': {'library': 'FUNFAM', 'version': '4.3.0'},\n",
       "   'entry': None},\n",
       "  'locations': [{'start': 147,\n",
       "    'end': 337,\n",
       "    'representative': False,\n",
       "    'hmmStart': 1,\n",
       "    'hmmEnd': 197,\n",
       "    'hmmLength': 197,\n",
       "    'hmmBounds': 'COMPLETE',\n",
       "    'evalue': 2.9e-87,\n",
       "    'score': 293.3,\n",
       "    'envelopeStart': 147,\n",
       "    'envelopeEnd': 337,\n",
       "    'postProcessed': False,\n",
       "    'location-fragments': [{'start': 147,\n",
       "      'end': 337,\n",
       "      'dc-status': 'CONTINUOUS'}]}],\n",
       "  'evalue': 2.2e-87,\n",
       "  'score': 293.7,\n",
       "  'model-ac': '3.40.50.720-FF-000057'}]"
      ]
     },
     "execution_count": 83,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['results'][0]['matches']#[1]['signature']#['accession']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "value = data['results']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Considerations\n",
    "- Not all signatures have the the same fields like signature -> entry -> type (Useful for filtering the Family options)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Functions for exploring json files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "interproscan-version 5.66-98.0\n",
      "results [{'sequence': 'MAKVLCVLYDDPVDGYPKTYARDDLPKIDHYPGGQTLPTPKAIDFTPGQLLGSVSGELGLRKYLESNGHTLVVTSDKDGPDSVFERELVDADVVISQPFWPAYLTPERIAKAKNLKLALTAGIGSDHVDLQSAIDRNVTVAEVTYCNSISVAEHVVMMILSLVRNYLPSHEWARKGGWNIADCVSHAYDLEAMHVGTIAAGRIGLAVLRRLAPFDVHLHYTDRHRLPESVEKELNLTWHATREDMYPVCDVVTLNIPLHSPTEHMINDETLKLFKRGAYIVNTARGKLCDRDAVARALESGRLAGYAGDVWFPQPAPKDHPWRTMPYNGMTPHISGTTLTAQARYAAGTREILECFFEGRPIRDEYLIVQGGALAGTGAHNYFKGNATGGSEEAAKFKKAV', 'md5': '5bd102e2f52d0ec7d80425db2e1ffb6b', 'matches': [{'signature': {'accession': 'G3DSA:3.40.50.720', 'name': '', 'description': None, 'signatureLibraryRelease': {'library': 'GENE3D', 'version': '4.3.0'}, 'entry': None}, 'locations': [{'start': 2, 'end': 382, 'representative': False, 'hmmStart': 1, 'hmmEnd': 381, 'hmmLength': 210, 'hmmBounds': 'N_TERMINAL_COMPLETE', 'evalue': 0.0, 'score': 750.9, 'envelopeStart': 2, 'envelopeEnd': 384, 'postProcessed': True, 'location-fragments': [{'start': 2, 'end': 146, 'dc-status': 'C_TERMINAL_DISC'}, {'start': 339, 'end': 382, 'dc-status': 'N_TERMINAL_DISC'}]}], 'evalue': 0.0, 'score': 751.1, 'model-ac': '2gsdA01'}, {'signature': {'accession': 'PS00670', 'name': 'D_2_HYDROXYACID_DH_2', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenases signature 2.', 'signatureLibraryRelease': {'library': 'PROSITE_PATTERNS', 'version': '2023_05'}, 'entry': {'accession': 'IPR029753', 'name': 'D-isomer_DH_CS', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain conserved site', 'type': 'CONSERVED_SITE', 'goXRefs': [], 'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8238'}, {'name': 'zwittermicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7694'}, {'name': '(-)-dehydrodiconiferyl alcohol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7557'}, {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7331'}, {'name': 'anaerobic energy metabolism (invertebrates, cytosol)', 'databaseName': 'MetaCyc', 'id': 'PWY-7383'}, {'name': 'stearate biosynthesis I (animals)', 'databaseName': 'MetaCyc', 'id': 'PWY-5972'}, {'name': 'autoinducer CAI-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7621'}, {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6973'}, {'name': 'septacidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8366'}, {'name': 'bisphenol A degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7757'}, {'name': 'pyruvate fermentation to (R)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-8274'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-389661'}, {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7610'}, {'name': 'photorespiration III', 'databaseName': 'MetaCyc', 'id': 'PWY-8363'}, {'name': 'sphingofungin B-D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8464'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-MMU-3899300'}, {'name': 'butanol and isobutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7396'}, {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA', 'databaseName': 'MetaCyc', 'id': 'PWY-5652'}, {'name': 'pyruvate fermentation to butanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-6583'}, {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7340'}, {'name': 'D-galactose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6693'}, {'name': 'sitosterol degradation to androstenedione', 'databaseName': 'MetaCyc', 'id': 'PWY-6948'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-977347'}, {'name': '1,2-propanediol biosynthesis from lactate (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7541'}, {'name': '(S)-camphor degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6989'}, {'name': 'meso-butanediol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6392'}, {'name': 'daunorubicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7352'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}, {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-8010'}, {'name': 'L-isoleucine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5101'}, {'name': 'representative whole plasma arachidonate-derived icosanoids I', 'databaseName': 'MetaCyc', 'id': 'PWY-8410'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-389661'}, {'name': 'kanosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5978'}, {'name': 'pyrethrin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6926'}, {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5882'}, {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7301'}, {'name': 'reductive TCA cycle II', 'databaseName': 'MetaCyc', 'id': 'PWY-5392'}, {'name': 'phosphinothricin tripeptide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6322'}, {'name': 'sulfolactate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6637'}, {'name': 'L-tyrosine degradation V (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8016'}, {'name': 'dTDP-beta-L-digitoxose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7657'}, {'name': 'ephedrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5883'}, {'name': 'oryzalexin A, B, and C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7489'}, {'name': 'L-galactose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7568'}, {'name': 'Signaling by TCF7L2 mutants', 'databaseName': 'Reactome', 'id': 'R-HSA-5339700'}, {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7606'}, {'name': 'hygromycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8367'}, {'name': 'L-ascorbate degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6704'}, {'name': 'dTDP-beta-L-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6976'}, {'name': 'superpathway of nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7342'}, {'name': 'terrequinone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7520'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-HSA-3769402'}, {'name': 'CDP-6-deoxy-D-gulose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8139'}, {'name': 'L-homomethionine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1186'}, {'name': 'L-canavanine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8419'}, {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7338'}, {'name': 'methylaspartate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6728'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-DME-3899300'}, {'name': 'tropane alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5317'}, {'name': 'TCA cycle V (2-oxoglutarate synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6969'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'CDP-ascarylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5830'}, {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)', 'databaseName': 'MetaCyc', 'id': 'PWY-7615'}, {'name': '3-sulfopropanediol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6634'}, {'name': 'nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5316'}, {'name': 'vancomycin resistance I', 'databaseName': 'MetaCyc', 'id': 'PWY-6454'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-BTA-4641265'}, {'name': '3-sulfopropanediol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8214'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-DME-3769402'}, {'name': 'tetramethylpyrazine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7403'}, {'name': 'pinitol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6738'}, {'name': 'guadinomine B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7693'}, {'name': '6-diazo-5-oxo-L-norleucine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8297'}, {'name': 'detoxification of reactive carbonyls in chloroplasts', 'databaseName': 'MetaCyc', 'id': 'PWY-6786'}, {'name': 'isethionate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-2601'}, {'name': '1,3-propanediol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7385'}, {'name': 'L-tryptophan degradation V (side chain pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-3162'}, {'name': 'arabidopyrone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7682'}, {'name': 'dTDP-beta-L-olivose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6974'}, {'name': 'sporopollenin precursors biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6733'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-HSA-3899300'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-RNO-3899300'}, {'name': 'propane degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7774'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7337'}, {'name': 'androstenedione degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8152'}, {'name': 'mupirocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8012'}, {'name': 'antimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8374'}, {'name': 'erinacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8408'}, {'name': 'jasmonic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-735'}, {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7300'}, {'name': 'pyruvate fermentation to hexanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6863'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-7115'}, {'name': 'L-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7787'}, {'name': 'suberin monomers biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1121'}, {'name': 'cis-geranyl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6672'}, {'name': 'representative whole plasma arachidonate-derived icosanoids II', 'databaseName': 'MetaCyc', 'id': 'PWY-8411'}, {'name': 'aurofusarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7695'}, {'name': 'bile acid biosynthesis, neutral pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6061'}, {'name': 'phenylethanol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5751'}, {'name': 'mithramycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7045'}, {'name': 'arachidonate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8397'}, {'name': 'aclacinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7354'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-DME-4641265'}, {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6946'}, {'name': 'heme d1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7554'}, {'name': 'methyl tert-butyl ether degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7779'}, {'name': 'fusicoccin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6659'}, {'name': '(Z)-phenylmethanethial S-oxide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6539'}, {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)', 'databaseName': 'MetaCyc', 'id': 'PWY-6318'}, {'name': 'myxochelin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8383'}, {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8132'}, {'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-977347'}, {'name': '11-cis-3-hydroxyretinal biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7043'}, {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8014'}, {'name': 'TCA cycle II (plants and fungi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5690'}, {'name': 'leporin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8370'}, {'name': 'superpathway of glucose and xylose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6901'}, {'name': 'photorespiration II', 'databaseName': 'MetaCyc', 'id': 'PWY-8362'}, {'name': 'abscisic acid degradation to phaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-5271'}, {'name': 'L-glucose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7130'}, {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7304'}, {'name': 'bikaverin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7692'}, {'name': '6-methylpretetramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7811'}, {'name': 'phytosterol biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2541'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-RNO-3769402'}, {'name': 'photorespiration I', 'databaseName': 'MetaCyc', 'id': 'PWY-181'}, {'name': '6-gingerol analog biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6920'}, {'name': 'dihydroconiferyl alcohol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6302'}, {'name': 'phosalacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7769'}, {'name': 'phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7742'}, {'name': 'mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWYG-321'}, {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7613'}, {'name': 'L-alanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8188'}, {'name': 'propane degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7775'}, {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen', 'databaseName': 'MetaCyc', 'id': 'PWY-8086'}, {'name': 'L-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7295'}, {'name': 'apicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7667'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-977347'}, {'name': '8-O-methylfusarubin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7689'}, {'name': '4-ethylphenol degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6080'}, {'name': 'Sensory processing of sound by inner hair cells of the cochlea', 'databaseName': 'Reactome', 'id': 'R-HSA-9662360'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-MMU-3769402'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-RNO-4641265'}, {'name': 't-anethole biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5867'}, {'name': 'the visual cycle (insects)', 'databaseName': 'MetaCyc', 'id': 'PWY-7042'}, {'name': '2-deoxy-D-ribose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8058'}, {'name': 'D-xylose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7294'}, {'name': 'quinoxaline-2-carboxylate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7734'}, {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-3722'}, {'name': 'sulfolactate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6638'}, {'name': '3-hydroxy-L-homotyrosine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7649'}, {'name': 'mannojirimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7531'}, {'name': 'cinchona alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5848'}, {'name': 'L-glutamate degradation XI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8190'}, {'name': 'dTDP-L-daunosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7814'}, {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5754'}, {'name': 'p-cymene degradation to p-cumate', 'databaseName': 'MetaCyc', 'id': 'PWY-741'}, {'name': 'limonene degradation IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8029'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-977347'}, {'name': 'salinosporamide A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6627'}, {'name': 'erythritol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7788'}, {'name': 'dTDP-beta-L-megosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7104'}, {'name': '2-keto-L-gulonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5782'}, {'name': 'UDP-beta-L-rhamnose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3261'}, {'name': 'androstenedione degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6944'}, {'name': 'validamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5818'}, {'name': 'phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7741'}, {'name': 'superpathway of glyoxylate cycle and fatty acid degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-561'}, {'name': 'sulfolactate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6616'}, {'name': 'myo-inositol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7241'}, {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6945'}, {'name': 'testosterone degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8155'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7726'}, {'name': 'alpha-solanine/alpha-chaconine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5666'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-HSA-4641265'}, {'name': 'dehydrophos biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6682'}, {'name': 'pyruvate fermentation to butanol II (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6883'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-MMU-4641265'}, {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7333'}, {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)', 'databaseName': 'MetaCyc', 'id': 'PWY-6053'}, {'name': 'juglone degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7825'}, {'name': '4-oxopentanoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7948'}, {'name': '(2R)-methoxymalonyl-[acp] biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8426'}, {'name': 'patulin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7490'}, {'name': 'cyathin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8409'}, {'name': 'cutin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-321'}, {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-7288'}, {'name': 'ecdysteroid metabolism (arthropods)', 'databaseName': 'MetaCyc', 'id': 'PWY-7321'}, {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7440'}, {'name': 'meso-butanediol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6391'}, {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7339'}, {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6626'}, {'name': 'prodigiosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7547'}, {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8017'}, {'name': 'tryptoquialanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7660'}, {'name': 'formaldehyde assimilation I (serine pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1622'}, {'name': 'L-serine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8011'}]}}, 'locations': [{'start': 245, 'end': 267, 'representative': False, 'level': 'STRONG', 'cigarAlignment': '2M2I4M1I2M1I1M4I1M2I1M1I1M', 'alignment': 'MYpvCDVVtLNiPlhspTehMiN', 'location-fragments': [{'start': 245, 'end': 267, 'dc-status': 'CONTINUOUS'}]}], 'model-ac': 'PS00670'}, {'signature': {'accession': 'cd05302', 'name': 'FDH', 'description': 'FDH', 'signatureLibraryRelease': {'library': 'CDD', 'version': '3.20'}, 'entry': {'accession': 'IPR033689', 'name': 'FDH_NAD-dep', 'description': 'NAD-dependent formate dehydrogenase', 'type': 'FAMILY', 'goXRefs': [{'name': 'formate dehydrogenase (NAD+) activity', 'databaseName': 'GO', 'category': 'MOLECULAR_FUNCTION', 'id': 'GO:0008863'}], 'pathwayXRefs': [{'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}]}}, 'locations': [{'start': 2, 'end': 378, 'representative': True, 'sites': [{'description': 'NAD binding site', 'numLocations': 25, 'siteLocations': [{'start': 283, 'end': 283, 'residue': 'T'}, {'start': 310, 'end': 310, 'residue': 'V'}, {'start': 262, 'end': 262, 'residue': 'T'}, {'start': 148, 'end': 148, 'residue': 'S'}, {'start': 151, 'end': 151, 'residue': 'V'}, {'start': 336, 'end': 336, 'residue': 'G'}, {'start': 203, 'end': 203, 'residue': 'I'}, {'start': 333, 'end': 333, 'residue': 'H'}, {'start': 223, 'end': 223, 'residue': 'R'}, {'start': 256, 'end': 256, 'residue': 'I'}, {'start': 202, 'end': 202, 'residue': 'R'}, {'start': 259, 'end': 259, 'residue': 'H'}, {'start': 255, 'end': 255, 'residue': 'N'}, {'start': 201, 'end': 201, 'residue': 'G'}, {'start': 222, 'end': 222, 'residue': 'D'}, {'start': 123, 'end': 123, 'residue': 'I'}, {'start': 99, 'end': 99, 'residue': 'F'}, {'start': 147, 'end': 147, 'residue': 'N'}, {'start': 224, 'end': 224, 'residue': 'H'}, {'start': 335, 'end': 335, 'residue': 'S'}, {'start': 257, 'end': 257, 'residue': 'P'}, {'start': 199, 'end': 199, 'residue': 'A'}, {'start': 284, 'end': 284, 'residue': 'A'}, {'start': 309, 'end': 309, 'residue': 'D'}, {'start': 285, 'end': 285, 'residue': 'R'}]}, {'description': 'dimerization interface', 'numLocations': 62, 'siteLocations': [{'start': 337, 'end': 337, 'residue': 'T'}, {'start': 193, 'end': 193, 'residue': 'M'}, {'start': 176, 'end': 176, 'residue': 'G'}, {'start': 181, 'end': 181, 'residue': 'A'}, {'start': 171, 'end': 171, 'residue': 'E'}, {'start': 50, 'end': 50, 'residue': 'L'}, {'start': 153, 'end': 153, 'residue': 'E'}, {'start': 167, 'end': 167, 'residue': 'L'}, {'start': 150, 'end': 150, 'residue': 'S'}, {'start': 214, 'end': 214, 'residue': 'F'}, {'start': 323, 'end': 323, 'residue': 'R'}, {'start': 161, 'end': 161, 'residue': 'S'}, {'start': 318, 'end': 318, 'residue': 'K'}, {'start': 213, 'end': 213, 'residue': 'P'}, {'start': 331, 'end': 331, 'residue': 'T'}, {'start': 209, 'end': 209, 'residue': 'R'}, {'start': 190, 'end': 190, 'residue': 'L'}, {'start': 9, 'end': 9, 'residue': 'Y'}, {'start': 330, 'end': 330, 'residue': 'M'}, {'start': 11, 'end': 11, 'residue': 'D'}, {'start': 179, 'end': 179, 'residue': 'N'}, {'start': 342, 'end': 342, 'residue': 'Q'}, {'start': 191, 'end': 191, 'residue': 'E'}, {'start': 178, 'end': 178, 'residue': 'W'}, {'start': 324, 'end': 324, 'residue': 'T'}, {'start': 316, 'end': 316, 'residue': 'A'}, {'start': 332, 'end': 332, 'residue': 'P'}, {'start': 177, 'end': 177, 'residue': 'G'}, {'start': 154, 'end': 154, 'residue': 'H'}, {'start': 169, 'end': 169, 'residue': 'S'}, {'start': 315, 'end': 315, 'residue': 'P'}, {'start': 333, 'end': 333, 'residue': 'H'}, {'start': 188, 'end': 188, 'residue': 'Y'}, {'start': 344, 'end': 344, 'residue': 'R'}, {'start': 172, 'end': 172, 'residue': 'W'}, {'start': 329, 'end': 329, 'residue': 'G'}, {'start': 311, 'end': 311, 'residue': 'W'}, {'start': 174, 'end': 174, 'residue': 'R'}, {'start': 340, 'end': 340, 'residue': 'T'}, {'start': 338, 'end': 338, 'residue': 'T'}, {'start': 166, 'end': 166, 'residue': 'Y'}, {'start': 314, 'end': 314, 'residue': 'Q'}, {'start': 184, 'end': 184, 'residue': 'V'}, {'start': 173, 'end': 173, 'residue': 'A'}, {'start': 334, 'end': 334, 'residue': 'I'}, {'start': 187, 'end': 187, 'residue': 'A'}, {'start': 157, 'end': 157, 'residue': 'M'}, {'start': 185, 'end': 185, 'residue': 'S'}, {'start': 170, 'end': 170, 'residue': 'H'}, {'start': 160, 'end': 160, 'residue': 'L'}, {'start': 156, 'end': 156, 'residue': 'V'}, {'start': 164, 'end': 164, 'residue': 'R'}, {'start': 339, 'end': 339, 'residue': 'L'}, {'start': 189, 'end': 189, 'residue': 'D'}, {'start': 182, 'end': 182, 'residue': 'D'}, {'start': 180, 'end': 180, 'residue': 'I'}, {'start': 149, 'end': 149, 'residue': 'I'}, {'start': 328, 'end': 328, 'residue': 'N'}, {'start': 10, 'end': 10, 'residue': 'D'}, {'start': 335, 'end': 335, 'residue': 'S'}, {'start': 183, 'end': 183, 'residue': 'C'}, {'start': 210, 'end': 210, 'residue': 'R'}]}], 'evalue': 0.0, 'score': 692.915, 'location-fragments': [{'start': 2, 'end': 378, 'dc-status': 'CONTINUOUS'}]}], 'model-ac': 'cd05302'}, {'signature': {'accession': 'PTHR42938', 'name': 'FORMATE DEHYDROGENASE 1', 'description': None, 'signatureLibraryRelease': {'library': 'PANTHER', 'version': '18.0'}, 'entry': None}, 'locations': [{'start': 72, 'end': 360, 'representative': False, 'hmmStart': 30, 'hmmEnd': 311, 'hmmLength': 0, 'hmmBounds': 'INCOMPLETE', 'envelopeStart': 51, 'envelopeEnd': 365, 'location-fragments': [{'start': 72, 'end': 360, 'dc-status': 'CONTINUOUS'}]}], 'accession': 'PTHR42938:SF9', 'name': 'FORMATE DEHYDROGENASE 1', 'evalue': 4.1e-63, 'score': 225.2, 'proteinClass': 'PC00092', 'graftPoint': 'PTN002873291', 'goXRefs': [{'name': 'formate dehydrogenase (NAD+) activity', 'databaseName': 'GO', 'category': 'MOLECULAR_FUNCTION', 'id': 'GO:0008863'}], 'model-ac': 'PTHR42938:SF9'}, {'signature': {'accession': 'G3DSA:3.40.50.720', 'name': '', 'description': None, 'signatureLibraryRelease': {'library': 'GENE3D', 'version': '4.3.0'}, 'entry': None}, 'locations': [{'start': 147, 'end': 338, 'representative': False, 'hmmStart': 1, 'hmmEnd': 381, 'hmmLength': 192, 'hmmBounds': 'INCOMPLETE', 'evalue': 0.0, 'score': 750.9, 'envelopeStart': 2, 'envelopeEnd': 384, 'postProcessed': True, 'location-fragments': [{'start': 147, 'end': 338, 'dc-status': 'CONTINUOUS'}]}], 'evalue': 0.0, 'score': 751.1, 'model-ac': '2gsdA02'}, {'signature': {'accession': 'PF00389', 'name': '2-Hacid_dh', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, catalytic domain', 'signatureLibraryRelease': {'library': 'PFAM', 'version': '36.0'}, 'entry': {'accession': 'IPR006139', 'name': 'D-isomer_2_OHA_DH_cat_dom', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, catalytic domain', 'type': 'DOMAIN', 'goXRefs': [{'name': 'oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor', 'databaseName': 'GO', 'category': 'MOLECULAR_FUNCTION', 'id': 'GO:0016616'}, {'name': 'NAD binding', 'databaseName': 'GO', 'category': 'MOLECULAR_FUNCTION', 'id': 'GO:0051287'}], 'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8238'}, {'name': 'zwittermicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7694'}, {'name': '(-)-dehydrodiconiferyl alcohol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7557'}, {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7331'}, {'name': 'anaerobic energy metabolism (invertebrates, cytosol)', 'databaseName': 'MetaCyc', 'id': 'PWY-7383'}, {'name': 'stearate biosynthesis I (animals)', 'databaseName': 'MetaCyc', 'id': 'PWY-5972'}, {'name': 'autoinducer CAI-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7621'}, {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6973'}, {'name': 'septacidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8366'}, {'name': 'bisphenol A degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7757'}, {'name': 'pyruvate fermentation to (R)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-8274'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-389661'}, {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7610'}, {'name': 'photorespiration III', 'databaseName': 'MetaCyc', 'id': 'PWY-8363'}, {'name': 'sphingofungin B-D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8464'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-MMU-3899300'}, {'name': 'butanol and isobutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7396'}, {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA', 'databaseName': 'MetaCyc', 'id': 'PWY-5652'}, {'name': 'pyruvate fermentation to butanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-6583'}, {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7340'}, {'name': 'D-galactose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6693'}, {'name': 'sitosterol degradation to androstenedione', 'databaseName': 'MetaCyc', 'id': 'PWY-6948'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-977347'}, {'name': '1,2-propanediol biosynthesis from lactate (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7541'}, {'name': '(S)-camphor degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6989'}, {'name': 'meso-butanediol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6392'}, {'name': 'daunorubicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7352'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}, {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-8010'}, {'name': 'L-isoleucine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5101'}, {'name': 'representative whole plasma arachidonate-derived icosanoids I', 'databaseName': 'MetaCyc', 'id': 'PWY-8410'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-389661'}, {'name': 'kanosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5978'}, {'name': 'pyrethrin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6926'}, {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5882'}, {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7301'}, {'name': 'reductive TCA cycle II', 'databaseName': 'MetaCyc', 'id': 'PWY-5392'}, {'name': 'phosphinothricin tripeptide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6322'}, {'name': 'sulfolactate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6637'}, {'name': 'L-tyrosine degradation V (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8016'}, {'name': 'dTDP-beta-L-digitoxose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7657'}, {'name': 'ephedrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5883'}, {'name': 'rosmarinic acid biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5049'}, {'name': 'oryzalexin A, B, and C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7489'}, {'name': 'L-galactose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7568'}, {'name': 'Signaling by TCF7L2 mutants', 'databaseName': 'Reactome', 'id': 'R-HSA-5339700'}, {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7606'}, {'name': 'hygromycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8367'}, {'name': 'L-ascorbate degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6704'}, {'name': 'dTDP-beta-L-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6976'}, {'name': 'superpathway of nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7342'}, {'name': 'terrequinone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7520'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-HSA-3769402'}, {'name': 'CDP-6-deoxy-D-gulose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8139'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-CEL-4641265'}, {'name': 'L-homomethionine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1186'}, {'name': 'L-canavanine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8419'}, {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7338'}, {'name': 'methylaspartate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6728'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-DME-3899300'}, {'name': 'tropane alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5317'}, {'name': 'TCA cycle V (2-oxoglutarate synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6969'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'CDP-ascarylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5830'}, {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)', 'databaseName': 'MetaCyc', 'id': 'PWY-7615'}, {'name': '3-sulfopropanediol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6634'}, {'name': 'nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5316'}, {'name': 'vancomycin resistance I', 'databaseName': 'MetaCyc', 'id': 'PWY-6454'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-BTA-4641265'}, {'name': '3-sulfopropanediol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8214'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-DME-3769402'}, {'name': 'tetramethylpyrazine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7403'}, {'name': 'pinitol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6738'}, {'name': 'guadinomine B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7693'}, {'name': '6-diazo-5-oxo-L-norleucine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8297'}, {'name': 'detoxification of reactive carbonyls in chloroplasts', 'databaseName': 'MetaCyc', 'id': 'PWY-6786'}, {'name': 'isethionate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-2601'}, {'name': '1,3-propanediol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7385'}, {'name': 'L-tryptophan degradation V (side chain pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-3162'}, {'name': 'arabidopyrone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7682'}, {'name': 'dTDP-beta-L-olivose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6974'}, {'name': 'sporopollenin precursors biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6733'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-HSA-3899300'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-RNO-3899300'}, {'name': 'propane degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7774'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7337'}, {'name': 'androstenedione degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8152'}, {'name': 'rosmarinic acid biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5048'}, {'name': 'mupirocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8012'}, {'name': 'antimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8374'}, {'name': 'erinacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8408'}, {'name': 'jasmonic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-735'}, {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7300'}, {'name': 'pyruvate fermentation to hexanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6863'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-7115'}, {'name': 'L-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7787'}, {'name': 'suberin monomers biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1121'}, {'name': 'cis-geranyl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6672'}, {'name': 'representative whole plasma arachidonate-derived icosanoids II', 'databaseName': 'MetaCyc', 'id': 'PWY-8411'}, {'name': 'aurofusarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7695'}, {'name': 'bile acid biosynthesis, neutral pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6061'}, {'name': 'phenylethanol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5751'}, {'name': 'mithramycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7045'}, {'name': 'arachidonate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8397'}, {'name': 'aclacinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7354'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-DME-4641265'}, {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6946'}, {'name': 'heme d1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7554'}, {'name': 'methyl tert-butyl ether degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7779'}, {'name': 'fusicoccin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6659'}, {'name': '(Z)-phenylmethanethial S-oxide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6539'}, {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)', 'databaseName': 'MetaCyc', 'id': 'PWY-6318'}, {'name': 'myxochelin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8383'}, {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8132'}, {'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-977347'}, {'name': '11-cis-3-hydroxyretinal biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7043'}, {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8014'}, {'name': 'TCA cycle II (plants and fungi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5690'}, {'name': 'leporin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8370'}, {'name': 'superpathway of glucose and xylose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6901'}, {'name': 'photorespiration II', 'databaseName': 'MetaCyc', 'id': 'PWY-8362'}, {'name': 'abscisic acid degradation to phaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-5271'}, {'name': 'L-glucose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7130'}, {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7304'}, {'name': 'bikaverin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7692'}, {'name': '6-methylpretetramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7811'}, {'name': 'phytosterol biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2541'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-RNO-3769402'}, {'name': 'photorespiration I', 'databaseName': 'MetaCyc', 'id': 'PWY-181'}, {'name': '6-gingerol analog biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6920'}, {'name': 'dihydroconiferyl alcohol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6302'}, {'name': 'phosalacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7769'}, {'name': 'phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7742'}, {'name': 'mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWYG-321'}, {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7613'}, {'name': 'L-alanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8188'}, {'name': 'propane degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7775'}, {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen', 'databaseName': 'MetaCyc', 'id': 'PWY-8086'}, {'name': 'L-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7295'}, {'name': 'apicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7667'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-977347'}, {'name': '8-O-methylfusarubin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7689'}, {'name': '4-ethylphenol degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6080'}, {'name': 'Sensory processing of sound by inner hair cells of the cochlea', 'databaseName': 'Reactome', 'id': 'R-HSA-9662360'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-MMU-3769402'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-RNO-4641265'}, {'name': 't-anethole biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5867'}, {'name': 'the visual cycle (insects)', 'databaseName': 'MetaCyc', 'id': 'PWY-7042'}, {'name': '2-deoxy-D-ribose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8058'}, {'name': 'D-xylose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7294'}, {'name': 'quinoxaline-2-carboxylate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7734'}, {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-3722'}, {'name': 'sulfolactate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6638'}, {'name': '3-hydroxy-L-homotyrosine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7649'}, {'name': 'mannojirimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7531'}, {'name': 'cinchona alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5848'}, {'name': 'L-glutamate degradation XI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8190'}, {'name': 'dTDP-L-daunosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7814'}, {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5754'}, {'name': 'p-cymene degradation to p-cumate', 'databaseName': 'MetaCyc', 'id': 'PWY-741'}, {'name': 'limonene degradation IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8029'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-977347'}, {'name': 'salinosporamide A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6627'}, {'name': 'erythritol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7788'}, {'name': 'dTDP-beta-L-megosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7104'}, {'name': '2-keto-L-gulonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5782'}, {'name': 'UDP-beta-L-rhamnose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3261'}, {'name': 'androstenedione degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6944'}, {'name': 'validamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5818'}, {'name': 'phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7741'}, {'name': 'superpathway of glyoxylate cycle and fatty acid degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-561'}, {'name': 'sulfolactate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6616'}, {'name': 'myo-inositol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7241'}, {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6945'}, {'name': 'testosterone degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8155'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7726'}, {'name': 'alpha-solanine/alpha-chaconine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5666'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-HSA-4641265'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-CEL-3769402'}, {'name': 'dehydrophos biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6682'}, {'name': 'pyruvate fermentation to butanol II (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6883'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-MMU-4641265'}, {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7333'}, {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)', 'databaseName': 'MetaCyc', 'id': 'PWY-6053'}, {'name': 'juglone degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7825'}, {'name': '4-oxopentanoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7948'}, {'name': '(2R)-methoxymalonyl-[acp] biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8426'}, {'name': 'patulin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7490'}, {'name': 'cyathin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8409'}, {'name': 'cutin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-321'}, {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-7288'}, {'name': 'ecdysteroid metabolism (arthropods)', 'databaseName': 'MetaCyc', 'id': 'PWY-7321'}, {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7440'}, {'name': 'meso-butanediol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6391'}, {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7339'}, {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6626'}, {'name': 'prodigiosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7547'}, {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8017'}, {'name': 'tryptoquialanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7660'}, {'name': 'formaldehyde assimilation I (serine pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1622'}, {'name': 'L-serine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8011'}]}}, 'locations': [{'start': 62, 'end': 363, 'representative': False, 'hmmStart': 13, 'hmmEnd': 130, 'hmmLength': 134, 'hmmBounds': 'INCOMPLETE', 'evalue': 1.4e-11, 'score': 56.0, 'envelopeStart': 55, 'envelopeEnd': 366, 'postProcessed': True, 'location-fragments': [{'start': 62, 'end': 155, 'dc-status': 'C_TERMINAL_DISC'}, {'start': 336, 'end': 363, 'dc-status': 'N_TERMINAL_DISC'}]}], 'evalue': 1e-11, 'score': 56.0, 'model-ac': 'PF00389'}, {'signature': {'accession': 'PS00671', 'name': 'D_2_HYDROXYACID_DH_3', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenases signature 3.', 'signatureLibraryRelease': {'library': 'PROSITE_PATTERNS', 'version': '2023_05'}, 'entry': {'accession': 'IPR029753', 'name': 'D-isomer_DH_CS', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain conserved site', 'type': 'CONSERVED_SITE', 'goXRefs': [], 'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8238'}, {'name': 'zwittermicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7694'}, {'name': '(-)-dehydrodiconiferyl alcohol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7557'}, {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7331'}, {'name': 'anaerobic energy metabolism (invertebrates, cytosol)', 'databaseName': 'MetaCyc', 'id': 'PWY-7383'}, {'name': 'stearate biosynthesis I (animals)', 'databaseName': 'MetaCyc', 'id': 'PWY-5972'}, {'name': 'autoinducer CAI-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7621'}, {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6973'}, {'name': 'septacidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8366'}, {'name': 'bisphenol A degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7757'}, {'name': 'pyruvate fermentation to (R)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-8274'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-389661'}, {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7610'}, {'name': 'photorespiration III', 'databaseName': 'MetaCyc', 'id': 'PWY-8363'}, {'name': 'sphingofungin B-D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8464'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-MMU-3899300'}, {'name': 'butanol and isobutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7396'}, {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA', 'databaseName': 'MetaCyc', 'id': 'PWY-5652'}, {'name': 'pyruvate fermentation to butanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-6583'}, {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7340'}, {'name': 'D-galactose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6693'}, {'name': 'sitosterol degradation to androstenedione', 'databaseName': 'MetaCyc', 'id': 'PWY-6948'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-977347'}, {'name': '1,2-propanediol biosynthesis from lactate (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7541'}, {'name': '(S)-camphor degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6989'}, {'name': 'meso-butanediol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6392'}, {'name': 'daunorubicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7352'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}, {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-8010'}, {'name': 'L-isoleucine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5101'}, {'name': 'representative whole plasma arachidonate-derived icosanoids I', 'databaseName': 'MetaCyc', 'id': 'PWY-8410'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-389661'}, {'name': 'kanosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5978'}, {'name': 'pyrethrin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6926'}, {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5882'}, {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7301'}, {'name': 'reductive TCA cycle II', 'databaseName': 'MetaCyc', 'id': 'PWY-5392'}, {'name': 'phosphinothricin tripeptide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6322'}, {'name': 'sulfolactate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6637'}, {'name': 'L-tyrosine degradation V (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8016'}, {'name': 'dTDP-beta-L-digitoxose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7657'}, {'name': 'ephedrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5883'}, {'name': 'oryzalexin A, B, and C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7489'}, {'name': 'L-galactose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7568'}, {'name': 'Signaling by TCF7L2 mutants', 'databaseName': 'Reactome', 'id': 'R-HSA-5339700'}, {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7606'}, {'name': 'hygromycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8367'}, {'name': 'L-ascorbate degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6704'}, {'name': 'dTDP-beta-L-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6976'}, {'name': 'superpathway of nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7342'}, {'name': 'terrequinone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7520'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-HSA-3769402'}, {'name': 'CDP-6-deoxy-D-gulose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8139'}, {'name': 'L-homomethionine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1186'}, {'name': 'L-canavanine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8419'}, {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7338'}, {'name': 'methylaspartate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6728'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-DME-3899300'}, {'name': 'tropane alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5317'}, {'name': 'TCA cycle V (2-oxoglutarate synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6969'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'CDP-ascarylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5830'}, {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)', 'databaseName': 'MetaCyc', 'id': 'PWY-7615'}, {'name': '3-sulfopropanediol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6634'}, {'name': 'nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5316'}, {'name': 'vancomycin resistance I', 'databaseName': 'MetaCyc', 'id': 'PWY-6454'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-BTA-4641265'}, {'name': '3-sulfopropanediol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8214'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-DME-3769402'}, {'name': 'tetramethylpyrazine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7403'}, {'name': 'pinitol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6738'}, {'name': 'guadinomine B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7693'}, {'name': '6-diazo-5-oxo-L-norleucine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8297'}, {'name': 'detoxification of reactive carbonyls in chloroplasts', 'databaseName': 'MetaCyc', 'id': 'PWY-6786'}, {'name': 'isethionate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-2601'}, {'name': '1,3-propanediol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7385'}, {'name': 'L-tryptophan degradation V (side chain pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-3162'}, {'name': 'arabidopyrone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7682'}, {'name': 'dTDP-beta-L-olivose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6974'}, {'name': 'sporopollenin precursors biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6733'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-HSA-3899300'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-RNO-3899300'}, {'name': 'propane degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7774'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7337'}, {'name': 'androstenedione degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8152'}, {'name': 'mupirocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8012'}, {'name': 'antimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8374'}, {'name': 'erinacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8408'}, {'name': 'jasmonic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-735'}, {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7300'}, {'name': 'pyruvate fermentation to hexanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6863'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-7115'}, {'name': 'L-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7787'}, {'name': 'suberin monomers biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1121'}, {'name': 'cis-geranyl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6672'}, {'name': 'representative whole plasma arachidonate-derived icosanoids II', 'databaseName': 'MetaCyc', 'id': 'PWY-8411'}, {'name': 'aurofusarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7695'}, {'name': 'bile acid biosynthesis, neutral pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6061'}, {'name': 'phenylethanol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5751'}, {'name': 'mithramycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7045'}, {'name': 'arachidonate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8397'}, {'name': 'aclacinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7354'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-DME-4641265'}, {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6946'}, {'name': 'heme d1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7554'}, {'name': 'methyl tert-butyl ether degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7779'}, {'name': 'fusicoccin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6659'}, {'name': '(Z)-phenylmethanethial S-oxide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6539'}, {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)', 'databaseName': 'MetaCyc', 'id': 'PWY-6318'}, {'name': 'myxochelin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8383'}, {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8132'}, {'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-977347'}, {'name': '11-cis-3-hydroxyretinal biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7043'}, {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8014'}, {'name': 'TCA cycle II (plants and fungi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5690'}, {'name': 'leporin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8370'}, {'name': 'superpathway of glucose and xylose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6901'}, {'name': 'photorespiration II', 'databaseName': 'MetaCyc', 'id': 'PWY-8362'}, {'name': 'abscisic acid degradation to phaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-5271'}, {'name': 'L-glucose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7130'}, {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7304'}, {'name': 'bikaverin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7692'}, {'name': '6-methylpretetramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7811'}, {'name': 'phytosterol biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2541'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-RNO-3769402'}, {'name': 'photorespiration I', 'databaseName': 'MetaCyc', 'id': 'PWY-181'}, {'name': '6-gingerol analog biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6920'}, {'name': 'dihydroconiferyl alcohol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6302'}, {'name': 'phosalacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7769'}, {'name': 'phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7742'}, {'name': 'mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWYG-321'}, {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7613'}, {'name': 'L-alanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8188'}, {'name': 'propane degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7775'}, {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen', 'databaseName': 'MetaCyc', 'id': 'PWY-8086'}, {'name': 'L-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7295'}, {'name': 'apicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7667'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-977347'}, {'name': '8-O-methylfusarubin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7689'}, {'name': '4-ethylphenol degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6080'}, {'name': 'Sensory processing of sound by inner hair cells of the cochlea', 'databaseName': 'Reactome', 'id': 'R-HSA-9662360'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-MMU-3769402'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-RNO-4641265'}, {'name': 't-anethole biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5867'}, {'name': 'the visual cycle (insects)', 'databaseName': 'MetaCyc', 'id': 'PWY-7042'}, {'name': '2-deoxy-D-ribose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8058'}, {'name': 'D-xylose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7294'}, {'name': 'quinoxaline-2-carboxylate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7734'}, {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-3722'}, {'name': 'sulfolactate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6638'}, {'name': '3-hydroxy-L-homotyrosine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7649'}, {'name': 'mannojirimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7531'}, {'name': 'cinchona alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5848'}, {'name': 'L-glutamate degradation XI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8190'}, {'name': 'dTDP-L-daunosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7814'}, {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5754'}, {'name': 'p-cymene degradation to p-cumate', 'databaseName': 'MetaCyc', 'id': 'PWY-741'}, {'name': 'limonene degradation IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8029'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-977347'}, {'name': 'salinosporamide A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6627'}, {'name': 'erythritol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7788'}, {'name': 'dTDP-beta-L-megosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7104'}, {'name': '2-keto-L-gulonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5782'}, {'name': 'UDP-beta-L-rhamnose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3261'}, {'name': 'androstenedione degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6944'}, {'name': 'validamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5818'}, {'name': 'phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7741'}, {'name': 'superpathway of glyoxylate cycle and fatty acid degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-561'}, {'name': 'sulfolactate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6616'}, {'name': 'myo-inositol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7241'}, {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6945'}, {'name': 'testosterone degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8155'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7726'}, {'name': 'alpha-solanine/alpha-chaconine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5666'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-HSA-4641265'}, {'name': 'dehydrophos biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6682'}, {'name': 'pyruvate fermentation to butanol II (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6883'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-MMU-4641265'}, {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7333'}, {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)', 'databaseName': 'MetaCyc', 'id': 'PWY-6053'}, {'name': 'juglone degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7825'}, {'name': '4-oxopentanoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7948'}, {'name': '(2R)-methoxymalonyl-[acp] biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8426'}, {'name': 'patulin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7490'}, {'name': 'cyathin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8409'}, {'name': 'cutin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-321'}, {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-7288'}, {'name': 'ecdysteroid metabolism (arthropods)', 'databaseName': 'MetaCyc', 'id': 'PWY-7321'}, {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7440'}, {'name': 'meso-butanediol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6391'}, {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7339'}, {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6626'}, {'name': 'prodigiosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7547'}, {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8017'}, {'name': 'tryptoquialanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7660'}, {'name': 'formaldehyde assimilation I (serine pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1622'}, {'name': 'L-serine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8011'}]}}, 'locations': [{'start': 274, 'end': 290, 'representative': False, 'level': 'STRONG', 'cigarAlignment': '2M1I1M1I4M1I3M1I3M', 'alignment': 'FKrGaYIVNtARGkLCD', 'location-fragments': [{'start': 274, 'end': 290, 'dc-status': 'CONTINUOUS'}]}], 'model-ac': 'PS00671'}, {'signature': {'accession': 'PF02826', 'name': '2-Hacid_dh_C', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD binding domain', 'signatureLibraryRelease': {'library': 'PFAM', 'version': '36.0'}, 'entry': {'accession': 'IPR006140', 'name': 'D-isomer_DH_NAD-bd', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain', 'type': 'DOMAIN', 'goXRefs': [{'name': 'NAD binding', 'databaseName': 'GO', 'category': 'MOLECULAR_FUNCTION', 'id': 'GO:0051287'}], 'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8238'}, {'name': 'zwittermicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7694'}, {'name': '(-)-dehydrodiconiferyl alcohol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7557'}, {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7331'}, {'name': 'anaerobic energy metabolism (invertebrates, cytosol)', 'databaseName': 'MetaCyc', 'id': 'PWY-7383'}, {'name': 'stearate biosynthesis I (animals)', 'databaseName': 'MetaCyc', 'id': 'PWY-5972'}, {'name': 'autoinducer CAI-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7621'}, {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6973'}, {'name': 'septacidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8366'}, {'name': 'bisphenol A degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7757'}, {'name': 'pyruvate fermentation to (R)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-8274'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-389661'}, {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7610'}, {'name': 'photorespiration III', 'databaseName': 'MetaCyc', 'id': 'PWY-8363'}, {'name': 'sphingofungin B-D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8464'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-MMU-3899300'}, {'name': 'butanol and isobutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7396'}, {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA', 'databaseName': 'MetaCyc', 'id': 'PWY-5652'}, {'name': 'pyruvate fermentation to butanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-6583'}, {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7340'}, {'name': 'D-galactose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6693'}, {'name': 'sitosterol degradation to androstenedione', 'databaseName': 'MetaCyc', 'id': 'PWY-6948'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-977347'}, {'name': '1,2-propanediol biosynthesis from lactate (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7541'}, {'name': '(S)-camphor degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6989'}, {'name': 'meso-butanediol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6392'}, {'name': 'daunorubicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7352'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}, {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-8010'}, {'name': 'L-isoleucine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5101'}, {'name': 'representative whole plasma arachidonate-derived icosanoids I', 'databaseName': 'MetaCyc', 'id': 'PWY-8410'}, {'name': \"inosine-5'-phosphate biosynthesis III\", 'databaseName': 'MetaCyc', 'id': 'PWY-7234'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-389661'}, {'name': 'kanosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5978'}, {'name': 'pyrethrin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6926'}, {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5882'}, {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7301'}, {'name': 'reductive TCA cycle II', 'databaseName': 'MetaCyc', 'id': 'PWY-5392'}, {'name': 'phosphinothricin tripeptide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6322'}, {'name': 'sulfolactate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6637'}, {'name': 'L-tyrosine degradation V (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8016'}, {'name': 'dTDP-beta-L-digitoxose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7657'}, {'name': 'ephedrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5883'}, {'name': 'rosmarinic acid biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5049'}, {'name': 'oryzalexin A, B, and C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7489'}, {'name': 'L-galactose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7568'}, {'name': 'Signaling by TCF7L2 mutants', 'databaseName': 'Reactome', 'id': 'R-HSA-5339700'}, {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7606'}, {'name': 'hygromycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8367'}, {'name': 'L-ascorbate degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6704'}, {'name': 'dTDP-beta-L-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6976'}, {'name': 'superpathway of nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7342'}, {'name': 'terrequinone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7520'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-HSA-3769402'}, {'name': 'CDP-6-deoxy-D-gulose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8139'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-CEL-4641265'}, {'name': 'L-homomethionine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1186'}, {'name': 'superpathway of 5-aminoimidazole ribonucleotide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6277'}, {'name': 'L-canavanine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8419'}, {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7338'}, {'name': 'methylaspartate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6728'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-DME-3899300'}, {'name': 'tropane alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5317'}, {'name': 'TCA cycle V (2-oxoglutarate synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6969'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'CDP-ascarylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5830'}, {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)', 'databaseName': 'MetaCyc', 'id': 'PWY-7615'}, {'name': '3-sulfopropanediol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6634'}, {'name': 'nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5316'}, {'name': 'vancomycin resistance I', 'databaseName': 'MetaCyc', 'id': 'PWY-6454'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-BTA-4641265'}, {'name': '3-sulfopropanediol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8214'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-DME-3769402'}, {'name': 'tetramethylpyrazine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7403'}, {'name': 'pinitol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6738'}, {'name': 'guadinomine B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7693'}, {'name': '6-diazo-5-oxo-L-norleucine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8297'}, {'name': 'detoxification of reactive carbonyls in chloroplasts', 'databaseName': 'MetaCyc', 'id': 'PWY-6786'}, {'name': 'isethionate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-2601'}, {'name': 'anaerobic energy metabolism (invertebrates, mitochondrial)', 'databaseName': 'MetaCyc', 'id': 'PWY-7384'}, {'name': '1,3-propanediol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7385'}, {'name': 'L-tryptophan degradation V (side chain pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-3162'}, {'name': 'arabidopyrone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7682'}, {'name': 'dTDP-beta-L-olivose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6974'}, {'name': 'sporopollenin precursors biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6733'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-HSA-3899300'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-RNO-3899300'}, {'name': 'propane degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7774'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7337'}, {'name': 'androstenedione degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8152'}, {'name': 'rosmarinic acid biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5048'}, {'name': 'mupirocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8012'}, {'name': 'antimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8374'}, {'name': 'erinacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8408'}, {'name': 'jasmonic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-735'}, {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7300'}, {'name': 'pyruvate fermentation to hexanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6863'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-7115'}, {'name': 'L-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7787'}, {'name': 'suberin monomers biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1121'}, {'name': 'cis-geranyl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6672'}, {'name': 'representative whole plasma arachidonate-derived icosanoids II', 'databaseName': 'MetaCyc', 'id': 'PWY-8411'}, {'name': 'aurofusarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7695'}, {'name': 'bile acid biosynthesis, neutral pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6061'}, {'name': 'phenylethanol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5751'}, {'name': 'mithramycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7045'}, {'name': 'arachidonate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8397'}, {'name': 'aclacinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7354'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-DME-4641265'}, {'name': '5-aminoimidazole ribonucleotide biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6122'}, {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6946'}, {'name': 'heme d1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7554'}, {'name': 'methyl tert-butyl ether degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7779'}, {'name': 'fusicoccin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6659'}, {'name': '(Z)-phenylmethanethial S-oxide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6539'}, {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)', 'databaseName': 'MetaCyc', 'id': 'PWY-6318'}, {'name': 'myxochelin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8383'}, {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8132'}, {'name': 'L-malate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7686'}, {'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-977347'}, {'name': '11-cis-3-hydroxyretinal biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7043'}, {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8014'}, {'name': 'TCA cycle II (plants and fungi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5690'}, {'name': 'leporin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8370'}, {'name': 'superpathway of glucose and xylose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6901'}, {'name': 'photorespiration II', 'databaseName': 'MetaCyc', 'id': 'PWY-8362'}, {'name': 'abscisic acid degradation to phaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-5271'}, {'name': 'L-glucose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7130'}, {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7304'}, {'name': 'bikaverin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7692'}, {'name': '6-methylpretetramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7811'}, {'name': 'phytosterol biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2541'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-RNO-3769402'}, {'name': 'photorespiration I', 'databaseName': 'MetaCyc', 'id': 'PWY-181'}, {'name': '6-gingerol analog biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6920'}, {'name': 'dihydroconiferyl alcohol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6302'}, {'name': 'phosalacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7769'}, {'name': 'phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7742'}, {'name': 'mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWYG-321'}, {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7613'}, {'name': 'L-alanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8188'}, {'name': 'propane degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7775'}, {'name': \"inosine-5'-phosphate biosynthesis I\", 'databaseName': 'MetaCyc', 'id': 'PWY-6123'}, {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen', 'databaseName': 'MetaCyc', 'id': 'PWY-8086'}, {'name': 'L-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7295'}, {'name': 'apicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7667'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-977347'}, {'name': '8-O-methylfusarubin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7689'}, {'name': '4-ethylphenol degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6080'}, {'name': 'Sensory processing of sound by inner hair cells of the cochlea', 'databaseName': 'Reactome', 'id': 'R-HSA-9662360'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-MMU-3769402'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-RNO-4641265'}, {'name': 't-anethole biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5867'}, {'name': 'the visual cycle (insects)', 'databaseName': 'MetaCyc', 'id': 'PWY-7042'}, {'name': '2-deoxy-D-ribose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8058'}, {'name': 'D-xylose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7294'}, {'name': 'quinoxaline-2-carboxylate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7734'}, {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-3722'}, {'name': 'sulfolactate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6638'}, {'name': '3-hydroxy-L-homotyrosine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7649'}, {'name': 'mannojirimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7531'}, {'name': 'cinchona alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5848'}, {'name': 'L-glutamate degradation XI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8190'}, {'name': 'dTDP-L-daunosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7814'}, {'name': 'chitin deacetylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7118'}, {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5754'}, {'name': 'p-cymene degradation to p-cumate', 'databaseName': 'MetaCyc', 'id': 'PWY-741'}, {'name': 'limonene degradation IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8029'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-977347'}, {'name': 'salinosporamide A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6627'}, {'name': 'erythritol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7788'}, {'name': 'dTDP-beta-L-megosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7104'}, {'name': '2-keto-L-gulonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5782'}, {'name': 'UDP-beta-L-rhamnose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3261'}, {'name': 'androstenedione degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6944'}, {'name': 'validamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5818'}, {'name': 'phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7741'}, {'name': 'superpathway of glyoxylate cycle and fatty acid degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-561'}, {'name': 'sulfolactate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6616'}, {'name': 'myo-inositol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7241'}, {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6945'}, {'name': 'testosterone degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8155'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7726'}, {'name': 'alpha-solanine/alpha-chaconine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5666'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-HSA-4641265'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-CEL-3769402'}, {'name': 'dehydrophos biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6682'}, {'name': 'pyruvate fermentation to butanol II (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6883'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-MMU-4641265'}, {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7333'}, {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)', 'databaseName': 'MetaCyc', 'id': 'PWY-6053'}, {'name': 'juglone degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7825'}, {'name': '4-oxopentanoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7948'}, {'name': '(2R)-methoxymalonyl-[acp] biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8426'}, {'name': 'patulin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7490'}, {'name': 'cyathin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8409'}, {'name': 'cutin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-321'}, {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-7288'}, {'name': 'ecdysteroid metabolism (arthropods)', 'databaseName': 'MetaCyc', 'id': 'PWY-7321'}, {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7440'}, {'name': 'meso-butanediol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6391'}, {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7339'}, {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6626'}, {'name': 'prodigiosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7547'}, {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8017'}, {'name': 'tryptoquialanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7660'}, {'name': 'formaldehyde assimilation I (serine pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1622'}, {'name': 'L-serine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8011'}, {'name': 'L-carnitine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-3641'}]}}, 'locations': [{'start': 156, 'end': 335, 'representative': False, 'hmmStart': 1, 'hmmEnd': 178, 'hmmLength': 178, 'hmmBounds': 'COMPLETE', 'evalue': 2.5e-44, 'score': 162.3, 'envelopeStart': 156, 'envelopeEnd': 335, 'postProcessed': True, 'location-fragments': [{'start': 156, 'end': 335, 'dc-status': 'CONTINUOUS'}]}], 'evalue': 1.8e-44, 'score': 162.7, 'model-ac': 'PF02826'}, {'signature': {'accession': 'MF_03210', 'name': 'Formate_dehydrogenase', 'description': 'Formate dehydrogenase, mitochondrial.', 'signatureLibraryRelease': {'library': 'HAMAP', 'version': '2023_05'}, 'entry': {'accession': 'IPR033689', 'name': 'FDH_NAD-dep', 'description': 'NAD-dependent formate dehydrogenase', 'type': 'FAMILY', 'goXRefs': [{'name': 'formate dehydrogenase (NAD+) activity', 'databaseName': 'GO', 'category': 'MOLECULAR_FUNCTION', 'id': 'GO:0008863'}], 'pathwayXRefs': [{'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}]}}, 'locations': [{'start': 2, 'end': 383, 'representative': False, 'score': 56.903183, 'alignment': 'Not available', 'location-fragments': [{'start': 2, 'end': 383, 'dc-status': 'CONTINUOUS'}]}], 'model-ac': 'MF_03210'}, {'signature': {'accession': 'SSF52283', 'name': 'Formate/glycerate dehydrogenase catalytic domain-like', 'description': None, 'signatureLibraryRelease': {'library': 'SUPERFAMILY', 'version': '1.75'}, 'entry': None}, 'locations': [{'start': 3, 'end': 173, 'representative': False, 'hmmLength': 187, 'location-fragments': [{'start': 3, 'end': 173, 'dc-status': 'CONTINUOUS'}]}], 'evalue': 9.26e-65, 'model-ac': '0045380'}, {'signature': {'accession': 'SSF51735', 'name': 'NAD(P)-binding Rossmann-fold domains', 'description': None, 'signatureLibraryRelease': {'library': 'SUPERFAMILY', 'version': '1.75'}, 'entry': {'accession': 'IPR036291', 'name': 'NAD(P)-bd_dom_sf', 'description': 'NAD(P)-binding domain superfamily', 'type': 'HOMOLOGOUS_SUPERFAMILY', 'goXRefs': [], 'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8238'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-194002'}, {'name': 'salvigenin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7325'}, {'name': 'quercetin glucoside degradation (Allium)', 'databaseName': 'MetaCyc', 'id': 'PWY-7133'}, {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6973'}, {'name': 'icosapentaenoate biosynthesis II (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7049'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-SPO-71064'}, {'name': 'nicotinate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5055'}, {'name': 'propene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5534'}, {'name': 'Escherichia coli serotype O:157/Salmonella enterica serotype O:30 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8224'}, {'name': 'Phenylalanine metabolism', 'databaseName': 'Reactome', 'id': 'R-BTA-8964208'}, {'name': 'UDP-beta-L-arabinose biosynthesis I (from UDP-alpha-D-xylose)', 'databaseName': 'MetaCyc', 'id': 'PWY-63'}, {'name': '2,5-xylenol and 3,5-xylenol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7698'}, {'name': 'salidroside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6802'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-193048'}, {'name': 'Synthesis of bile acids and bile salts via 24-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-MMU-193775'}, {'name': 'clorobiocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8368'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-977347'}, {'name': 'ascorbate glutathione cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-2261'}, {'name': 'Phase I - Functionalization of compounds', 'databaseName': 'Reactome', 'id': 'R-HSA-211945'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-DDI-5628897'}, {'name': 'fructan degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-862'}, {'name': 'meso-butanediol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6392'}, {'name': 'dipicolinate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8088'}, {'name': 'glyoxylate assimilation', 'databaseName': 'MetaCyc', 'id': 'PWY-5744'}, {'name': \"3,3'-thiodipropanoate degradation\", 'databaseName': 'MetaCyc', 'id': 'PWY-7465'}, {'name': 'arachidonate biosynthesis V (8-detaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7725'}, {'name': 'gondoate biosynthesis (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-7663'}, {'name': 'Lipid particle organization', 'databaseName': 'Reactome', 'id': 'R-CEL-8964572'}, {'name': 'hypoglycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5826'}, {'name': 'alpha-linolenic acid (ALA) metabolism', 'databaseName': 'Reactome', 'id': 'R-DME-2046106'}, {'name': 'Association of TriC/CCT with target proteins during biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-390471'}, {'name': 'UDP-N-acetyl-alpha-D-mannosaminouronate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7335'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-GGA-5652227'}, {'name': 'palmitoleate biosynthesis I (from (5Z)-dodec-5-enoate)', 'databaseName': 'MetaCyc', 'id': 'PWY-6282'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-194002'}, {'name': 'D-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-8331'}, {'name': 'Respiratory electron transport', 'databaseName': 'Reactome', 'id': 'R-MMU-611105'}, {'name': 'harzianum A and trichodermin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7714'}, {'name': 'limonene degradation III (to perillate)', 'databaseName': 'MetaCyc', 'id': 'PWY-6526'}, {'name': 'L-dopa degradation II (bacterial)', 'databaseName': 'MetaCyc', 'id': 'PWY-8110'}, {'name': 'Ion homeostasis', 'databaseName': 'Reactome', 'id': 'R-MMU-5578775'}, {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7606'}, {'name': 'sorgoleone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5987'}, {'name': 'tetrapyrrole biosynthesis I (from glutamate)', 'databaseName': 'MetaCyc', 'id': 'PWY-5188'}, {'name': 'Ion homeostasis', 'databaseName': 'Reactome', 'id': 'R-HSA-5578775'}, {'name': 'D-carnitine degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7471'}, {'name': 'cytosolic NADPH production (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7268'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-70370'}, {'name': '3,6-anhydro-alpha-L-galactopyranose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7562'}, {'name': 'anandamide biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8053'}, {'name': 'CMP-pseudaminate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6143'}, {'name': 'pyruvate fermentation to isobutanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7111'}, {'name': '(4S)-carveol and (4S)-dihydrocarveol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5927'}, {'name': 'Bicarbonate transporters', 'databaseName': 'Reactome', 'id': 'R-HSA-425381'}, {'name': 'pederin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8049'}, {'name': 'indole-3-acetate biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-581'}, {'name': 'pyruvate-derived opine biosynthesis II (marine invertebrates)', 'databaseName': 'MetaCyc', 'id': 'PWY-7351'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CFA-193048'}, {'name': 'CDP-ascarylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5830'}, {'name': '2-aminoethylphosphonate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6832'}, {'name': 'chaetoglobosin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7612'}, {'name': 'Wax biosynthesis', 'databaseName': 'Reactome', 'id': 'R-GGA-9640463'}, {'name': 'histamine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6181'}, {'name': '3-sulfopropanediol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8214'}, {'name': 'the visual cycle I (vertebrates)', 'databaseName': 'MetaCyc', 'id': 'PWY-6861'}, {'name': 'Streptococcus pneumoniae serotype 4 capsular polysaccharide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8461'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-DME-3769402'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-5652227'}, {'name': '3-chlorobenzoate degradation I (via chlorocatechol)', 'databaseName': 'MetaCyc', 'id': 'PWY-6088'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-GGA-173599'}, {'name': 'L-arabinose degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5517'}, {'name': '(4R)-4-[(E)-2-butenyl]-4-methyl-L-threonine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8441'}, {'name': 'eNOS activation', 'databaseName': 'Reactome', 'id': 'R-MMU-203615'}, {'name': 'alizarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5802'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-193144'}, {'name': 'CMP-legionaminate biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6749'}, {'name': 'L-arginine degradation XII', 'databaseName': 'MetaCyc', 'id': 'PWY-7523'}, {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7184'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-BTA-70268'}, {'name': 'virginiae butanolide type gamma-butyrolactones biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7881'}, {'name': 'p-cumate degradation to 2-hydroxypentadienoate', 'databaseName': 'MetaCyc', 'id': 'PWY-5163'}, {'name': 'Interconversion of nucleotide di- and triphosphates', 'databaseName': 'Reactome', 'id': 'R-MMU-499943'}, {'name': 'myo-inositol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2301'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-DDI-6798695'}, {'name': 'sulfoacetaldehyde degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6718'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-BTA-71403'}, {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7300'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-CEL-70171'}, {'name': '3-methylarginine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6511'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-7115'}, {'name': 'palmitate biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-8279'}, {'name': 'docosahexaenoate biosynthesis II (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-7585'}, {'name': 'representative whole plasma arachidonate-derived icosanoids II', 'databaseName': 'MetaCyc', 'id': 'PWY-8411'}, {'name': 'aurofusarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7695'}, {'name': 'alazopeptin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8293'}, {'name': 'Vitamin B5 (pantothenate) metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-199220'}, {'name': 'neurosporaxanthin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6681'}, {'name': 'arachidonate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8397'}, {'name': '1,2-dichlorobenzene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6090'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-DDI-5365859'}, {'name': 'kanamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7000'}, {'name': 'arachidonate biosynthesis I (6-desaturase, lower eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5353'}, {'name': 'selenate reduction', 'databaseName': 'MetaCyc', 'id': 'PWY-6932'}, {'name': 'D-galactose degradation I (Leloir pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-6317'}, {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors', 'databaseName': 'Reactome', 'id': 'R-DME-8866907'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-RNO-70171'}, {'name': 'apicidin F biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7668'}, {'name': 'benzoate biosynthesis III (CoA-dependent, non-beta-oxidative)', 'databaseName': 'MetaCyc', 'id': 'PWY-6446'}, {'name': 'L-threonate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7874'}, {'name': 'pyruvate fermentation to (R)-acetoin I', 'databaseName': 'MetaCyc', 'id': 'PWY-5938'}, {'name': 'palmitate biosynthesis II (type II fatty acid synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-5971'}, {'name': 'L-isoleucine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-5078'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-70370'}, {'name': 'pyrimidine deoxyribonucleotide phosphorylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7197'}, {'name': 'citronellol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6670'}, {'name': 'L-lysine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-2941'}, {'name': 'peptidoglycan cross-bridge biosynthesis II (E. faecium)', 'databaseName': 'MetaCyc', 'id': 'PWY-6461'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-2161541'}, {'name': 'Wax biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-9640463'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-SCE-70370'}, {'name': 'gibberellin biosynthesis IV (Gibberella fujikuroi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5047'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-DME-70895'}, {'name': 'ultra-long-chain fatty acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8041'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-2161541'}, {'name': 'wighteone and luteone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-4502'}, {'name': 'eupatolitin 3-O-glucoside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7157'}, {'name': \"UDP-N,N'-diacetylbacillosamine biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-7028'}, {'name': 'Ub-specific processing proteases', 'databaseName': 'Reactome', 'id': 'R-BTA-5689880'}, {'name': 'chloramphenicol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8032'}, {'name': 'photorespiration I', 'databaseName': 'MetaCyc', 'id': 'PWY-181'}, {'name': 'NiFe(CO)(CN)2 cofactor biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8319'}, {'name': 'dihydroconiferyl alcohol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6302'}, {'name': 'homophytochelatin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6841'}, {'name': 'Synthesis of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-RNO-77111'}, {'name': 'IM-2 type gamma-butyrolactones biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7882'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-SCE-5365859'}, {'name': '(+)-camphor biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6990'}, {'name': 'triethylamine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7085'}, {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-SPO-70635'}, {'name': 'monolignol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-361'}, {'name': 'L-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7295'}, {'name': 'galloylated catechin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7448'}, {'name': 'ChREBP activates metabolic gene expression', 'databaseName': 'Reactome', 'id': 'R-MMU-163765'}, {'name': 'apicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7667'}, {'name': 'alkane biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7033'}, {'name': 'Ca2+ activated K+ channels', 'databaseName': 'Reactome', 'id': 'R-RNO-1296052'}, {'name': 'leinamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8343'}, {'name': 'limonene degradation I (D-limonene)', 'databaseName': 'MetaCyc', 'id': 'PWY-5923'}, {'name': 'acetyl-CoA fermentation to butanoate', 'databaseName': 'MetaCyc', 'id': 'PWY-5676'}, {'name': '5-N-acetylardeemin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7543'}, {'name': 'quinoxaline-2-carboxylate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7734'}, {'name': 'adenosine ribonucleotides de novo biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7219'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-BTA-70263'}, {'name': 'limonene degradation IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8029'}, {'name': 'UDP-yelosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7842'}, {'name': '2-keto-L-gulonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5782'}, {'name': 'cholesterol biosynthesis (algae, late side-chain reductase)', 'databaseName': 'MetaCyc', 'id': 'PWY-8191'}, {'name': 'Platelet degranulation', 'databaseName': 'Reactome', 'id': 'R-CEL-114608'}, {'name': 'Synthesis of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-HSA-77111'}, {'name': 'UDP-N-acetyl-alpha-D-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7334'}, {'name': 'vanchrobactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7987'}, {'name': 'Biosynthesis of E-series 18(S)-resolvins', 'databaseName': 'Reactome', 'id': 'R-RNO-9018896'}, {'name': 'L-isoleucine biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-5103'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-HSA-2453902'}, {'name': '(+)-pisatin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2467'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-70370'}, {'name': 'Bicarbonate transporters', 'databaseName': 'Reactome', 'id': 'R-BTA-425381'}, {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)', 'databaseName': 'Reactome', 'id': 'R-RNO-2162123'}, {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors', 'databaseName': 'Reactome', 'id': 'R-DRE-8866904'}, {'name': 'bacilysin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7626'}, {'name': 'TP53 regulates transcription of several additional cell death genes whose specific roles in p53-dependent apoptosis remain uncertain', 'databaseName': 'Reactome', 'id': 'R-HSA-6803205'}, {'name': 'momilactone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7477'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-BTA-71384'}, {'name': 'ethene biosynthesis IV (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7126'}, {'name': 'norspermidine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6562'}, {'name': '2-allylmalonyl-CoA biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8427'}, {'name': 'pyrimidine deoxyribonucleotides biosynthesis from CTP', 'databaseName': 'MetaCyc', 'id': 'PWY-7210'}, {'name': 'bacteriochlorophyll e biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7760'}, {'name': 'prodigiosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7547'}, {'name': 'heptadecane biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6622'}, {'name': 'Beta-oxidation of very long chain fatty acids', 'databaseName': 'Reactome', 'id': 'R-DRE-390247'}, {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8017'}, {'name': 'tryptoquialanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7660'}, {'name': '3beta-hydroxysesquiterpene lactone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7497'}, {'name': 'hypotaurine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7387'}, {'name': 'cyanidin diglucoside biosynthesis (acyl-glucose dependent)', 'databaseName': 'MetaCyc', 'id': 'PWY-7256'}, {'name': 'L-pyrrolysine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6994'}, {'name': 'rutin degradation (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-7134'}, {'name': 'steviol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7070'}, {'name': 'anaerobic energy metabolism (invertebrates, cytosol)', 'databaseName': 'MetaCyc', 'id': 'PWY-7383'}, {'name': 'stearate biosynthesis I (animals)', 'databaseName': 'MetaCyc', 'id': 'PWY-5972'}, {'name': 'paromomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7018'}, {'name': 'septacidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8366'}, {'name': 'Acyl chain remodeling of CL', 'databaseName': 'Reactome', 'id': 'R-MMU-1482798'}, {'name': 'FR-900098 and FR-33289 antibiotics biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7419'}, {'name': 'even iso-branched-chain fatty acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8175'}, {'name': 'wax esters biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5885'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-RNO-156581'}, {'name': 'photorespiration III', 'databaseName': 'MetaCyc', 'id': 'PWY-8363'}, {'name': 'Phenylalanine metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-8964208'}, {'name': 'astaxanthin biosynthesis (flowering plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-7636'}, {'name': 'Acyl chain remodeling of CL', 'databaseName': 'Reactome', 'id': 'R-HSA-1482798'}, {'name': 'ceramide and sphingolipid recycling and degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7119'}, {'name': '3-chlorotoluene degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6103'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-DDI-1855183'}, {'name': 'pyoluteorin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7930'}, {'name': '3-chlorotoluene degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6104'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-75105'}, {'name': 'glutathione-mediated detoxification II', 'databaseName': 'MetaCyc', 'id': 'PWY-6842'}, {'name': 'hyperxanthone E biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8442'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-193144'}, {'name': \"6'-deoxychalcone metabolism\", 'databaseName': 'MetaCyc', 'id': 'PWY-5161'}, {'name': 'Prednisone ADME', 'databaseName': 'Reactome', 'id': 'R-BTA-9757110'}, {'name': 'arginomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7569'}, {'name': 'L-isoleucine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5101'}, {'name': 'coumestrol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6332'}, {'name': 'plasmalogen biosynthesis II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8335'}, {'name': 'peptidoglycan cross-bridge biosynthesis IV (Weissella viridescens)', 'databaseName': 'MetaCyc', 'id': 'PWY-6463'}, {'name': 'Synthesis of bile acids and bile salts via 27-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-MMU-193807'}, {'name': 'Utilization of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-HSA-77108'}, {'name': \"6,7,4'-trihydroxyisoflavone biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-5061'}, {'name': 'pyrethrin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6926'}, {'name': '(8E,10E)-dodeca-8,10-dienol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7654'}, {'name': 'cinnamate and 3-hydroxycinnamate degradation to 2-hydroxypentadienoate', 'databaseName': 'MetaCyc', 'id': 'PWY-6690'}, {'name': 'quinate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6416'}, {'name': 'phosphopantothenate biosynthesis III (archaea)', 'databaseName': 'MetaCyc', 'id': 'PWY-6654'}, {'name': 'vestitol and sativan biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5729'}, {'name': 'all trans undecaprenyl diphosphate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5816'}, {'name': 'glycolysis VI (from fructose)', 'databaseName': 'MetaCyc', 'id': 'PWY-8404'}, {'name': 'bacteriochlorophyll c biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7759'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-SPO-196757'}, {'name': 'Shigella boydii serotype 6 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8232'}, {'name': 'type I lipoteichoic acid biosynthesis (S. aureus)', 'databaseName': 'MetaCyc', 'id': 'PWY-7817'}, {'name': 'sulfolactate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6637'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-HSA-70171'}, {'name': 'maysin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5793'}, {'name': 'Signaling by TCF7L2 mutants', 'databaseName': 'Reactome', 'id': 'R-HSA-5339700'}, {'name': 'L-ascorbate degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6704'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-HSA-3769402'}, {'name': 'dechlorogriseofulvin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7655'}, {'name': 'meleagrin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7607'}, {'name': 'CDP-6-deoxy-D-gulose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8139'}, {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77352'}, {'name': '(R)- and (S)-3-hydroxybutanoate biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7216'}, {'name': 'putrescine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-0'}, {'name': 'Cholesterol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SPO-191273'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NADP-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-241'}, {'name': 'GDP-fucose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-6787639'}, {'name': '1,5-anhydrofructose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6992'}, {'name': 'RHOD GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-RNO-9013405'}, {'name': 'hypericin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5780'}, {'name': 'geraniol and nerol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6678'}, {'name': '7-dehydroporiferasterol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7155'}, {'name': 'Complex I biogenesis', 'databaseName': 'Reactome', 'id': 'R-RNO-6799198'}, {'name': 'PPARA activates gene expression', 'databaseName': 'Reactome', 'id': 'R-HSA-1989781'}, {'name': 'superpathway of pterocarpan biosynthesis (via formononetin)', 'databaseName': 'MetaCyc', 'id': 'PWY-2229'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-SCE-2453902'}, {'name': '(-)-maackiain biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2464'}, {'name': 'fumagillin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8407'}, {'name': 'noscapine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7138'}, {'name': 'protein SAMPylation and SAMP-mediated thiolation', 'databaseName': 'MetaCyc', 'id': 'PWY-7887'}, {'name': 'GDP-fucose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-6787639'}, {'name': 'Sperm Motility And Taxes', 'databaseName': 'Reactome', 'id': 'R-CEL-1300642'}, {'name': 'L-methionine biosynthesis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7977'}, {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors', 'databaseName': 'Reactome', 'id': 'R-DRE-8866907'}, {'name': 'phaselate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6320'}, {'name': 'cGMP effects', 'databaseName': 'Reactome', 'id': 'R-HSA-418457'}, {'name': 'pyruvate fermentation to hexanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6863'}, {'name': 'yersiniabactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6407'}, {'name': 'retinol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6857'}, {'name': 'gibberellin inactivation III (epoxidation)', 'databaseName': 'MetaCyc', 'id': 'PWY-6494'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-SPO-5628897'}, {'name': 'chitin degradation I (archaea)', 'databaseName': 'MetaCyc', 'id': 'PWY-6855'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-DME-2161541'}, {'name': 'Complex I biogenesis', 'databaseName': 'Reactome', 'id': 'R-BTA-6799198'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-1614635'}, {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)', 'databaseName': 'Reactome', 'id': 'R-BTA-2162123'}, {'name': '3-hydroxypropanoate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-5743'}, {'name': 'Utilization of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-RNO-77108'}, {'name': 'L-lysine biosynthesis V', 'databaseName': 'MetaCyc', 'id': 'PWY-3081'}, {'name': 'heme b biosynthesis III (from siroheme)', 'databaseName': 'MetaCyc', 'id': 'PWY-7552'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-MMU-156581'}, {'name': 'furaneol and mesifurane biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5975'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-1614635'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-SSC-8964539'}, {'name': 'tylosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7415'}, {'name': 'papaverine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7363'}, {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6946'}, {'name': 'methyl tert-butyl ether degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7779'}, {'name': 'D-erythronate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7873'}, {'name': 'beta-alanine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8120'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-193048'}, {'name': 'Phase I - Functionalization of compounds', 'databaseName': 'Reactome', 'id': 'R-RNO-211945'}, {'name': 'aloin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6312'}, {'name': 'CDP-diacylglycerol biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-5981'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-977347'}, {'name': 'gliotoxin inactivation', 'databaseName': 'MetaCyc', 'id': 'PWY-7534'}, {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8014'}, {'name': 'neopentalenoketolactone and pentalenate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6919'}, {'name': 'CDP-diacylglycerol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5667'}, {'name': 'heme degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-5874'}, {'name': 'aureobasidin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7665'}, {'name': 'Biosynthesis of D-series resolvins', 'databaseName': 'Reactome', 'id': 'R-HSA-9018676'}, {'name': 'indole-3-acetate activation II', 'databaseName': 'MetaCyc', 'id': 'PWY-1921'}, {'name': 'benzene degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5450'}, {'name': 'glycine lipid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8082'}, {'name': 'Cytoprotection by HMOX1', 'databaseName': 'Reactome', 'id': 'R-RNO-9707564'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-DME-71403'}, {'name': 'mannitol cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6531'}, {'name': 'Biosynthesis of D-series resolvins', 'databaseName': 'Reactome', 'id': 'R-MMU-9018676'}, {'name': 'aspyridone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7565'}, {'name': 'phosalacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7769'}, {'name': 'hinokiresinol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7632'}, {'name': 'Escherichia coli serotype O:117 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8220'}, {'name': 'luteolinidin 5-O-glucoside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7252'}, {'name': 'juvenile hormone III biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6575'}, {'name': 'kievitone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-4681'}, {'name': 'GDP-fucose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CEL-6787639'}, {'name': 'luteolin triglucuronide degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7445'}, {'name': 'propane degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7775'}, {'name': 'GDP-fucose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DME-6787639'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-SCE-156581'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-SCE-8964539'}, {'name': 'sterigmatocystin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5956'}, {'name': 'bacillibactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5903'}, {'name': 'naringenin biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7397'}, {'name': '8-O-methylfusarubin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7689'}, {'name': '4-ethylphenol degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6080'}, {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7187'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-MMU-3769402'}, {'name': 'D-xylose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7294'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-1614635'}, {'name': 'bacteriochlorophyll d biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7758'}, {'name': 'UDP-N-acetyl-D-galactosamine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5514'}, {'name': 'dTDP-beta-D-fucofuranose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7312'}, {'name': '3-hydroxy-L-homotyrosine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7649'}, {'name': \"myxol-2' fucoside biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6279'}, {'name': 'glycerol degradation to butanol', 'databaseName': 'MetaCyc', 'id': 'PWY-7003'}, {'name': 'Beta-oxidation of pristanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-389887'}, {'name': 'farnesylcysteine salvage pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6577'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SPO-5652227'}, {'name': 'CLEC7A (Dectin-1) induces NFAT activation', 'databaseName': 'Reactome', 'id': 'R-HSA-5607763'}, {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77348'}, {'name': 'L-methionine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5082'}, {'name': 'sorbitol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5530'}, {'name': 'superpathway of glyoxylate cycle and fatty acid degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-561'}, {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6945'}, {'name': 'cyanidin 3,7-diglucoside polyacylation biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7464'}, {'name': 'homofuraneol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7910'}, {'name': 'GDP-L-fucose biosynthesis I (from GDP-D-mannose)', 'databaseName': 'MetaCyc', 'id': 'PWY-66'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7726'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-CEL-3769402'}, {'name': 'ascorbate recycling (cytosolic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6370'}, {'name': 'brassinosteroids inactivation', 'databaseName': 'MetaCyc', 'id': 'PWY-6546'}, {'name': 'Synthesis of bile acids and bile salts via 7alpha-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-RNO-193368'}, {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA', 'databaseName': 'Reactome', 'id': 'R-CEL-77352'}, {'name': 'mannitol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-3861'}, {'name': 'aloenin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6310'}, {'name': 'patulin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7490'}, {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7440'}, {'name': 'ecdysteroid metabolism (arthropods)', 'databaseName': 'MetaCyc', 'id': 'PWY-7321'}, {'name': 'ternatin C3 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7280'}, {'name': 'adipate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-8354'}, {'name': 'D-arabinose degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-8334'}, {'name': 'soybean saponin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5203'}, {'name': 'oleandomycin activation/inactivation', 'databaseName': 'MetaCyc', 'id': 'PWY-6972'}, {'name': '2,3-dihydroxybenzoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5901'}, {'name': 'UDP-alpha-D-galactose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7344'}, {'name': 'capsiconiate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6027'}, {'name': 'formaldehyde assimilation I (serine pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1622'}, {'name': 'naphthalene degradation (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5427'}, {'name': 'tabtoxinine-beta-lactam biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7954'}, {'name': 'neoxaline biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7608'}, {'name': 'paromamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7014'}, {'name': 'Prednisone ADME', 'databaseName': 'Reactome', 'id': 'R-DRE-9757110'}, {'name': 'di-homo-gamma-linolenate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8396'}, {'name': 'divinyl ether biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5409'}, {'name': '(-)-dehydrodiconiferyl alcohol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7557'}, {'name': 'Defective AHCY causes HMAHCHD', 'databaseName': 'Reactome', 'id': 'R-HSA-5578997'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-SSC-9033241'}, {'name': 'shikimate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6419'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-SPO-71384'}, {'name': 'pyruvate fermentation to acetate V', 'databaseName': 'MetaCyc', 'id': 'PWY-5537'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-GGA-70268'}, {'name': 'A-factor gamma-butyrolactone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7878'}, {'name': 'Transferrin endocytosis and recycling', 'databaseName': 'Reactome', 'id': 'R-RNO-917977'}, {'name': 'GDP-L-colitose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5740'}, {'name': 'fatty acid beta-oxidation III (unsaturated, odd number)', 'databaseName': 'MetaCyc', 'id': 'PWY-5137'}, {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7610'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-MMU-3899300'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-MMU-71384'}, {'name': 'guanosine ribonucleotides de novo biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7221'}, {'name': 'dTDP-alpha-D-forosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6808'}, {'name': 'daunorubicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7352'}, {'name': 'Sphingolipid de novo biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SCE-1660661'}, {'name': 'stearate biosynthesis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-8280'}, {'name': 'aureothin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8033'}, {'name': 'Beta-oxidation of pristanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-389887'}, {'name': 'UDP-2-acetamido-4-amino-2,4,6-trideoxy-alpha-D-galactose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7467'}, {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors', 'databaseName': 'Reactome', 'id': 'R-DME-8866904'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-BTA-71064'}, {'name': 'trans-zeatin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2681'}, {'name': 'L-tyrosine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-3461'}, {'name': 'L-arabinose degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-8329'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-MMU-173599'}, {'name': 'glycolysis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-1042'}, {'name': 'dTDP-6-deoxy-alpha-D-allose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7413'}, {'name': 'chlorophyll cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-5068'}, {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5882'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-HSA-6798695'}, {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7301'}, {'name': 'adenosine deoxyribonucleotides de novo biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7220'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-MMU-1483166'}, {'name': 'bacteriochlorophyll a biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5526'}, {'name': 'ochratoxin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8116'}, {'name': 'UTP and CTP de novo biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7176'}, {'name': 'Regulation of insulin secretion', 'databaseName': 'Reactome', 'id': 'R-HSA-422356'}, {'name': 'Wax biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-9640463'}, {'name': 'terrequinone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7520'}, {'name': 'Tetrahydrobiopterin (BH4) synthesis, recycling, salvage and regulation', 'databaseName': 'Reactome', 'id': 'R-DDI-1474151'}, {'name': 'Phenylalanine metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-8964208'}, {'name': 'folate transformations I', 'databaseName': 'MetaCyc', 'id': 'PWY-2201'}, {'name': '(-)-medicarpin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2463'}, {'name': 'ascomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8424'}, {'name': 'linustatin bioactivation', 'databaseName': 'MetaCyc', 'id': 'PWY-7091'}, {'name': 'staphyloferrin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8008'}, {'name': 'xanthohumol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5135'}, {'name': 'cob(II)yrinate a,c-diamide biosynthesis I (early cobalt insertion)', 'databaseName': 'MetaCyc', 'id': 'PWY-7377'}, {'name': 'stephacidin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7603'}, {'name': 'Heme degradation', 'databaseName': 'Reactome', 'id': 'R-BTA-189483'}, {'name': 'bacterial bioluminescence', 'databaseName': 'MetaCyc', 'id': 'PWY-7723'}, {'name': 'Beta oxidation of myristoyl-CoA to lauroyl-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77285'}, {'name': 'Complex I biogenesis', 'databaseName': 'Reactome', 'id': 'R-MMU-6799198'}, {'name': 'L-glutamate degradation X', 'databaseName': 'MetaCyc', 'id': 'PWY-5766'}, {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)', 'databaseName': 'MetaCyc', 'id': 'PWY-7615'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-SPO-6798695'}, {'name': 'Beta-oxidation of pristanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-389887'}, {'name': 'pectin degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7248'}, {'name': 'acrylate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6373'}, {'name': 'bixin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5305'}, {'name': 'L-ascorbate biosynthesis I (plants, L-galactose pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-882'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-DDI-71336'}, {'name': 'brassinolide biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-699'}, {'name': '2-methylpropene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7778'}, {'name': 'sphingolipid biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-5129'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-CEL-5661270'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-HSA-9033241'}, {'name': 'hydroxycinnamate sugar acid ester biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7461'}, {'name': 'alginate biosynthesis I (algal)', 'databaseName': 'MetaCyc', 'id': 'PWY-6073'}, {'name': 'D-apionate degradation I (xylose isomerase family decarboxylase)', 'databaseName': 'MetaCyc', 'id': 'PWY-8091'}, {'name': 'arabidopyrone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7682'}, {'name': 'Transcriptional activation of mitochondrial biogenesis', 'databaseName': 'Reactome', 'id': 'R-MMU-2151201'}, {'name': 'Synthesis of Lipoxins (LX)', 'databaseName': 'Reactome', 'id': 'R-HSA-2142700'}, {'name': 'griseofulvin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7653'}, {'name': 'homotaurine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6321'}, {'name': 'nepetalactone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8069'}, {'name': 'mupirocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8012'}, {'name': 'acyl-[acyl-carrier protein] thioesterase pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-5142'}, {'name': 'Ub-specific processing proteases', 'databaseName': 'Reactome', 'id': 'R-RNO-5689880'}, {'name': 'phosphatidylinositol biosynthesis I (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-6580'}, {'name': 'erinacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8408'}, {'name': 'jasmonic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-735'}, {'name': 'flavonol acylglucoside biosynthesis II - isorhamnetin derivatives', 'databaseName': 'MetaCyc', 'id': 'PWY-7171'}, {'name': 'Prednisone ADME', 'databaseName': 'Reactome', 'id': 'R-RNO-9757110'}, {'name': 'Amaryllidacea alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7826'}, {'name': 'pyruvate fermentation to (S)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-5481'}, {'name': 'Biosynthesis of E-series 18(S)-resolvins', 'databaseName': 'Reactome', 'id': 'R-MMU-9018896'}, {'name': '2-amino-3-hydroxycyclopent-2-enone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7536'}, {'name': 'Beta oxidation of palmitoyl-CoA to myristoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77305'}, {'name': 'acridone alkaloid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5958'}, {'name': 'anthocyanidin acylglucoside and acylsambubioside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7679'}, {'name': 'ferulate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6343'}, {'name': 'fusicoccin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6659'}, {'name': 'lovastatin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7535'}, {'name': 'icosapentaenoate biosynthesis I (lower eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-6958'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-DME-70171'}, {'name': 'gallate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6707'}, {'name': 'formaldehyde oxidation VI (H4MPT pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1723'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-CEL-70895'}, {'name': 'L-arginine biosynthesis IV (archaea)', 'databaseName': 'MetaCyc', 'id': 'PWY-7400'}, {'name': 'quebrachitol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8078'}, {'name': 'actinomycin D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7718'}, {'name': '2,5-dichlorotoluene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6191'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-SSC-5365859'}, {'name': 'pentagalloylglucose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5475'}, {'name': 'calystegine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5318'}, {'name': 'Utilization of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-BTA-77108'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-DME-1614635'}, {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7304'}, {'name': 'bile acid 7beta-dehydroxylation', 'databaseName': 'MetaCyc', 'id': 'PWY-8134'}, {'name': 'TYSND1 cleaves peroxisomal proteins', 'databaseName': 'Reactome', 'id': 'R-HSA-9033500'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-70895'}, {'name': 'S-benzyl-L-cysteine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-8418'}, {'name': '3-methylbutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6871'}, {'name': 'siroheme biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5194'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-GGA-70263'}, {'name': 'gibberellin biosynthesis III (early C-13 hydroxylation)', 'databaseName': 'MetaCyc', 'id': 'PWY-5035'}, {'name': '3-dehydroquinate biosynthesis II (archaea)', 'databaseName': 'MetaCyc', 'id': 'PWY-6160'}, {'name': '3-hydroxyquinaldate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7733'}, {'name': 'Ub-specific processing proteases', 'databaseName': 'Reactome', 'id': 'R-DRE-5689880'}, {'name': 'dTDP-alpha-D-ravidosamine and dTDP-4-acetyl-alpha-D-ravidosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7688'}, {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7613'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-DME-77346'}, {'name': 'nivalenol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7713'}, {'name': 'mycinamicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7412'}, {'name': 'CDP-ribitol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8416'}, {'name': 'gamma-coniciene and coniine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5748'}, {'name': 'equisetin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7669'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-XTR-1483166'}, {'name': 'docosahexaenoate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8400'}, {'name': 'L-threonine degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-5436'}, {'name': 'ethylmalonyl-CoA pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-5741'}, {'name': 'glycerol-3-phosphate shuttle', 'databaseName': 'MetaCyc', 'id': 'PWY-6118'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-DRE-156581'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-MMU-5365859'}, {'name': 'acylceramide biosynthesis and processing', 'databaseName': 'MetaCyc', 'id': 'PWY-8042'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-BTA-1614635'}, {'name': 'mannojirimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7531'}, {'name': 'sulfolactate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6638'}, {'name': 'L-glutamate degradation XI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8190'}, {'name': 'dTDP-L-daunosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7814'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-70268'}, {'name': 'phosphatidylcholine biosynthesis VII', 'databaseName': 'MetaCyc', 'id': 'PWY-7470'}, {'name': 'erythritol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7788'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-GGA-70171'}, {'name': 'alpha-linolenic acid (ALA) metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-2046106'}, {'name': 'androstenedione degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6944'}, {'name': 'RHOF GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-RNO-9035034'}, {'name': 'tenellin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7561'}, {'name': 'icosapentaenoate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8399'}, {'name': 'dTDP-D-desosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6942'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-HSA-173599'}, {'name': 'D-glucuronate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-5525'}, {'name': 'viridicatumtoxin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7659'}, {'name': 'resveratrol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-84'}, {'name': 'dehydrophos biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6682'}, {'name': 'Ca2+ activated K+ channels', 'databaseName': 'Reactome', 'id': 'R-HSA-1296052'}, {'name': 'aspoquinolone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8388'}, {'name': 'myo-, chiro- and scyllo-inositol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7237'}, {'name': 'Synthesis of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-MMU-77111'}, {'name': 'Mycothiol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MTU-879299'}, {'name': 'crotonyl-CoA/ethylmalonyl-CoA/hydroxybutyryl-CoA cycle (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7854'}, {'name': 'vibriobactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6374'}, {'name': 'diazepinomicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8382'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77346'}, {'name': '4-oxopentanoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7948'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-SCE-77346'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosa-4,7,10,13,16-pentaenoate biosynthesis II (4-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7728'}, {'name': 'CMP phosphorylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7205'}, {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6626'}, {'name': 'fatty acid biosynthesis initiation (type II)', 'databaseName': 'MetaCyc', 'id': 'PWY-4381'}, {'name': 'Escherichia coli serotype O:128 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8222'}, {'name': 'oleate beta-oxidation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7307'}, {'name': 'sanguinarine and macarpine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5287'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-RNO-71336'}, {'name': 'mycobacterial sulfolipid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7746'}, {'name': 'cyclohexane-1-carboxylate degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-301'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-CEL-70370'}, {'name': 'Beta-oxidation of very long chain fatty acids', 'databaseName': 'Reactome', 'id': 'R-SCE-390247'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-DRE-9033241'}, {'name': 'CDP-4-dehydro-3,6-dideoxy-D-glucose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5833'}, {'name': 'cholesterol degradation to androstenedione III (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8151'}, {'name': 'methylerythritol phosphate pathway II', 'databaseName': 'MetaCyc', 'id': 'PWY-7560'}, {'name': '8-amino-7-oxononanoate biosynthesis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-8203'}, {'name': 'tetrahydrofolate salvage from 5,10-methenyltetrahydrofolate', 'databaseName': 'MetaCyc', 'id': 'PWY-6613'}, {'name': '3-hydroxypropanoate/4-hydroxybutanate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-5789'}, {'name': 'cuticular wax biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-282'}, {'name': 'docosahexaenoate biosynthesis I (lower eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-7053'}, {'name': 'benzoate degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-2503'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-CEL-173599'}, {'name': 'Beta-oxidation of very long chain fatty acids', 'databaseName': 'Reactome', 'id': 'R-MMU-390247'}, {'name': 'ChREBP activates metabolic gene expression', 'databaseName': 'Reactome', 'id': 'R-RNO-163765'}, {'name': 'propanoate fermentation to 2-methylbutanoate', 'databaseName': 'MetaCyc', 'id': 'PWY-5109'}, {'name': '1,3,6,7-tetrahydroxyxanthone biosynthesis (from benzoyl-CoA)', 'databaseName': 'MetaCyc', 'id': 'PWY-5001'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-CEL-71384'}, {'name': 'cichoriin interconversion', 'databaseName': 'MetaCyc', 'id': 'PWY-7057'}, {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7340'}, {'name': 'Platelet degranulation', 'databaseName': 'Reactome', 'id': 'R-MMU-114608'}, {'name': '9-lipoxygenase and 9-hydroperoxide lyase pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-5408'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-RNO-5628897'}, {'name': '(S)-camphor degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6989'}, {'name': 'RHOF GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-9035034'}, {'name': 'L-arabinose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-5515'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-SCE-9033241'}, {'name': 'adenosine deoxyribonucleotides de novo biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7227'}, {'name': 'butirosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7019'}, {'name': 'valproate beta-oxidation', 'databaseName': 'MetaCyc', 'id': 'PWY-8182'}, {'name': 'seleno-amino acid biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-6936'}, {'name': 'ergotamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6495'}, {'name': 'L-lysine degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-5283'}, {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-CEL-77310'}, {'name': 'raspberry ketone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5393'}, {'name': 'succinate fermentation to butanoate', 'databaseName': 'MetaCyc', 'id': 'PWY-5677'}, {'name': 'D-glucuronate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6501'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-DDI-70171'}, {'name': 'colanic acid (Escherichia coli K12) biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8243'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-SPO-156581'}, {'name': 'cinnamoyl-CoA biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6457'}, {'name': 'Metabolism of ingested SeMet, Sec, MeSec into H2Se', 'databaseName': 'Reactome', 'id': 'R-HSA-2408508'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-RNO-70263'}, {'name': 'Platelet degranulation', 'databaseName': 'Reactome', 'id': 'R-HSA-114608'}, {'name': 'poly(ribitol phosphate) wall teichoic acid biosynthesis II (S. aureus)', 'databaseName': 'MetaCyc', 'id': 'PWY-7816'}, {'name': 'cylindrospermopsin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8045'}, {'name': 'L-arginine biosynthesis III (via N-acetyl-L-citrulline)', 'databaseName': 'MetaCyc', 'id': 'PWY-5154'}, {'name': 'ferrichrome A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7571'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-389661'}, {'name': 'linoleate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8395'}, {'name': 'sterol biosynthesis (methylotrophs)', 'databaseName': 'MetaCyc', 'id': 'PWY-8026'}, {'name': 'cis-vaccenate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5973'}, {'name': 'CMP-diacetamido-8-epilegionaminic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7719'}, {'name': 'holomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7690'}, {'name': 'marneral biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6005'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-MMU-2453902'}, {'name': 'ferulate and sinapate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5168'}, {'name': 'reductive TCA cycle II', 'databaseName': 'MetaCyc', 'id': 'PWY-5392'}, {'name': 'adipate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8347'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-70370'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-SCE-5628897'}, {'name': 'D-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7786'}, {'name': 'oryzalexin A, B, and C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7489'}, {'name': 'taxol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5660'}, {'name': 'versicolorin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5955'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-SSC-70171'}, {'name': 'dTDP-beta-L-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6976'}, {'name': 'citreoisocoumarin and bikisocoumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7696'}, {'name': 'superpathway of nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7342'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-RNO-9033241'}, {'name': 'chorismate biosynthesis from 3-dehydroquinate', 'databaseName': 'MetaCyc', 'id': 'PWY-6163'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-MMU-5661270'}, {'name': 'Sphingolipid de novo biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-1660661'}, {'name': \"abscisic acid degradation to 7'-hydroxyabscisate\", 'databaseName': 'MetaCyc', 'id': 'PWY-7642'}, {'name': 'Synthesis of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-DRE-77111'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-DDI-71403'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-SCE-71384'}, {'name': 'pyruvate fermentation to acetate VI', 'databaseName': 'MetaCyc', 'id': 'PWY-5538'}, {'name': 'sciadonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6598'}, {'name': 'cyclosporin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8443'}, {'name': 'guanosine deoxyribonucleotides de novo biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7226'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-SSC-71403'}, {'name': 'flavonoid biosynthesis (in equisetum)', 'databaseName': 'MetaCyc', 'id': 'PWY-6787'}, {'name': 'L-rhamnose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6713'}, {'name': '3-sulfopropanediol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6634'}, {'name': 'beta-D-mannosyl phosphomycoketide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7740'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-SPO-77346'}, {'name': 'L-lysine biosynthesis VI', 'databaseName': 'MetaCyc', 'id': 'PWY-5097'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-BTA-4641265'}, {'name': 'Escherichia coli serotype O:169 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8233'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-193048'}, {'name': 'arachidonate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7592'}, {'name': 'Escherichia coli serotype O:55/Salmonella enterica serotype O:50 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8217'}, {'name': 'pinitol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6738'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-8964539'}, {'name': '6-diazo-5-oxo-L-norleucine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8297'}, {'name': 'Ub-specific processing proteases', 'databaseName': 'Reactome', 'id': 'R-HSA-5689880'}, {'name': 'Salmonella enterica serotype O:18 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8206'}, {'name': 'detoxification of reactive carbonyls in chloroplasts', 'databaseName': 'MetaCyc', 'id': 'PWY-6786'}, {'name': 'L-tryptophan degradation V (side chain pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-3162'}, {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77352'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-RNO-3899300'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': 'testosterone and androsterone degradation to androstendione (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6943'}, {'name': 'Defective GALE causes EDG', 'databaseName': 'Reactome', 'id': 'R-HSA-5609977'}, {'name': 'androstenedione degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8152'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-BTA-196757'}, {'name': 'echinocandin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7650'}, {'name': 'orientin and isoorientin biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7188'}, {'name': 'tetrahydropteridine recycling', 'databaseName': 'MetaCyc', 'id': 'PWY-8099'}, {'name': 'L-fucose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8444'}, {'name': 'Beta oxidation of palmitoyl-CoA to myristoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77305'}, {'name': 'L-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7787'}, {'name': 'blasticidin S biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7570'}, {'name': 'neolinustatin bioactivation', 'databaseName': 'MetaCyc', 'id': 'PWY-7092'}, {'name': 'L-phenylalanine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-3462'}, {'name': 'rifamycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5984'}, {'name': 'spermidine hydroxycinnamic acid conjugates biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6442'}, {'name': 'bile acid biosynthesis, neutral pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6061'}, {'name': 'Escherichia coli serotype O:149/Shigella boydii serotype O1 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8253'}, {'name': 'serotonin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6313'}, {'name': 'Bicarbonate transporters', 'databaseName': 'Reactome', 'id': 'R-DME-425381'}, {'name': 'heme d1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7554'}, {'name': 'methyl ketone biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7007'}, {'name': 'Escherichia coli serotype O:104 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7530'}, {'name': 'GDP-mycosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7573'}, {'name': 'podophyllotoxin glucosides metabolism', 'databaseName': 'MetaCyc', 'id': 'PWY-7491'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-BTA-156581'}, {'name': 'grayanate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8326'}, {'name': 'Retinoid metabolism and transport', 'databaseName': 'Reactome', 'id': 'R-MMU-975634'}, {'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'GDP-L-galactose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5115'}, {'name': 'Lipid particle organization', 'databaseName': 'Reactome', 'id': 'R-RNO-8964572'}, {'name': 'L-leucine degradation IV (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-7767'}, {'name': 'reductive glycine pathway of autotrophic CO2 fixation', 'databaseName': 'MetaCyc', 'id': 'PWY-8303'}, {'name': 'caffeoylglucarate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6673'}, {'name': 'Cytoprotection by HMOX1', 'databaseName': 'Reactome', 'id': 'R-HSA-9707564'}, {'name': 'fatty acid biosynthesis initiation (plant mitochondria)', 'databaseName': 'MetaCyc', 'id': 'PWY-6799'}, {'name': 'Ion homeostasis', 'databaseName': 'Reactome', 'id': 'R-RNO-5578775'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-GGA-5661270'}, {'name': 'FCGR3A-mediated IL10 synthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-9664323'}, {'name': 'L-glucose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7130'}, {'name': 'sulfazecin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8381'}, {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77348'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-193144'}, {'name': 'L-proline biosynthesis III (from L-ornithine)', 'databaseName': 'MetaCyc', 'id': 'PWY-3341'}, {'name': 'benzoyl-CoA biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6458'}, {'name': 'petroselinate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5367'}, {'name': 'TP53 Regulates Transcription of Genes Involved in Cytochrome C Release', 'databaseName': 'Reactome', 'id': 'R-MMU-6803204'}, {'name': 'Prednisone ADME', 'databaseName': 'Reactome', 'id': 'R-MMU-9757110'}, {'name': 'yeast mannan degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-8435'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-DME-196757'}, {'name': 'Tetrahydrobiopterin (BH4) synthesis, recycling, salvage and regulation', 'databaseName': 'Reactome', 'id': 'R-RNO-1474151'}, {'name': 'phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7742'}, {'name': 'L-arginine degradation XIV (oxidative Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-6344'}, {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors', 'databaseName': 'Reactome', 'id': 'R-HSA-8866907'}, {'name': 'carminate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8447'}, {'name': 'dapdiamides biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7706'}, {'name': '2-deoxy-D-ribose degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-8060'}, {'name': 'dTDP-3-acetamido-3,6-dideoxy-alpha-D-glucose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7318'}, {'name': 'anguibactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7986'}, {'name': 'chitin derivatives degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6906'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-977347'}, {'name': 'UDP-alpha-D-galacturonate biosynthesis I (from UDP-D-glucuronate)', 'databaseName': 'MetaCyc', 'id': 'PWY-4861'}, {'name': 'Lipid particle organization', 'databaseName': 'Reactome', 'id': 'R-SCE-8964572'}, {'name': 'Wax biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DME-9640463'}, {'name': 'superpathway of mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6113'}, {'name': 'benzoyl-CoA degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-1361'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-PFA-1855183'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-PFA-71336'}, {'name': 't-anethole biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5867'}, {'name': 'D-galacturonate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6486'}, {'name': 'chitin deacetylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7118'}, {'name': 'aurachin A, B, C and D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7407'}, {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77310'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-977347'}, {'name': 'L-tryptophan degradation XI (mammalian, via kynurenine)', 'databaseName': 'MetaCyc', 'id': 'PWY-6309'}, {'name': 'gamma-butyrobetaine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8339'}, {'name': 'D-xylose degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6760'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-CEL-8964539'}, {'name': 'glucosylglycerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7902'}, {'name': 'validamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5818'}, {'name': 'Sphingolipid de novo biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SPO-1660661'}, {'name': 'stearate biosynthesis II (bacteria and plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-5989'}, {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-BTA-70635'}, {'name': 'poly(ribitol phosphate) wall teichoic acid biosynthesis I (B. subtilis)', 'databaseName': 'MetaCyc', 'id': 'PWY-7815'}, {'name': 'starch degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-842'}, {'name': 'Role of phospholipids in phagocytosis', 'databaseName': 'Reactome', 'id': 'R-HSA-2029485'}, {'name': 'secologanin and strictosidine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5290'}, {'name': 'superpathway of dTDP-glucose-derived O-antigen building blocks biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7317'}, {'name': 'superpathway of UDP-N-acetylglucosamine-derived O-antigen building blocks biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7332'}, {'name': 'paxilline and diprenylpaxilline biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7493'}, {'name': 'malonate degradation I (biotin-independent)', 'databaseName': 'MetaCyc', 'id': 'PWY-5794'}, {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7333'}, {'name': '4-hydroxy-4-methyl-L-glutamate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7701'}, {'name': 'lotaustralin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6002'}, {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)', 'databaseName': 'MetaCyc', 'id': 'PWY-6053'}, {'name': 'The retinoid cycle in cones (daylight vision)', 'databaseName': 'Reactome', 'id': 'R-SCE-2187335'}, {'name': 'juglone degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7825'}, {'name': 'Mineralocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-193993'}, {'name': 'RHOQ GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-XTR-9013406'}, {'name': 'Dimycocersyl phthiocerol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MTU-9635470'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-BTA-1855183'}, {'name': 'Sphingolipid de novo biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DDI-1660661'}, {'name': 'deoxynivalenol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7712'}, {'name': 'ajmaline and sarpagine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5301'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-71064'}, {'name': 'L-proline biosynthesis II (from arginine)', 'databaseName': 'MetaCyc', 'id': 'PWY-4981'}, {'name': 'CDP-tyvelose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5834'}, {'name': 'dTDP-alpha-D-mycaminose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7414'}, {'name': 'L-leucine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5076'}, {'name': 'matairesinol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5466'}, {'name': 'cucurbitacin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8434'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-GGA-373920'}, {'name': 'lipoate biosynthesis and incorporation IV (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7382'}, {'name': 'Entner-Doudoroff pathway I', 'databaseName': 'MetaCyc', 'id': 'PWY-8004'}, {'name': 'streptovaricin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8035'}, {'name': 'naphthalene degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-7620'}, {'name': 'arachidonate biosynthesis II (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-7583'}, {'name': 'bisphenol A degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7757'}, {'name': 'Phase I - Functionalization of compounds', 'databaseName': 'Reactome', 'id': 'R-MMU-211945'}, {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-70635'}, {'name': '2-O-acetyl-3-O-trans-coutarate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7460'}, {'name': 'ChREBP activates metabolic gene expression', 'databaseName': 'Reactome', 'id': 'R-DDI-163765'}, {'name': 'sphingofungin B-D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8464'}, {'name': 'UDP-3-acetamido-2-amino-2,3-dideoxy-alpha-D-glucopyranose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8273'}, {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77350'}, {'name': 'abscisic acid degradation to neophaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-7640'}, {'name': 'pyruvate fermentation to butanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-6583'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-194002'}, {'name': 'brassinolide biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-2582'}, {'name': 'anandamide biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-8051'}, {'name': 'RHOD GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-9013405'}, {'name': '(-)-microperfuranone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7556'}, {'name': 'Phenylalanine metabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-8964208'}, {'name': 'UDP-N-acetyl-beta-L-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7330'}, {'name': \"(1'S,5'S)-averufin biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-5954'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-CEL-5365859'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-71064'}, {'name': 'cyclohexane-1-carboxyl-CoA biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8044'}, {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids', 'databaseName': 'Reactome', 'id': 'R-CEL-77288'}, {'name': 'Respiratory electron transport', 'databaseName': 'Reactome', 'id': 'R-BTA-611105'}, {'name': 'fumiquinazoline D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7542'}, {'name': 'representative whole plasma arachidonate-derived icosanoids I', 'databaseName': 'MetaCyc', 'id': 'PWY-8410'}, {'name': 'glycolysis II (from fructose 6-phosphate)', 'databaseName': 'MetaCyc', 'id': 'PWY-5484'}, {'name': 'reductive acetyl coenzyme A pathway II (autotrophic methanogens)', 'databaseName': 'MetaCyc', 'id': 'PWY-7784'}, {'name': 'proanthocyanidins biosynthesis from flavanols', 'databaseName': 'MetaCyc', 'id': 'PWY-641'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-DME-70370'}, {'name': 'Synthesis of bile acids and bile salts via 7alpha-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-DME-193368'}, {'name': 'kanosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5978'}, {'name': 'L-proline biosynthesis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-4281'}, {'name': 'rRNA processing in the mitochondrion', 'databaseName': 'Reactome', 'id': 'R-HSA-8868766'}, {'name': 'pyruvate-derived opine biosynthesis I (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-8450'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-SSC-70268'}, {'name': 'Escherichia coli serotype O:127 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8231'}, {'name': '1D-myo-inositol hexakisphosphate biosynthesis IV (Dictyostelium)', 'databaseName': 'MetaCyc', 'id': 'PWY-6372'}, {'name': 'Synthesis of bile acids and bile salts via 24-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-HSA-193775'}, {'name': 'UDP-alpha-D-glucuronate biosynthesis (from UDP-glucose)', 'databaseName': 'MetaCyc', 'id': 'PWY-7346'}, {'name': 'teichuronic acid biosynthesis (B. subtilis 168)', 'databaseName': 'MetaCyc', 'id': 'PWY-7820'}, {'name': 'L-tyrosine degradation V (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8016'}, {'name': 'dTDP-beta-L-digitoxose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7657'}, {'name': 'ephedrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5883'}, {'name': 'D-apiose degradation II (to D-apionate)', 'databaseName': 'MetaCyc', 'id': 'PWY-8093'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-DME-156581'}, {'name': '(5Z)-dodecenoate biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7858'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-8964539'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-75105'}, {'name': 'menthol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3061'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-71064'}, {'name': 'Biosynthesis of D-series resolvins', 'databaseName': 'Reactome', 'id': 'R-RNO-9018676'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-RNO-173599'}, {'name': 'toluene degradation to benzoyl-CoA (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-81'}, {'name': 'acetylaszonalenin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7532'}, {'name': 'L-canavanine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8419'}, {'name': 'methylaspartate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6728'}, {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7338'}, {'name': 'D-arginine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6422'}, {'name': 'D-xylose degradation to ethylene glycol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7178'}, {'name': 'tropane alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5317'}, {'name': 'pinosylvin metabolism', 'databaseName': 'MetaCyc', 'id': 'PWY-5045'}, {'name': '2-chloroacrylate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7425'}, {'name': 'cardenolide glucosides biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6036'}, {'name': 'acetaldehyde biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6333'}, {'name': 'penigequinolone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8389'}, {'name': '2-hydroxyparaconate and itatartarate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8019'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-XTR-77346'}, {'name': 'Mineralocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-193993'}, {'name': 'D-fructuronate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7242'}, {'name': 'nocardicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7797'}, {'name': 'Escherichia coli serotype O:107 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8219'}, {'name': 'fatty acid salvage', 'databaseName': 'MetaCyc', 'id': 'PWY-7094'}, {'name': 'roquefortine C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7605'}, {'name': 'saframycin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7671'}, {'name': 'guadinomine B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7693'}, {'name': 'Mineralocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-193993'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SCE-193144'}, {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids', 'databaseName': 'Reactome', 'id': 'R-HSA-77288'}, {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)', 'databaseName': 'Reactome', 'id': 'R-SSC-2162123'}, {'name': 'emodin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8166'}, {'name': 'Galactose catabolism', 'databaseName': 'Reactome', 'id': 'R-SPO-70370'}, {'name': 'propane degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7774'}, {'name': 'icosapentaenoate biosynthesis III (8-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7724'}, {'name': 'RHOD GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-MMU-9013405'}, {'name': 'kauralexin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6887'}, {'name': 'aloesone and aloesin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-4801'}, {'name': 'acinetoferrin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7989'}, {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-6545'}, {'name': 'rosamicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7677'}, {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids', 'databaseName': 'Reactome', 'id': 'R-RNO-77288'}, {'name': 'anthocyanidin 3-malylglucoside biosynthesis (acyl-glucose dependent)', 'databaseName': 'MetaCyc', 'id': 'PWY-7261'}, {'name': 'Beta-oxidation of pristanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-DME-389887'}, {'name': 'Cytoprotection by HMOX1', 'databaseName': 'Reactome', 'id': 'R-MMU-9707564'}, {'name': 'Wax biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-9640463'}, {'name': 'phenylethanol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5751'}, {'name': 'Wax biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-9640463'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-DME-4641265'}, {'name': 'dimethyl sulfoxide degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6050'}, {'name': 'Beta-oxidation of very long chain fatty acids', 'databaseName': 'Reactome', 'id': 'R-RNO-390247'}, {'name': 'Mineralocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CFA-193993'}, {'name': 'gossypol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5773'}, {'name': 'wax esters biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5884'}, {'name': 'Complex I biogenesis', 'databaseName': 'Reactome', 'id': 'R-DDI-6799198'}, {'name': 'sch210971 and sch210972 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7702'}, {'name': '6-methoxypodophyllotoxin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5479'}, {'name': 'isopimaric acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5422'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77346'}, {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8132'}, {'name': 'aromatic biogenic amine degradation (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-7431'}, {'name': 'apigeninidin 5-O-glucoside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7253'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-193144'}, {'name': 'Synthesis of Leukotrienes (LT) and Eoxins (EX)', 'databaseName': 'Reactome', 'id': 'R-HSA-2142691'}, {'name': 'chlorophyll a biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5064'}, {'name': 'naringenin C-glucosylation', 'databaseName': 'MetaCyc', 'id': 'PWY-6602'}, {'name': 'CDC42 GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-9013148'}, {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-CEL-77350'}, {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-MMU-70635'}, {'name': 'TCA cycle II (plants and fungi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5690'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-DRE-2453902'}, {'name': 'superpathway of glucose and xylose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6901'}, {'name': 'juniperonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7619'}, {'name': 'dhurrin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5976'}, {'name': 'erythritol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8373'}, {'name': '2-methyl-branched fatty acid beta-oxidation', 'databaseName': 'MetaCyc', 'id': 'PWY-8181'}, {'name': 'dimethylsulfoniopropanoate degradation II (cleavage)', 'databaseName': 'MetaCyc', 'id': 'PWY-6056'}, {'name': '3-dehydroquinate biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6164'}, {'name': 'strychnine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8216'}, {'name': 'cyanidin dimalonylglucoside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7452'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SCE-5652227'}, {'name': 'NR1H2 & NR1H3 regulate gene expression linked to lipogenesis', 'databaseName': 'Reactome', 'id': 'R-HSA-9029558'}, {'name': 'glutaryl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5177'}, {'name': 'Vitamin B5 (pantothenate) metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-199220'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-DRE-5365859'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-DME-5628897'}, {'name': 'naphthomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8038'}, {'name': 'gamma-butyrobetaine degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-8337'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-BTA-70171'}, {'name': 'thiocoraline biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7737'}, {'name': 'L-alanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8188'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DRE-193048'}, {'name': 'saponin biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-5759'}, {'name': 'violdelphin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7458'}, {'name': 'tetrahydrofolate biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6614'}, {'name': 'fusaric acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7672'}, {'name': 'bile acids 7-O epimerization', 'databaseName': 'MetaCyc', 'id': 'PWY-8254'}, {'name': 'Cholesterol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-191273'}, {'name': 'CMP-N-acetyl-7-O-acetylneuraminate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7529'}, {'name': 'daidzin and daidzein degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6996'}, {'name': 'pyruvate fermentation to ethanol II', 'databaseName': 'MetaCyc', 'id': 'PWY-5486'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77346'}, {'name': 'Sensory processing of sound by inner hair cells of the cochlea', 'databaseName': 'Reactome', 'id': 'R-HSA-9662360'}, {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)', 'databaseName': 'Reactome', 'id': 'R-HSA-2162123'}, {'name': 'poly-hydroxy fatty acids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6710'}, {'name': 'CDP-paratose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5832'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-CEL-77346'}, {'name': 'sulfoacetate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6593'}, {'name': 'Synthesis of Prostaglandins (PG) and Thromboxanes (TX)', 'databaseName': 'Reactome', 'id': 'R-MMU-2162123'}, {'name': 'alkane oxidation', 'databaseName': 'MetaCyc', 'id': 'PWY-2724'}, {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-DDI-70635'}, {'name': 'rubber biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5815'}, {'name': 'clavulanate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5679'}, {'name': 'RHOJ GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-9013409'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-RNO-2453902'}, {'name': 'L-arginine degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8187'}, {'name': 'archaeosine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7923'}, {'name': 'Respiratory electron transport', 'databaseName': 'Reactome', 'id': 'R-HSA-611105'}, {'name': 'dimycocerosyl phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7744'}, {'name': 'phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7741'}, {'name': 'Threonine catabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-8849175'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-70895'}, {'name': 'sulfolactate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6616'}, {'name': 'myo-inositol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7241'}, {'name': 'galactolipid biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7666'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-2161541'}, {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77350'}, {'name': '(Kdo)2-lipid A biosynthesis (P. putida)', 'databaseName': 'MetaCyc', 'id': 'PWY-8075'}, {'name': 'tunicamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7821'}, {'name': 'Beta oxidation of myristoyl-CoA to lauroyl-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77285'}, {'name': 'dTDP-3-acetamido-alpha-D-fucose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6953'}, {'name': 'UDP-N-acetyl-alpha-D-galactosaminuronate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7336'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-SCE-71336'}, {'name': 'GDP-fucose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DDI-6787639'}, {'name': 'VEGFR2 mediated cell proliferation', 'databaseName': 'Reactome', 'id': 'R-HSA-5218921'}, {'name': 'aminopropanol phosphate biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7378'}, {'name': 'cutin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-321'}, {'name': 'petrobactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6289'}, {'name': 'kappa-carrageenan degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6821'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-RNO-5365859'}, {'name': 'pyruvate fermentation to ethanol III', 'databaseName': 'MetaCyc', 'id': 'PWY-6587'}, {'name': 'fatty acid beta-oxidation V (unsaturated, odd number, di-isomerase-dependent)', 'databaseName': 'MetaCyc', 'id': 'PWY-6837'}, {'name': 'chelerythrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7507'}, {'name': '1,4-dimethylbenzene degradation to 4-methylbenzoate', 'databaseName': 'MetaCyc', 'id': 'PWY-5429'}, {'name': 'DAG and IP3 signaling', 'databaseName': 'Reactome', 'id': 'R-HSA-1489509'}, {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77350'}, {'name': 'D-xylose degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-8330'}, {'name': 'sangivamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6721'}, {'name': 'D-altritol and galactitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7862'}, {'name': 'bombykol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7423'}, {'name': 'lampranthin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7265'}, {'name': 'asperlicin E biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7549'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-DDI-77346'}, {'name': 'formaldehyde oxidation VII (THF pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-7909'}, {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7331'}, {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77310'}, {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors', 'databaseName': 'Reactome', 'id': 'R-MMU-8866904'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-SPO-1614635'}, {'name': 'Negative regulation of activity of TFAP2 (AP-2) family transcription factors', 'databaseName': 'Reactome', 'id': 'R-HSA-8866904'}, {'name': 'RHOQ GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-HSA-9013406'}, {'name': 'L-rhamnose degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6714'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-RNO-196757'}, {'name': '1,2,4-trichlorobenzene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6091'}, {'name': 'type IV lipoteichoic acid biosynthesis (S. pneumoniae)', 'databaseName': 'MetaCyc', 'id': 'PWY-7818'}, {'name': 'pyruvate fermentation to (R)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-8274'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-CEL-70268'}, {'name': 'Glyoxylate metabolism and glycine degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-389661'}, {'name': 'nystatin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7624'}, {'name': 'mycolyl-arabinogalactan-peptidoglycan complex biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6397'}, {'name': 'Essential pentosuria', 'databaseName': 'Reactome', 'id': 'R-HSA-5662853'}, {'name': 'D-galactose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6693'}, {'name': 'arachidonate biosynthesis IV (8-detaturase, lower eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-7601'}, {'name': 'grixazone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7153'}, {'name': 'Beta-oxidation of very long chain fatty acids', 'databaseName': 'Reactome', 'id': 'R-HSA-390247'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DRE-194002'}, {'name': 'The retinoid cycle in cones (daylight vision)', 'databaseName': 'Reactome', 'id': 'R-HSA-2187335'}, {'name': 'L-carnitine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-3602'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77346'}, {'name': 'superpathway of hyoscyamine (atropine) and scopolamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7341'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-CEL-1614635'}, {'name': 'sulfoquinovose degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7446'}, {'name': 'tRNA processing in the mitochondrion', 'databaseName': 'Reactome', 'id': 'R-HSA-6785470'}, {'name': 'Cholesterol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-191273'}, {'name': 'Plasmalogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-XTR-75896'}, {'name': 'retinoate biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6872'}, {'name': 'pseudomonine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7983'}, {'name': 'tetrahydroxyxanthone biosynthesis (from 3-hydroxybenzoate)', 'databaseName': 'MetaCyc', 'id': 'PWY-5002'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-75105'}, {'name': 'amaranthin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5400'}, {'name': 'spectinabilin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8034'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-BTA-5661270'}, {'name': 'menaquinol-4 biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7998'}, {'name': 'pentalenolactone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6915'}, {'name': 'cis-alkene biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7869'}, {'name': 'tridecaprenyl diphosphate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5893'}, {'name': 'Synthesis of very long-chain fatty acyl-CoAs', 'databaseName': 'Reactome', 'id': 'R-RNO-75876'}, {'name': 'The retinoid cycle in cones (daylight vision)', 'databaseName': 'Reactome', 'id': 'R-MMU-2187335'}, {'name': 'ursodeoxycholate biosynthesis (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-7588'}, {'name': 'phosphinothricin tripeptide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6322'}, {'name': 'capsaicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5710'}, {'name': 'rosmarinic acid biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5049'}, {'name': 'L-galactose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7568'}, {'name': '4-aminobutanoate degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-5022'}, {'name': '(9Z)-tricosene biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7035'}, {'name': 'D-apionate degradation III (RLP transcarboxylase/hydrolase)', 'databaseName': 'MetaCyc', 'id': 'PWY-8092'}, {'name': 'neomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7016'}, {'name': 'avenacin A-2 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7474'}, {'name': 'dTDP-N-acetylviosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7316'}, {'name': 'nicotine degradation III (VPP pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-7128'}, {'name': 'Synthesis of Lipoxins (LX)', 'databaseName': 'Reactome', 'id': 'R-RNO-2142700'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-CEL-70263'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-CEL-2161541'}, {'name': 'mitochondrial fatty acid beta-oxidation of unsaturated fatty acids', 'databaseName': 'Reactome', 'id': 'R-MMU-77288'}, {'name': 'Transcriptional activation of mitochondrial biogenesis', 'databaseName': 'Reactome', 'id': 'R-DME-2151201'}, {'name': 'bile acid 7alpha-dehydroxylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7754'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-RNO-71384'}, {'name': 'folate transformations II (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-3841'}, {'name': 'gliotoxin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7533'}, {'name': 'Synthesis of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-BTA-77111'}, {'name': 'carbon disulfide oxidation III (metazoa)', 'databaseName': 'MetaCyc', 'id': 'PWY-7926'}, {'name': 'narbomycin, pikromycin and novapikromycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7421'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'Nuclear signaling by ERBB4', 'databaseName': 'Reactome', 'id': 'R-MMU-1251985'}, {'name': 'Retinoid cycle disease events', 'databaseName': 'Reactome', 'id': 'R-HSA-2453864'}, {'name': 'oleate biosynthesis IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-7664'}, {'name': 'GDP-N-acetyl-alpha-D-perosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8225'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-5652227'}, {'name': 'traumatin and (Z)-3-hexen-1-yl acetate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5410'}, {'name': 'noradrenaline and adrenaline degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6342'}, {'name': 'flavonol acylglucoside biosynthesis III - quercetin derivatives', 'databaseName': 'MetaCyc', 'id': 'PWY-7172'}, {'name': 'plumbagin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6314'}, {'name': 'vancomycin resistance I', 'databaseName': 'MetaCyc', 'id': 'PWY-6454'}, {'name': '4-hydroxybenzoate biosynthesis III (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-6435'}, {'name': 'digitoxigenin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6032'}, {'name': 'tetramethylpyrazine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7403'}, {'name': 'Transferrin endocytosis and recycling', 'databaseName': 'Reactome', 'id': 'R-HSA-917977'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-MMU-1855183'}, {'name': 'Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-DRE-77346'}, {'name': 'leucodelphinidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5152'}, {'name': 'avenacin A-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7473'}, {'name': 'aucuparin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7739'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-HSA-1855183'}, {'name': 'penicillin G and penicillin V biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7716'}, {'name': 'sporopollenin precursors biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6733'}, {'name': 'L-arginine degradation XI', 'databaseName': 'MetaCyc', 'id': 'PWY-5024'}, {'name': 'Entner-Doudoroff pathway III (semi-phosphorylative)', 'databaseName': 'MetaCyc', 'id': 'PWY-2221'}, {'name': 'Alpha-oxidation of phytanate', 'databaseName': 'Reactome', 'id': 'R-MMU-389599'}, {'name': 'apratoxin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8361'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SPO-193144'}, {'name': 'ulvan degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7647'}, {'name': '2,4-dichlorotoluene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6190'}, {'name': 'diphenyl ethers degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7747'}, {'name': '3-amino-5-hydroxybenzoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5979'}, {'name': 'very long chain fatty acid biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5080'}, {'name': 'des-methyl avenacin A-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7475'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-DME-1483166'}, {'name': 'Cholesterol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-191273'}, {'name': '(-)-glycinol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2761'}, {'name': 'methylglyoxal degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-5459'}, {'name': 'suberin monomers biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1121'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-5652227'}, {'name': 'tRNA modification in the mitochondrion', 'databaseName': 'Reactome', 'id': 'R-HSA-6787450'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-193048'}, {'name': 'Tetrahydrobiopterin (BH4) synthesis, recycling, salvage and regulation', 'databaseName': 'Reactome', 'id': 'R-MMU-1474151'}, {'name': 'Plasmalogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-75896'}, {'name': '2-hydroxypenta-2,4-dienoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5162'}, {'name': 'cholesterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8239'}, {'name': 'TP53 Regulates Transcription of Genes Involved in Cytochrome C Release', 'databaseName': 'Reactome', 'id': 'R-RNO-6803204'}, {'name': 'Sperm Motility And Taxes', 'databaseName': 'Reactome', 'id': 'R-DME-1300642'}, {'name': 'mithramycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7045'}, {'name': 'Tetrahydrobiopterin (BH4) synthesis, recycling, salvage and regulation', 'databaseName': 'Reactome', 'id': 'R-HSA-1474151'}, {'name': '1,3-dimethylbenzene degradation to 3-methylbenzoate', 'databaseName': 'MetaCyc', 'id': 'PWY-5428'}, {'name': 'Plasmalogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DME-75896'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-BTA-70895'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-BTA-6798695'}, {'name': 'octanoyl-[acyl-carrier protein] biosynthesis (mitochondria, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7388'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-BTA-5365859'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-DME-1855183'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-DME-9033241'}, {'name': 'Platelet degranulation', 'databaseName': 'Reactome', 'id': 'R-RNO-114608'}, {'name': 'ChREBP activates metabolic gene expression', 'databaseName': 'Reactome', 'id': 'R-HSA-163765'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-DRE-8964539'}, {'name': 'Retinoid metabolism and transport', 'databaseName': 'Reactome', 'id': 'R-HSA-975634'}, {'name': '2-chloroacrylate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7428'}, {'name': 'myxochelin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8383'}, {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-CEL-77348'}, {'name': 'puerarin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-8445'}, {'name': '11-cis-3-hydroxyretinal biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7043'}, {'name': 'S-adenosyl-L-methionine salvage II', 'databaseName': 'MetaCyc', 'id': 'PWY-5041'}, {'name': 'L-tyrosine biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-6120'}, {'name': 'hydroxycinnamic acid serotonin amides biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5473'}, {'name': 'leporin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8370'}, {'name': '8-amino-7-oxononanoate biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6519'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-CEL-5628897'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DRE-75105'}, {'name': 'D-carnitine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7472'}, {'name': 'D-apionate degradation II (RLP decarboxylase)', 'databaseName': 'MetaCyc', 'id': 'PWY-8090'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-SSC-70263'}, {'name': 'carbon disulfide oxidation II (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5336'}, {'name': '2,3-cis-flavanols biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6035'}, {'name': 'Peroxisomal protein import', 'databaseName': 'Reactome', 'id': 'R-MMU-9033241'}, {'name': 'rhizobitoxine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7906'}, {'name': 'pyruvate fermentation to (S)-acetoin', 'databaseName': 'MetaCyc', 'id': 'PWY-6389'}, {'name': 'dZTP biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8289'}, {'name': 'L-fucose degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8318'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-BTA-2453902'}, {'name': 'putrebactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6378'}, {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77348'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-DDI-70263'}, {'name': 'polyacyltrehalose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7738'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-RNO-6798695'}, {'name': 'thalianol and derivatives biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5992'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CEL-193048'}, {'name': 'cellulose and hemicellulose degradation (cellulolosome)', 'databaseName': 'MetaCyc', 'id': 'PWY-6784'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-DRE-70263'}, {'name': 'Heme degradation', 'databaseName': 'Reactome', 'id': 'R-RNO-189483'}, {'name': 'peptidoglycan cross-bridge biosynthesis III (Enterococcus faecalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-6462'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-CEL-156581'}, {'name': '2-deoxy-D-ribose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8058'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-HSA-5628897'}, {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77352'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-XTR-70263'}, {'name': 'ferrichrome biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7577'}, {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5754'}, {'name': 'piperine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5752'}, {'name': 'linamarin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-3121'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-MMU-5628897'}, {'name': 'Lipid particle organization', 'databaseName': 'Reactome', 'id': 'R-HSA-8964572'}, {'name': 'peramine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7600'}, {'name': 'Lipid particle organization', 'databaseName': 'Reactome', 'id': 'R-MMU-8964572'}, {'name': 'Synthesis of very long-chain fatty acyl-CoAs', 'databaseName': 'Reactome', 'id': 'R-HSA-75876'}, {'name': '1,3-dichlorobenzene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6081'}, {'name': 'isopropanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6876'}, {'name': 'diphyllin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6820'}, {'name': 'Activation of gene expression by SREBF (SREBP)', 'databaseName': 'Reactome', 'id': 'R-HSA-2426168'}, {'name': 'iso-bile acids biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7756'}, {'name': 'lauryl-hydroxychlorobactene glucoside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7940'}, {'name': 'L-histidine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5030'}, {'name': 'UDP-beta-L-rhamnose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3261'}, {'name': 'pyrimidine deoxyribonucleosides degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7181'}, {'name': 'plasmalogen biosynthesis I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-7782'}, {'name': 'Nuclear signaling by ERBB4', 'databaseName': 'Reactome', 'id': 'R-DME-1251985'}, {'name': 'chlorogenic acid biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6039'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-SCE-71064'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DDI-75105'}, {'name': '2,4-xylenol degradation to protocatechuate', 'databaseName': 'MetaCyc', 'id': 'PWY-7703'}, {'name': 'oleate beta-oxidation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7291'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-RNO-71403'}, {'name': '2,3-trans-flavanols biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6029'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-HSA-4641265'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-BTA-8964539'}, {'name': 'salicortin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6763'}, {'name': 'di-myo-inositol phosphate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6664'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-MMU-4641265'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-DRE-6798695'}, {'name': 'tetracycline and oxytetracycline biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7812'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-GGA-352875'}, {'name': 'odd iso-branched-chain fatty acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8174'}, {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77310'}, {'name': 'free phenylpropanoid acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-2181'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-SPO-70171'}, {'name': 'pyrrolomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8210'}, {'name': 'alpha-linolenate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8398'}, {'name': 'methoxylated aromatic compound degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8305'}, {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-7288'}, {'name': 'vindoline, vindorosine and vinblastine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5292'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CEL-75105'}, {'name': 'meso-butanediol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6391'}, {'name': 'glycyrrhetinate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7066'}, {'name': 'C4 photosynthetic carbon assimilation cycle, PEPCK type', 'databaseName': 'MetaCyc', 'id': 'PWY-7117'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-HSA-71403'}, {'name': 'erythritol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-8372'}, {'name': '(2S,3E)-2-amino-4-methoxy-but-3-enoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7907'}, {'name': 'fatty acid alpha-oxidation I (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2501'}, {'name': 'Synthesis of bile acids and bile salts via 24-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-RNO-193775'}, {'name': 'L-serine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8011'}, {'name': 'The retinoid cycle in cones (daylight vision)', 'databaseName': 'Reactome', 'id': 'R-CEL-2187335'}, {'name': 'alpha-cyclopiazonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7555'}, {'name': 'L-carnitine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-3641'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-CEL-1483166'}, {'name': 'echinomycin and triostin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7735'}, {'name': 'dTDP-N-acetylthomosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7315'}, {'name': 'Beta oxidation of myristoyl-CoA to lauroyl-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77285'}, {'name': 'vitamin K-epoxide cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-7999'}, {'name': 'itaconate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5749'}, {'name': 'eNOS activation', 'databaseName': 'Reactome', 'id': 'R-RNO-203615'}, {'name': 'ergosterol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7154'}, {'name': 'iso-bile acids biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7755'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-DME-8964539'}, {'name': 'fatty acid biosynthesis initiation (type I)', 'databaseName': 'MetaCyc', 'id': 'PWY-5966'}, {'name': 'autoinducer CAI-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7621'}, {'name': 'palmitate biosynthesis I (type I fatty acid synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-5994'}, {'name': 'indolmycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7770'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-RNO-5661270'}, {'name': 'bryostatin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8047'}, {'name': 'calonectrin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7711'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-HSA-71336'}, {'name': 'Transcriptional activation of mitochondrial biogenesis', 'databaseName': 'Reactome', 'id': 'R-HSA-2151201'}, {'name': 'oleate biosynthesis I (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-5147'}, {'name': 'L-homophenylalanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7275'}, {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA', 'databaseName': 'MetaCyc', 'id': 'PWY-5652'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-70268'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-DME-70263'}, {'name': 'zerumbone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6265'}, {'name': 'sitosterol degradation to androstenedione', 'databaseName': 'MetaCyc', 'id': 'PWY-6948'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-HSA-1483166'}, {'name': 'Beta oxidation of octanoyl-CoA to hexanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77348'}, {'name': 'UDP-N-acetylmuramoyl-pentapeptide biosynthesis II (lysine-containing)', 'databaseName': 'MetaCyc', 'id': 'PWY-6386'}, {'name': 'dTDP-sibirosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8380'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-2161541'}, {'name': 'zymosterol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6074'}, {'name': 'Synthesis of Lipoxins (LX)', 'databaseName': 'Reactome', 'id': 'R-MMU-2142700'}, {'name': 'cremeomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8296'}, {'name': 'rhizocticin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7510'}, {'name': 'platensimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8179'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-194002'}, {'name': 'Escherichia coli serotype O:71/Salmonella enterica serotype O:28ac O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8241'}, {'name': 'N-hydroxy-L-pipecolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7861'}, {'name': 'aflatrem biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7540'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-SCE-70171'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-SCE-1855183'}, {'name': 'pyrroloquinoline quinone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6420'}, {'name': 'kavain biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8103'}, {'name': 'UDP-N-acetylmuramoyl-pentapeptide biosynthesis I (meso-diaminopimelate containing)', 'databaseName': 'MetaCyc', 'id': 'PWY-6387'}, {'name': 'dalcochinin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5821'}, {'name': 'L-phenylalanine degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-7158'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-DME-70268'}, {'name': 'succinoglycan biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8133'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-SPO-5365859'}, {'name': 'stachyose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6527'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-CEL-4641265'}, {'name': 'Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77310'}, {'name': 'ansatrienin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8040'}, {'name': 'Escherichia coli serotype O:49 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8272'}, {'name': 'Phenylalanine metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-8964208'}, {'name': 'Acyl chain remodeling of CL', 'databaseName': 'Reactome', 'id': 'R-RNO-1482798'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-DDI-156581'}, {'name': 'TCA cycle V (2-oxoglutarate synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6969'}, {'name': 'Synthesis of IP2, IP, and Ins in the cytosol', 'databaseName': 'Reactome', 'id': 'R-RNO-1855183'}, {'name': 'L-histidine degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-5031'}, {'name': 'spermidine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6559'}, {'name': 'Salmonella enterica serotype O:39 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8236'}, {'name': 'nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5316'}, {'name': 'very long chain fatty acid biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7036'}, {'name': 'ectoine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7855'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-DRE-70268'}, {'name': 'Synthesis of bile acids and bile salts via 27-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-RNO-193807'}, {'name': 'chlorobenzene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6083'}, {'name': 'Heme degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-189483'}, {'name': '(4R)-carveol and (4R)-dihydrocarveol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-5922'}, {'name': 'anaerobic energy metabolism (invertebrates, mitochondrial)', 'databaseName': 'MetaCyc', 'id': 'PWY-7384'}, {'name': 'docosahexaenoate biosynthesis IV (4-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7727'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-HSA-3899300'}, {'name': 'Plasmalogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-75896'}, {'name': 'Plasmalogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-75896'}, {'name': 'rhizobactin 1021 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-761'}, {'name': 'fusarin C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7673'}, {'name': 'Beta oxidation of hexanoyl-CoA to butanoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-SSC-77350'}, {'name': 'Synthesis of very long-chain fatty acyl-CoAs', 'databaseName': 'Reactome', 'id': 'R-XTR-75876'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-SCE-5661270'}, {'name': 'achromobactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6574'}, {'name': 'Complex I biogenesis', 'databaseName': 'Reactome', 'id': 'R-HSA-6799198'}, {'name': 'stellatic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7736'}, {'name': 'bassianin and desmethylbassianin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7563'}, {'name': 'cis-geranyl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6672'}, {'name': 'D-xylose degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-8020'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-HSA-196757'}, {'name': 'oryzalexin D and E biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7478'}, {'name': 'candicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6722'}, {'name': 'Biosynthesis of E-series 18(S)-resolvins', 'databaseName': 'Reactome', 'id': 'R-HSA-9018896'}, {'name': 'Synthesis of bile acids and bile salts via 7alpha-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-MMU-193368'}, {'name': \"3,3'-disulfanediyldipropannoate degradation\", 'databaseName': 'MetaCyc', 'id': 'PWY-7462'}, {'name': '(Z)-phenylmethanethial S-oxide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6539'}, {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)', 'databaseName': 'MetaCyc', 'id': 'PWY-6318'}, {'name': 'oleate beta-oxidation (thioesterase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7292'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-CEL-71336'}, {'name': 'PLC beta mediated events', 'databaseName': 'Reactome', 'id': 'R-HSA-112043'}, {'name': 'D-sorbitol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-4101'}, {'name': 'L-malate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7686'}, {'name': 'starch degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-6737'}, {'name': 'UDP-alpha-D-xylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-4821'}, {'name': 'camptothecin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6326'}, {'name': '(1,4)-beta-D-xylan degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6717'}, {'name': 'tetrahydromethanopterin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6148'}, {'name': 'Pyruvate metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-70268'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-SPO-71403'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-HSA-71384'}, {'name': '(R,R)-butanediol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5951'}, {'name': 'UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7090'}, {'name': 'photorespiration II', 'databaseName': 'MetaCyc', 'id': 'PWY-8362'}, {'name': 'Androgen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-193048'}, {'name': 'Fructose biosynthesis', 'databaseName': 'Reactome', 'id': 'R-BTA-5652227'}, {'name': 'staphyloxanthin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5875'}, {'name': '6-methoxymellein biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-4922'}, {'name': 'eugenol and isoeugenol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5859'}, {'name': 'valinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8336'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-SPO-2453902'}, {'name': 'Beta oxidation of myristoyl-CoA to lauroyl-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77285'}, {'name': 'fatty acid beta-oxidation II (plant peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-5136'}, {'name': 'avenanthramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8157'}, {'name': 'sulfoacetaldehyde degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-8062'}, {'name': 'aromatic polyketides biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6316'}, {'name': '(2S)-ethylmalonyl-CoA biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8425'}, {'name': 'shisonin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5284'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-SPO-1483166'}, {'name': '3,4-dichlorotoluene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6192'}, {'name': 'rutin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6848'}, {'name': 'Mineralocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-193993'}, {'name': 'icosapentaenoate biosynthesis IV (bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-7050'}, {'name': 'Acyl chain remodeling of CL', 'databaseName': 'Reactome', 'id': 'R-SSC-1482798'}, {'name': 'alpha-linolenic acid (ALA) metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-2046106'}, {'name': 'mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWYG-321'}, {'name': 'folate polyglutamylation', 'databaseName': 'MetaCyc', 'id': 'PWY-2161'}, {'name': 'methylglyoxal degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-5458'}, {'name': 'daurichromenate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8084'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-SCE-196757'}, {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen', 'databaseName': 'MetaCyc', 'id': 'PWY-8086'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-MMU-70263'}, {'name': 'Nuclear signaling by ERBB4', 'databaseName': 'Reactome', 'id': 'R-HSA-1251985'}, {'name': 'guanosine deoxyribonucleotides de novo biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7222'}, {'name': 'formaldehyde oxidation II (glutathione-dependent)', 'databaseName': 'MetaCyc', 'id': 'PWY-1801'}, {'name': '2,4,5-trichlorophenoxyacetate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6200'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-SCE-70263'}, {'name': 'D-galactose detoxification', 'databaseName': 'MetaCyc', 'id': 'PWY-3821'}, {'name': 'atranorin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8327'}, {'name': 'purine deoxyribonucleosides salvage', 'databaseName': 'MetaCyc', 'id': 'PWY-7224'}, {'name': 'mannosylerythritol lipids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8371'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-HSA-5365859'}, {'name': '(4R)-carvone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5928'}, {'name': 'L-Ndelta-acetylornithine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6922'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-DME-71384'}, {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-3722'}, {'name': 'GDP-6-deoxy-D-talose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5738'}, {'name': 'cinchona alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5848'}, {'name': 'colchicine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5846'}, {'name': 'D-galacturonate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8391'}, {'name': 'superpathway of anthocyanin biosynthesis (from cyanidin and cyanidin 3-O-glucoside)', 'databaseName': 'MetaCyc', 'id': 'PWY-5313'}, {'name': 'p-cymene degradation to p-cumate', 'databaseName': 'MetaCyc', 'id': 'PWY-741'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-BTA-173599'}, {'name': 'juvenile hormone III biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6650'}, {'name': 'xylitol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8393'}, {'name': 'pyruvate fermentation to acetate III', 'databaseName': 'MetaCyc', 'id': 'PWY-5483'}, {'name': 'salicylate biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-981'}, {'name': 'phytocassanes biosynthesis, shared reactions', 'databaseName': 'MetaCyc', 'id': 'PWY-7484'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-MMU-71336'}, {'name': '4-nitrotoluene degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-5644'}, {'name': 'dTDP-beta-L-megosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7104'}, {'name': 'lyngbyatoxin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7708'}, {'name': 'pinocembrin C-glucosylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7189'}, {'name': 'Alpha-oxidation of phytanate', 'databaseName': 'Reactome', 'id': 'R-HSA-389599'}, {'name': 'formaldehyde oxidation V (bacillithiol-dependent)', 'databaseName': 'MetaCyc', 'id': 'PWY-7908'}, {'name': 'methymycin, neomethymycin and novamethymycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7422'}, {'name': 'abscisic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-695'}, {'name': 'Estrogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CEL-193144'}, {'name': 'protein N-glycosylation (Haloferax volcanii)', 'databaseName': 'MetaCyc', 'id': 'PWY-7661'}, {'name': 'Synthesis of bile acids and bile salts via 7alpha-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-HSA-193368'}, {'name': 'acrylate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8180'}, {'name': 'L-phenylalanine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5079'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-8964539'}, {'name': 'bile acids 3-O-epimerization', 'databaseName': 'MetaCyc', 'id': 'PWY-8255'}, {'name': 'Heme degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-189483'}, {'name': 'hydroxylated fatty acid biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-6433'}, {'name': 'ethylbenzene degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-481'}, {'name': 'diadinoxanthin and fucoxanthin biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-7950'}, {'name': 'pyruvate fermentation to butanol II (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6883'}, {'name': 'Beta oxidation of palmitoyl-CoA to myristoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-MMU-77305'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-DME-71336'}, {'name': 'Ca2+ activated K+ channels', 'databaseName': 'Reactome', 'id': 'R-MMU-1296052'}, {'name': 'benzoate biosynthesis I (CoA-dependent, beta-oxidative)', 'databaseName': 'MetaCyc', 'id': 'PWY-6443'}, {'name': 'superpathway of photosynthetic hydrogen production', 'databaseName': 'MetaCyc', 'id': 'PWY-7731'}, {'name': 'FCERI mediated Ca+2 mobilization', 'databaseName': 'Reactome', 'id': 'R-HSA-2871809'}, {'name': '(2R)-methoxymalonyl-[acp] biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8426'}, {'name': 'RHOQ GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-RNO-9013406'}, {'name': 'cyathin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8409'}, {'name': 'acetyl-CoA biosynthesis from citrate', 'databaseName': 'MetaCyc', 'id': 'PWY-5172'}, {'name': 'chanoclavine I aldehyde biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6493'}, {'name': 'terminal olefins biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7029'}, {'name': 'morphine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5270'}, {'name': 'Beta oxidation of palmitoyl-CoA to myristoyl-CoA', 'databaseName': 'Reactome', 'id': 'R-RNO-77305'}, {'name': 'fusaridione A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7670'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-SPO-8964539'}, {'name': 'alpha-linolenic acid (ALA) metabolism', 'databaseName': 'Reactome', 'id': 'R-MMU-2046106'}, {'name': 'RHOF GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-MMU-9035034'}, {'name': 'Plasmalogen biosynthesis', 'databaseName': 'Reactome', 'id': 'R-DRE-75896'}, {'name': 'Cytoprotection by HMOX1', 'databaseName': 'Reactome', 'id': 'R-BTA-9707564'}, {'name': 'mitochondrial NADPH production (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7269'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-194002'}, {'name': '(S,S)-butanediol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6390'}, {'name': 'zwittermicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7694'}, {'name': 'fatty acid beta-oxidation IV (unsaturated, even number)', 'databaseName': 'MetaCyc', 'id': 'PWY-5138'}, {'name': 'Cholesterol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-191273'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SPO-75105'}, {'name': '4-hydroxy-3-prenylbenzoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7303'}, {'name': 'RHOQ GTPase cycle', 'databaseName': 'Reactome', 'id': 'R-MMU-9013406'}, {'name': 'NFE2L2 regulates pentose phosphate pathway genes', 'databaseName': 'Reactome', 'id': 'R-HSA-9818028'}, {'name': 'Antigen activates B Cell Receptor (BCR) leading to generation of second messengers', 'databaseName': 'Reactome', 'id': 'R-HSA-983695'}, {'name': 'pelargonidin conjugates biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5139'}, {'name': 'L-lysine biosynthesis III', 'databaseName': 'MetaCyc', 'id': 'PWY-2942'}, {'name': 'justicidin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6824'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-GGA-352882'}, {'name': 'Synthesis of bile acids and bile salts via 27-hydroxycholesterol', 'databaseName': 'Reactome', 'id': 'R-HSA-193807'}, {'name': 'Cholesterol biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SCE-191273'}, {'name': 'butanol and isobutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7396'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-SCE-1614635'}, {'name': 'Transcriptional activation of mitochondrial biogenesis', 'databaseName': 'Reactome', 'id': 'R-RNO-2151201'}, {'name': 'gallotannin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5477'}, {'name': 'geraniol and geranial biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5829'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-RNO-1483166'}, {'name': '1,2-propanediol biosynthesis from lactate (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7541'}, {'name': 'vitamin B6 degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-5499'}, {'name': 'pyruvate fermentation to ethanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-5480'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-70895'}, {'name': 'Beta oxidation of butanoyl-CoA to acetyl-CoA', 'databaseName': 'Reactome', 'id': 'R-HSA-77352'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-DME-5365859'}, {'name': 'phloridzin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6515'}, {'name': 'anditomin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7599'}, {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-8010'}, {'name': 'crotonate fermentation (to acetate and cyclohexane carboxylate)', 'databaseName': 'MetaCyc', 'id': 'PWY-7401'}, {'name': 'L-ascorbate biosynthesis V (euglena, D-galacturonate pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-6415'}, {'name': 'Ion homeostasis', 'databaseName': 'Reactome', 'id': 'R-DME-5578775'}, {'name': 'jadomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6679'}, {'name': 'demethylmenaquinol-8 biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6262'}, {'name': 'Ub-specific processing proteases', 'databaseName': 'Reactome', 'id': 'R-MMU-5689880'}, {'name': 'domoic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8083'}, {'name': 'mammalian high-mannose N-glycan degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-8436'}, {'name': 'erythromycin D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7106'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-SPO-2161541'}, {'name': 'Vitamin B5 (pantothenate) metabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-199220'}, {'name': 'flavonol acylglucoside biosynthesis I - kaempferol derivatives', 'databaseName': 'MetaCyc', 'id': 'PWY-7168'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-DRE-70171'}, {'name': 'gentiodelphin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5307'}, {'name': 'corallopyronin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8438'}, {'name': 'Glucocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-CFA-194002'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-HSA-70263'}, {'name': 'Utilization of Ketone Bodies', 'databaseName': 'Reactome', 'id': 'R-MMU-77108'}, {'name': 'chaxamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8036'}, {'name': '2-deoxy-alpha-D-ribose 1-phosphate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7180'}, {'name': 'Ca2+ activated K+ channels', 'databaseName': 'Reactome', 'id': 'R-GGA-1296052'}, {'name': 'dimycocerosyl triglycosyl phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7743'}, {'name': 'Glycolysis', 'databaseName': 'Reactome', 'id': 'R-MMU-70171'}, {'name': 'hygromycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8367'}, {'name': 'pyoverdine I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6409'}, {'name': 'Beta-oxidation of very long chain fatty acids', 'databaseName': 'Reactome', 'id': 'R-DME-390247'}, {'name': 'Urea cycle', 'databaseName': 'Reactome', 'id': 'R-SCE-70635'}, {'name': 'Escherichia coli serotype O:86 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7290'}, {'name': 'acetate and ATP formation from acetyl-CoA II', 'databaseName': 'MetaCyc', 'id': 'PWY-5535'}, {'name': 'L-isoleucine biosynthesis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-5104'}, {'name': 'heme degradation V', 'databaseName': 'MetaCyc', 'id': 'PWY-7846'}, {'name': 'L-homomethionine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1186'}, {'name': 'demethylmenaquinol-6 biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7372'}, {'name': 'oleandomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6971'}, {'name': 'CDP-abequose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5831'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-DME-3899300'}, {'name': 'phosphatidate biosynthesis (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7411'}, {'name': 'trans-3-hydroxy-L-proline degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7515'}, {'name': 'Branched-chain amino acid catabolism', 'databaseName': 'Reactome', 'id': 'R-DDI-70895'}, {'name': 'starch degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6735'}, {'name': 'TP53 Regulates Metabolic Genes', 'databaseName': 'Reactome', 'id': 'R-PFA-5628897'}, {'name': 'L-valine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-5057'}, {'name': 'Ca2+ activated K+ channels', 'databaseName': 'Reactome', 'id': 'R-BTA-1296052'}, {'name': 'Metabolism of folate and pterines', 'databaseName': 'Reactome', 'id': 'R-MMU-196757'}, {'name': '5,6-dehydrokavain biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-8463'}, {'name': 'Prednisone ADME', 'databaseName': 'Reactome', 'id': 'R-HSA-9757110'}, {'name': 'L-malate degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7685'}, {'name': 'serinol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6456'}, {'name': 'alkane biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7032'}, {'name': 'salicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6766'}, {'name': 'isethionate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-2601'}, {'name': 'bile acids 12-O-epimerization', 'databaseName': 'MetaCyc', 'id': 'PWY-8256'}, {'name': 'L-tryptophan degradation X (mammalian, via tryptamine)', 'databaseName': 'MetaCyc', 'id': 'PWY-6307'}, {'name': 'Glutamate and glutamine metabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-8964539'}, {'name': '1,3-propanediol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7385'}, {'name': 'palmitoleate biosynthesis II (plants and bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-5366'}, {'name': 'dTDP-beta-L-olivose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6974'}, {'name': 'D-glucuronate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8390'}, {'name': 'Sphingolipid de novo biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-1660661'}, {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7337'}, {'name': 'rosmarinic acid biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5048'}, {'name': 'flavonoid di-C-glucosylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7897'}, {'name': 'superpathway of UDP-glucose-derived O-antigen building blocks biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7328'}, {'name': 'antimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8374'}, {'name': 'Abacavir metabolism', 'databaseName': 'Reactome', 'id': 'R-SCE-2161541'}, {'name': 'palmitoyl ethanolamide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8055'}, {'name': 'eNOS activation', 'databaseName': 'Reactome', 'id': 'R-DDI-203615'}, {'name': 'carrageenan biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6403'}, {'name': 'alpha-cyclopiazonate detoxification', 'databaseName': 'MetaCyc', 'id': 'PWY-7558'}, {'name': 'Nuclear signaling by ERBB4', 'databaseName': 'Reactome', 'id': 'R-DRE-1251985'}, {'name': 'Lysine catabolism', 'databaseName': 'Reactome', 'id': 'R-HSA-71064'}, {'name': 'streptomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5940'}, {'name': 'anteiso-branched-chain fatty acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8173'}, {'name': 'anthocyanidin modification (Arabidopsis)', 'databaseName': 'MetaCyc', 'id': 'PWY-7450'}, {'name': 'aclacinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7354'}, {'name': 'brassicicene C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7517'}, {'name': 'Fatty acyl-CoA biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-75105'}, {'name': 'Transferrin endocytosis and recycling', 'databaseName': 'Reactome', 'id': 'R-MMU-917977'}, {'name': 'salvianin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5268'}, {'name': 'fumigaclavine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7059'}, {'name': 'D-arabinitol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8392'}, {'name': '(S,S)-butanediol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6388'}, {'name': '3-methylquinoline degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-721'}, {'name': 'daphnin interconversion', 'databaseName': 'MetaCyc', 'id': 'PWY-7056'}, {'name': 'Activation of the TFAP2 (AP-2) family of transcription factors', 'databaseName': 'Reactome', 'id': 'R-MMU-8866907'}, {'name': 'sulfoquinovose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7722'}, {'name': 'glucosinolate biosynthesis from tryptophan', 'databaseName': 'MetaCyc', 'id': 'PWY-601'}, {'name': '1D-myo-inositol hexakisphosphate biosynthesis III (Spirodela polyrrhiza)', 'databaseName': 'MetaCyc', 'id': 'PWY-4661'}, {'name': 'Respiratory electron transport', 'databaseName': 'Reactome', 'id': 'R-RNO-611105'}, {'name': 'UDP-N-acetyl-D-galactosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5512'}, {'name': 'Acetylcholine inhibits contraction of outer hair cells', 'databaseName': 'Reactome', 'id': 'R-HSA-9667769'}, {'name': 'Spodoptera littoralis pheromone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7656'}, {'name': 'triclosan resistance', 'databaseName': 'MetaCyc', 'id': 'PWY-7096'}, {'name': 'stigma estolide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6453'}, {'name': 'abscisic acid degradation to phaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-5271'}, {'name': 'limonene degradation II (L-limonene)', 'databaseName': 'MetaCyc', 'id': 'PWY-5924'}, {'name': 'methanol oxidation to carbon dioxide', 'databaseName': 'MetaCyc', 'id': 'PWY-7616'}, {'name': 'Escherichia coli serotype O:183/Shigella boydii serotype O:10 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8234'}, {'name': 'tricin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7995'}, {'name': 'bikaverin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7692'}, {'name': '6-methylpretetramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7811'}, {'name': 'beta myrcene degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7136'}, {'name': 'phytosterol biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2541'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-RNO-3769402'}, {'name': 'Synthesis of very long-chain fatty acyl-CoAs', 'databaseName': 'Reactome', 'id': 'R-DRE-75876'}, {'name': 'Synthesis of very long-chain fatty acyl-CoAs', 'databaseName': 'Reactome', 'id': 'R-MMU-75876'}, {'name': 'Sulfur amino acid metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-1614635'}, {'name': '6-gingerol analog biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6920'}, {'name': 'alginate biosynthesis II (bacterial)', 'databaseName': 'MetaCyc', 'id': 'PWY-6082'}, {'name': 'icosapentaenoate biosynthesis V (8-desaturase, lower eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-7602'}, {'name': 'Mineralocorticoid biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-193993'}, {'name': 'baumannoferrin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7988'}, {'name': '10-methylstearate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8172'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-CEL-71403'}, {'name': 'stipitatate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7687'}, {'name': 'The canonical retinoid cycle in rods (twilight vision)', 'databaseName': 'Reactome', 'id': 'R-XTR-2453902'}, {'name': 'Pentose phosphate pathway', 'databaseName': 'Reactome', 'id': 'R-SPO-71336'}, {'name': 'alcaligin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6379'}, {'name': 'phenylethanol glycoconjugate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7074'}, {'name': 'NFE2L2 regulating TCA cycle genes', 'databaseName': 'Reactome', 'id': 'R-HSA-9818025'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-SPO-5661270'}, {'name': 'progesterone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7299'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-RNO-4641265'}, {'name': 'pyrimidine deoxyribonucleotides de novo biosynthesis IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7198'}, {'name': 'the visual cycle (insects)', 'databaseName': 'MetaCyc', 'id': 'PWY-7042'}, {'name': 'factor 430 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5196'}, {'name': 'Ethanol oxidation', 'databaseName': 'Reactome', 'id': 'R-DDI-71384'}, {'name': 'T-2 toxin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7730'}, {'name': 'Transcriptional activation of mitochondrial biogenesis', 'databaseName': 'Reactome', 'id': 'R-SSC-2151201'}, {'name': 'glycerol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6131'}, {'name': 'Synthesis of very long-chain fatty acyl-CoAs', 'databaseName': 'Reactome', 'id': 'R-CEL-75876'}, {'name': 'formate assimilation into 5,10-methylenetetrahydrofolate', 'databaseName': 'MetaCyc', 'id': 'PWY-1722'}, {'name': 'phenolic malonylglucosides biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6930'}, {'name': 'salinosporamide A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6627'}, {'name': 'Formation of xylulose-5-phosphate', 'databaseName': 'Reactome', 'id': 'R-HSA-5661270'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-DRE-1483166'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-MMU-6798695'}, {'name': 'D-arabinose degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5519'}, {'name': 'Interconversion of nucleotide di- and triphosphates', 'databaseName': 'Reactome', 'id': 'R-HSA-499943'}, {'name': 'Neutrophil degranulation', 'databaseName': 'Reactome', 'id': 'R-CEL-6798695'}, {'name': '2-oxoglutarate-derived opine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8451'}, {'name': 'kainate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8080'}, {'name': 'aurachin RE biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7405'}, {'name': 'Alpha-oxidation of phytanate', 'databaseName': 'Reactome', 'id': 'R-RNO-389599'}, {'name': 'polyhydroxydecanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6657'}, {'name': 'testosterone degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8155'}, {'name': '4-deoxy-L-threo-hex-4-enopyranuronate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6507'}, {'name': 'Escherichia coli serotype O:51/Salmonella enterica serotype O:57 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8242'}, {'name': 'alpha-solanine/alpha-chaconine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5666'}, {'name': 'chlorophyll a biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5086'}, {'name': 'RA biosynthesis pathway', 'databaseName': 'Reactome', 'id': 'R-XTR-5365859'}, {'name': 'Methylation', 'databaseName': 'Reactome', 'id': 'R-HSA-156581'}, {'name': '(S)-propane-1,2-diol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7013'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-SCE-71403'}, {'name': 'dodecaprenyl diphosphate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5817'}, {'name': 'Synthesis of PA', 'databaseName': 'Reactome', 'id': 'R-SCE-1483166'}, {'name': 'TP53 Regulates Transcription of Genes Involved in Cytochrome C Release', 'databaseName': 'Reactome', 'id': 'R-HSA-6803204'}, {'name': 'Gluconeogenesis', 'databaseName': 'Reactome', 'id': 'R-SPO-70263'}, {'name': '11-oxyandrogens biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8202'}, {'name': 'Vitamin B5 (pantothenate) metabolism', 'databaseName': 'Reactome', 'id': 'R-RNO-199220'}, {'name': 'hentriaconta-3,6,9,12,15,19,22,25,28-nonaene biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7027'}, {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7339'}, {'name': 'Bicarbonate transporters', 'databaseName': 'Reactome', 'id': 'R-MMU-425381'}, {'name': 'ophthalmate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8043'}, {'name': 'backdoor pathway of androgen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8200'}, {'name': 'acinetobactin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7984'}, {'name': 'protein S-nitrosylation and denitrosylation', 'databaseName': 'MetaCyc', 'id': 'PWY-7798'}, {'name': 'Formation of the active cofactor, UDP-glucuronate', 'databaseName': 'Reactome', 'id': 'R-DME-173599'}, {'name': 'Citric acid cycle (TCA cycle)', 'databaseName': 'Reactome', 'id': 'R-MMU-71403'}, {'name': 'vernolate biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-7634'}, {'name': 'eNOS activation', 'databaseName': 'Reactome', 'id': 'R-HSA-203615'}, {'name': 'GDP-D-perosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5739'}, {'name': 'Escherichia coli serotype O:21/Salmonella enterica serotype O:38 O antigen biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8235'}, {'name': 'lolitrem B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7572'}]}}, 'locations': [{'start': 150, 'end': 336, 'representative': False, 'hmmLength': 188, 'location-fragments': [{'start': 150, 'end': 336, 'dc-status': 'CONTINUOUS'}]}], 'evalue': 1.05e-50, 'model-ac': '0053195'}, {'signature': {'accession': 'PS00065', 'name': 'D_2_HYDROXYACID_DH_1', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenases NAD-binding signature.', 'signatureLibraryRelease': {'library': 'PROSITE_PATTERNS', 'version': '2023_05'}, 'entry': {'accession': 'IPR029752', 'name': 'D-isomer_DH_CS1', 'description': 'D-isomer specific 2-hydroxyacid dehydrogenase, NAD-binding domain conserved site 1', 'type': 'CONSERVED_SITE', 'goXRefs': [], 'pathwayXRefs': [{'name': '24-epi-campesterol, fucosterol, and clionasterol biosynthesis (diatoms)', 'databaseName': 'MetaCyc', 'id': 'PWY-8238'}, {'name': 'zwittermicin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7694'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-XTR-983168'}, {'name': '(-)-dehydrodiconiferyl alcohol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7557'}, {'name': 'UDP-N-acetyl-beta-L-quinovosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7331'}, {'name': 'anaerobic energy metabolism (invertebrates, cytosol)', 'databaseName': 'MetaCyc', 'id': 'PWY-7383'}, {'name': 'stearate biosynthesis I (animals)', 'databaseName': 'MetaCyc', 'id': 'PWY-5972'}, {'name': 'autoinducer CAI-1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7621'}, {'name': 'dTDP-alpha-D-olivose, dTDP-alpha-D-oliose and dTDP-alpha-D-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6973'}, {'name': 'septacidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8366'}, {'name': 'bisphenol A degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7757'}, {'name': 'pyruvate fermentation to (R)-lactate', 'databaseName': 'MetaCyc', 'id': 'PWY-8274'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-RNO-983168'}, {'name': 'GDP-6-deoxy-D-altro-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7610'}, {'name': 'photorespiration III', 'databaseName': 'MetaCyc', 'id': 'PWY-8363'}, {'name': 'salidroside biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6802'}, {'name': 'sphingofungin B-D biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8464'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-MMU-3899300'}, {'name': 'butanol and isobutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7396'}, {'name': '2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA', 'databaseName': 'MetaCyc', 'id': 'PWY-5652'}, {'name': 'pyruvate fermentation to butanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-6583'}, {'name': '9-cis, 11-trans-octadecadienoyl-CoA degradation (isomerase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7340'}, {'name': 'D-galactose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6693'}, {'name': 'sitosterol degradation to androstenedione', 'databaseName': 'MetaCyc', 'id': 'PWY-6948'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-RNO-977347'}, {'name': '1,2-propanediol biosynthesis from lactate (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7541'}, {'name': '(S)-camphor degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6989'}, {'name': 'pyruvate fermentation to ethanol I', 'databaseName': 'MetaCyc', 'id': 'PWY-5480'}, {'name': 'meso-butanediol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-6392'}, {'name': 'daunorubicin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7352'}, {'name': 'oxalate degradation VI', 'databaseName': 'MetaCyc', 'id': 'PWY-7985'}, {'name': 'L-cysteine biosynthesis IX (Trichomonas vaginalis)', 'databaseName': 'MetaCyc', 'id': 'PWY-8010'}, {'name': 'L-isoleucine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5101'}, {'name': 'representative whole plasma arachidonate-derived icosanoids I', 'databaseName': 'MetaCyc', 'id': 'PWY-8410'}, {'name': 'kanosamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5978'}, {'name': 'pyrethrin I biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6926'}, {'name': 'epoxypseudoisoeugenol-2-methylbutanoate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5882'}, {'name': 'dTDP-4-O-demethyl-beta-L-noviose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7301'}, {'name': 'reductive TCA cycle II', 'databaseName': 'MetaCyc', 'id': 'PWY-5392'}, {'name': 'phosphinothricin tripeptide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6322'}, {'name': 'sulfolactate degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-6637'}, {'name': 'L-tyrosine degradation V (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8016'}, {'name': 'dTDP-beta-L-digitoxose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7657'}, {'name': 'ephedrine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5883'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-CEL-983168'}, {'name': 'rosmarinic acid biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-5049'}, {'name': 'oryzalexin A, B, and C biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7489'}, {'name': 'L-galactose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7568'}, {'name': 'Signaling by TCF7L2 mutants', 'databaseName': 'Reactome', 'id': 'R-HSA-5339700'}, {'name': 'docosahexaenoate biosynthesis III (6-desaturase, mammals)', 'databaseName': 'MetaCyc', 'id': 'PWY-7606'}, {'name': 'hygromycin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8367'}, {'name': 'L-ascorbate degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-6704'}, {'name': 'dTDP-beta-L-mycarose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6976'}, {'name': 'superpathway of nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7342'}, {'name': 'terrequinone A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7520'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-HSA-3769402'}, {'name': 'CDP-6-deoxy-D-gulose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8139'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-CEL-4641265'}, {'name': 'L-homomethionine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1186'}, {'name': 'pyruvate fermentation to isobutanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7111'}, {'name': 'L-canavanine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8419'}, {'name': '10-trans-heptadecenoyl-CoA degradation (reductase-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7338'}, {'name': 'methylaspartate cycle', 'databaseName': 'MetaCyc', 'id': 'PWY-6728'}, {'name': 'D-xylose degradation to ethylene glycol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7178'}, {'name': 'tropane alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5317'}, {'name': 'TCA cycle V (2-oxoglutarate synthase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6969'}, {'name': 'acetaldehyde biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6333'}, {'name': 'oxalate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6696'}, {'name': 'CDP-ascarylose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5830'}, {'name': 'pterocarpan phytoalexins modification (maackiain, medicarpin, pisatin, phaseollin)', 'databaseName': 'MetaCyc', 'id': 'PWY-7615'}, {'name': '3-sulfopropanediol degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-6634'}, {'name': 'traumatin and (Z)-3-hexen-1-yl acetate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5410'}, {'name': 'noradrenaline and adrenaline degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6342'}, {'name': 'nicotine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5316'}, {'name': 'L-valine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-5057'}, {'name': 'vancomycin resistance I', 'databaseName': 'MetaCyc', 'id': 'PWY-6454'}, {'name': '3-sulfopropanediol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8214'}, {'name': 'tetramethylpyrazine degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7403'}, {'name': 'pinitol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6738'}, {'name': 'guadinomine B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7693'}, {'name': '6-diazo-5-oxo-L-norleucine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8297'}, {'name': 'detoxification of reactive carbonyls in chloroplasts', 'databaseName': 'MetaCyc', 'id': 'PWY-6786'}, {'name': 'isethionate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-2601'}, {'name': '1,3-propanediol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-7385'}, {'name': 'L-tryptophan degradation V (side chain pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-3162'}, {'name': 'arabidopyrone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7682'}, {'name': 'dTDP-beta-L-olivose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6974'}, {'name': 'sporopollenin precursors biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6733'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-HSA-3899300'}, {'name': 'SUMOylation of transcription cofactors', 'databaseName': 'Reactome', 'id': 'R-RNO-3899300'}, {'name': 'propane degradation I', 'databaseName': 'MetaCyc', 'id': 'PWY-7774'}, {'name': 'formate oxidation to CO2', 'databaseName': 'MetaCyc', 'id': 'PWY-1881'}, {'name': '10-cis-heptadecenoyl-CoA degradation (yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7337'}, {'name': 'androstenedione degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8152'}, {'name': 'rosmarinic acid biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-5048'}, {'name': 'mupirocin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8012'}, {'name': 'antimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8374'}, {'name': 'erinacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8408'}, {'name': 'jasmonic acid biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-735'}, {'name': 'ecdysone and 20-hydroxyecdysone biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7300'}, {'name': 'pyruvate fermentation to hexanol (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6863'}, {'name': 'C4 photosynthetic carbon assimilation cycle, NAD-ME type', 'databaseName': 'MetaCyc', 'id': 'PWY-7115'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-MMU-983168'}, {'name': 'L-threitol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7787'}, {'name': 'suberin monomers biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-1121'}, {'name': 'cis-geranyl-CoA degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6672'}, {'name': 'representative whole plasma arachidonate-derived icosanoids II', 'databaseName': 'MetaCyc', 'id': 'PWY-8411'}, {'name': 'aurofusarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7695'}, {'name': 'bile acid biosynthesis, neutral pathway', 'databaseName': 'MetaCyc', 'id': 'PWY-6061'}, {'name': 'phenylethanol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5751'}, {'name': 'mithramycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7045'}, {'name': 'arachidonate metabolites biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8397'}, {'name': 'kanamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7000'}, {'name': 'aclacinomycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7354'}, {'name': 'cholesterol degradation to androstenedione II (cholesterol dehydrogenase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6946'}, {'name': 'serotonin degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6313'}, {'name': 'heme d1 biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7554'}, {'name': 'methyl tert-butyl ether degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7779'}, {'name': 'fusicoccin A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6659'}, {'name': '(Z)-phenylmethanethial S-oxide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6539'}, {'name': 'L-phenylalanine degradation IV (mammalian, via side chain)', 'databaseName': 'MetaCyc', 'id': 'PWY-6318'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-BTA-983168'}, {'name': 'myxochelin A and B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8383'}, {'name': 'S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-8132'}, {'name': 'purine nucleobases degradation II (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-5497'}, {'name': 'L-isoleucine degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-5078'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-HSA-977347'}, {'name': '11-cis-3-hydroxyretinal biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7043'}, {'name': 'L-phenylalanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8014'}, {'name': 'TCA cycle II (plants and fungi)', 'databaseName': 'MetaCyc', 'id': 'PWY-5690'}, {'name': 'leporin B biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8370'}, {'name': 'superpathway of glucose and xylose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-6901'}, {'name': 'photorespiration II', 'databaseName': 'MetaCyc', 'id': 'PWY-8362'}, {'name': 'abscisic acid degradation to phaseic acid', 'databaseName': 'MetaCyc', 'id': 'PWY-5271'}, {'name': 'L-glucose degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7130'}, {'name': 'erythritol biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8373'}, {'name': '3-amino-4,7-dihydroxy-coumarin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7304'}, {'name': 'bikaverin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7692'}, {'name': '6-methylpretetramide biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7811'}, {'name': '3-methylbutanol biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6871'}, {'name': 'phytosterol biosynthesis (plants)', 'databaseName': 'MetaCyc', 'id': 'PWY-2541'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-RNO-3769402'}, {'name': 'photorespiration I', 'databaseName': 'MetaCyc', 'id': 'PWY-181'}, {'name': '6-gingerol analog biosynthesis (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6920'}, {'name': 'dihydroconiferyl alcohol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6302'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-DRE-983168'}, {'name': 'phosalacine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7769'}, {'name': 'phenolphthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7742'}, {'name': 'mycolate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWYG-321'}, {'name': 'GDP-6-deoxy-D-manno-heptose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7613'}, {'name': 'L-alanine degradation VI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8188'}, {'name': 'propane degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7775'}, {'name': '(S)-lactate fermentation to propanoate, acetate and hydrogen', 'databaseName': 'MetaCyc', 'id': 'PWY-8086'}, {'name': 'monolignol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-361'}, {'name': 'L-arabinose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7295'}, {'name': 'apicidin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7667'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-MMU-977347'}, {'name': '8-O-methylfusarubin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7689'}, {'name': 'pyruvate fermentation to ethanol II', 'databaseName': 'MetaCyc', 'id': 'PWY-5486'}, {'name': '4-ethylphenol degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6080'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-MMU-3769402'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-RNO-4641265'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-DME-983168'}, {'name': 't-anethole biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5867'}, {'name': 'the visual cycle (insects)', 'databaseName': 'MetaCyc', 'id': 'PWY-7042'}, {'name': '2-deoxy-D-ribose degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-8058'}, {'name': 'D-xylose degradation IV', 'databaseName': 'MetaCyc', 'id': 'PWY-7294'}, {'name': 'quinoxaline-2-carboxylate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7734'}, {'name': 'glycine betaine biosynthesis II (Gram-positive bacteria)', 'databaseName': 'MetaCyc', 'id': 'PWY-3722'}, {'name': 'sulfolactate degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-6638'}, {'name': '3-hydroxy-L-homotyrosine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7649'}, {'name': 'mannojirimycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7531'}, {'name': 'cinchona alkaloids biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5848'}, {'name': 'L-glutamate degradation XI (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8190'}, {'name': 'dTDP-L-daunosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7814'}, {'name': '4-hydroxybenzoate biosynthesis I (eukaryotes)', 'databaseName': 'MetaCyc', 'id': 'PWY-5754'}, {'name': 'p-cymene degradation to p-cumate', 'databaseName': 'MetaCyc', 'id': 'PWY-741'}, {'name': 'limonene degradation IV (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8029'}, {'name': 'Serine biosynthesis', 'databaseName': 'Reactome', 'id': 'R-SSC-977347'}, {'name': 'salinosporamide A biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6627'}, {'name': 'erythritol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7788'}, {'name': 'dTDP-beta-L-megosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7104'}, {'name': '2-keto-L-gulonate biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5782'}, {'name': 'UDP-beta-L-rhamnose biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-3261'}, {'name': 'androstenedione degradation I (aerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-6944'}, {'name': 'validamycin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5818'}, {'name': 'L-phenylalanine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5079'}, {'name': 'L-methionine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5082'}, {'name': 'phthiocerol biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7741'}, {'name': 'superpathway of glyoxylate cycle and fatty acid degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-561'}, {'name': 'myo-inositol degradation II', 'databaseName': 'MetaCyc', 'id': 'PWY-7241'}, {'name': 'cholesterol degradation to androstenedione I (cholesterol oxidase)', 'databaseName': 'MetaCyc', 'id': 'PWY-6945'}, {'name': 'testosterone degradation (anaerobic)', 'databaseName': 'MetaCyc', 'id': 'PWY-8155'}, {'name': '(4Z,7Z,10Z,13Z,16Z)-docosapentaenoate biosynthesis (6-desaturase)', 'databaseName': 'MetaCyc', 'id': 'PWY-7726'}, {'name': 'alpha-solanine/alpha-chaconine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-5666'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-HSA-4641265'}, {'name': 'Deactivation of the beta-catenin transactivating complex', 'databaseName': 'Reactome', 'id': 'R-CEL-3769402'}, {'name': 'dehydrophos biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-6682'}, {'name': 'pyruvate fermentation to butanol II (engineered)', 'databaseName': 'MetaCyc', 'id': 'PWY-6883'}, {'name': '(S)-propane-1,2-diol degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7013'}, {'name': 'Antigen processing: Ubiquitination & Proteasome degradation', 'databaseName': 'Reactome', 'id': 'R-HSA-983168'}, {'name': 'Repression of WNT target genes', 'databaseName': 'Reactome', 'id': 'R-MMU-4641265'}, {'name': 'UDP-N-acetyl-alpha-D-fucosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7333'}, {'name': 'dimethylsulfoniopropanoate biosynthesis III (algae and phytoplankton)', 'databaseName': 'MetaCyc', 'id': 'PWY-6053'}, {'name': 'juglone degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7825'}, {'name': '4-oxopentanoate degradation', 'databaseName': 'MetaCyc', 'id': 'PWY-7948'}, {'name': '(2R)-methoxymalonyl-[acp] biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8426'}, {'name': 'patulin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7490'}, {'name': 'cyathin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-8409'}, {'name': 'cutin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-321'}, {'name': 'fatty acid beta-oxidation VII (yeast peroxisome)', 'databaseName': 'MetaCyc', 'id': 'PWY-7288'}, {'name': 'ecdysteroid metabolism (arthropods)', 'databaseName': 'MetaCyc', 'id': 'PWY-7321'}, {'name': 'dTDP-beta-L-4-epi-vancosamine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7440'}, {'name': 'pyruvate fermentation to ethanol III', 'databaseName': 'MetaCyc', 'id': 'PWY-6587'}, {'name': 'meso-butanediol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-6391'}, {'name': '10-trans-heptadecenoyl-CoA degradation (MFE-dependent, yeast)', 'databaseName': 'MetaCyc', 'id': 'PWY-7339'}, {'name': \"cytidine-5'-diphosphate-glycerol biosynthesis\", 'databaseName': 'MetaCyc', 'id': 'PWY-6626'}, {'name': 'prodigiosin biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7547'}, {'name': 'erythritol biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-8372'}, {'name': 'L-tryptophan degradation XIII (reductive Stickland reaction)', 'databaseName': 'MetaCyc', 'id': 'PWY-8017'}, {'name': 'tryptoquialanine biosynthesis', 'databaseName': 'MetaCyc', 'id': 'PWY-7660'}, {'name': 'formaldehyde assimilation I (serine pathway)', 'databaseName': 'MetaCyc', 'id': 'PWY-1622'}, {'name': 'L-serine biosynthesis II', 'databaseName': 'MetaCyc', 'id': 'PWY-8011'}, {'name': 'L-leucine degradation III', 'databaseName': 'MetaCyc', 'id': 'PWY-5076'}, {'name': 'paromamine biosynthesis I', 'databaseName': 'MetaCyc', 'id': 'PWY-7014'}]}}, 'locations': [{'start': 195, 'end': 222, 'representative': False, 'level': 'STRONG', 'cigarAlignment': '5M1I4M13I2M1I2M', 'alignment': 'VGTIAaGRIGlavlrrlapfdvhLHyTD', 'location-fragments': [{'start': 195, 'end': 222, 'dc-status': 'CONTINUOUS'}]}], 'model-ac': 'PS00065'}, {'signature': {'accession': 'G3DSA:3.40.50.720:FF:000057', 'name': None, 'description': 'Formate dehydrogenase', 'signatureLibraryRelease': {'library': 'FUNFAM', 'version': '4.3.0'}, 'entry': None}, 'locations': [{'start': 147, 'end': 337, 'representative': False, 'hmmStart': 1, 'hmmEnd': 197, 'hmmLength': 197, 'hmmBounds': 'COMPLETE', 'evalue': 2.9e-87, 'score': 293.3, 'envelopeStart': 147, 'envelopeEnd': 337, 'postProcessed': False, 'location-fragments': [{'start': 147, 'end': 337, 'dc-status': 'CONTINUOUS'}]}], 'evalue': 2.2e-87, 'score': 293.7, 'model-ac': '3.40.50.720-FF-000057'}], 'xref': [{'name': 'EMBOSS_001', 'id': 'EMBOSS_001'}]}]\n"
     ]
    }
   ],
   "source": [
    "# Iterating over all key-value pairs\n",
    "for key, value in data.items():\n",
    "    print(key, value)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "def search_json(json_data, pattern):\n",
    "    matches = []\n",
    "\n",
    "    def search_recursive(data):\n",
    "        if isinstance(data, dict):\n",
    "            for key, value in data.items():\n",
    "                if isinstance(value, (dict, list)):\n",
    "                    search_recursive(value)\n",
    "                elif isinstance(value, str) and pattern in value:\n",
    "                    matches.append(value)\n",
    "        elif isinstance(data, list):\n",
    "            for item in data:\n",
    "                search_recursive(item)\n",
    "\n",
    "    search_recursive(json_data)\n",
    "    return matches"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['IPR029753',\n",
       " 'IPR033689',\n",
       " 'IPR006139',\n",
       " 'IPR029753',\n",
       " 'IPR006140',\n",
       " 'IPR033689',\n",
       " 'IPR036291',\n",
       " 'IPR029752']"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "search_json(data,\"IPR0\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In all the data, look for the IPR0 pattern where Type == FAMILY.\n",
    "In all the data[0]['results']['matches'][n] look for the pattern where entry{'Type' == 'FAMILY'}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Looking first for family IDs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = []\n",
    "for n in range(len(data['results'][0]['matches'])):\n",
    "    try:\n",
    "        signature_entry_type = data['results'][0]['matches'][n][\"signature\"].get('entry', {}).get('type')\n",
    "        if signature_entry_type == 'FAMILY':\n",
    "            a.append(data['results'][0]['matches'][n][\"signature\"]['entry']['accession'])\n",
    "    except AttributeError:\n",
    "        # If 'signature' or 'entry' is not found, continue with the next iteration\n",
    "        continue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'IPR033689'"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a[0]\n",
    "#search_json(a,\"IPR0\") # This should look for only Family Ids "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "metadata": {},
   "outputs": [],
   "source": [
    "#data['results'][0]['matches'][2][\"signature\"].get('entry', {}).get('type') == 'FAMILY'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Looking into hmm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "b = []\n",
    "for n in range(len(data['results'][0]['matches'])):\n",
    "    try:\n",
    "        signature_library_type = data['results'][0]['matches'][n][\"signature\"]['signatureLibraryRelease'].get('library')\n",
    "        if signature_library_type == 'PFAM':\n",
    "            b.append(data['results'][0]['matches'][n][\"signature\"]['accession'])\n",
    "    except AttributeError:\n",
    "        # If 'signature' or 'entry' is not found, continue with the next iteration\n",
    "        continue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'PF00389'"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'PF00389'"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['results'][0]['matches'][5][\"signature\"]['accession']"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "MSA_tool",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.18"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
